{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#!/usr/bin/env python3\n",
    "# -*- coding: utf-8 -*-\n",
    "from django.db import models\n",
    "# Create your models here.\n",
    "import psycopg2\n",
    "#from config import config\n",
    "from django.test.html import Element\n",
    "import os\n",
    "\n",
    "from configparser import ConfigParser\n",
    "\n",
    "\n",
    "def config(section='postgresql'):\n",
    "    if os.getcwd()[0:12]!=\"/home/traore\":\n",
    "    \tfilename=os.getcwd()+'/database.ini'\n",
    "    else:\n",
    "    \tfilename=os.getcwd()+'/database2.ini'\n",
    "    # create a parser\n",
    "    parser = ConfigParser()\n",
    "    # read config file\n",
    "    parser.read(filename)\n",
    "    # get section, default to postgresql\n",
    "    db = {}\n",
    "    if parser.has_section(section):\n",
    "        params = parser.items(section)\n",
    "        for param in params:\n",
    "            db[param[0]] = param[1]\n",
    "    else:\n",
    "        raise Exception('Section {0} not found in the {1} file'.format(section, filename))\n",
    " \n",
    "    return db\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "def connect():\n",
    "    \"\"\" Connect to the PostgreSQL database server \"\"\"\n",
    "    conn = None\n",
    "    try:\n",
    "        # read connection parameters\n",
    "        params = config()\n",
    "        # connect to the PostgreSQL server\n",
    "        print('Connecting to the PostgreSQL database...')\n",
    "        conn = psycopg2.connect(**params)\n",
    "        # create a cursor\n",
    "        cur = conn.cursor()\n",
    "        # execute a statement\n",
    "        print('PostgreSQL database version:')\n",
    "        cur.execute('SELECT version()')\n",
    "        # display the PostgreSQL database server version\n",
    "        db_version = cur.fetchone()\n",
    "        print(db_version)\n",
    "     # close the communication with the PostgreSQL\n",
    "        cur.close()\n",
    "    except (Exception, psycopg2.DatabaseError) as error:\n",
    "        print(error)\n",
    "    finally:\n",
    "        if conn is not None:\n",
    "            conn.close()\n",
    "            print('Database connection closed.')\n",
    "\n",
    "\n",
    "def read_data_sql(requete,element):\n",
    "    \"\"\"\n",
    "    input requete outupt data\n",
    "    requete='select * from chips.experiment;'\n",
    "    \"\"\"\n",
    "    conn = None\n",
    "    params = config()\n",
    "    conn = psycopg2.connect(**params)\n",
    "    cursor = conn.cursor()#cursor_factory=psycopg2.extras.DictCursor\n",
    "    cursor.execute(requete)\n",
    "    colnames = [desc[0] for desc in cursor.description]\n",
    "    memory = cursor.fetchall()\n",
    "    conn.close()\n",
    "    cpt=0\n",
    "    data=dict()\n",
    "    for ele in range(len(memory)):\n",
    "        cpt=cpt+1\n",
    "        col=[]\n",
    "        for ele1 in memory[ele]:\n",
    "            col.append(str(ele1))\n",
    "        dictionary ={'_key':cpt,\"_value\":dict(zip(colnames,col))}\n",
    "        data[cpt]=dictionary\n",
    "    name=element+'.csv'     \n",
    "    return data\n",
    "\n",
    "def lecture_query_tage(file_query):\n",
    "    \"\"\"\n",
    "    function sert a lire les requetes prédefinir seulement\n",
    "    \"\"\"\n",
    "    query_tag=dict()\n",
    "    with open(file_query,\"r\") as fiche:\n",
    "        data=fiche.readlines()\n",
    "        for line in data:\n",
    "            if line[0]!=\"#\":\n",
    "                if line[0:9]==\"Tag_query\":\n",
    "                    _key=line[11:].replace('\\n','').replace('\"','')\n",
    "                elif line[0:5]==\"query\":\n",
    "                    query_tag[_key]=(line[6:].replace('\\n','').replace('\"',''))            \n",
    "    return query_tag\n",
    "\n",
    "\n",
    "repertoire_requete=lecture_query_tage(os.getcwd()+'/query_tag.ini')\n",
    "\n",
    "def execution_requete(element,value_send,colname_send,order_ele=None):\n",
    "    ### parametres\n",
    "    if os.getcwd()[0:12]!=\"/home/traore\":\n",
    "    \tmy_ppty=\"chips\"\n",
    "    else:\n",
    "    \tmy_ppty=\"public\" #CHIPS\n",
    "\n",
    "   \n",
    "    # creation/tri des infos :(creation de structures de données)\n",
    "    ###Todo\n",
    "    \n",
    "    # Bilan Chiffres nbr de puces par especes\n",
    "    # pas Arabidopsis dans nouveau champ common_name d'array_type et \n",
    "    # analysis_type='Arrays'\n",
    "    # partie requete\n",
    "    my_querySpecies=\"select array_type_name, count(hybridization_id) \"+\\\n",
    "    \"from \"+my_ppty+\".hybridization \"+\\\n",
    "    \"where project_id in (select project_id from \"+my_ppty+\".project where is_public='yes') \"+\\\n",
    "    \"and experiment_id in (select experiment_id from \"+my_ppty+\".experiment \"+\\\n",
    "    \"where analysis_type='Arrays') \"+\\\n",
    "    \"and array_type_name in \"+\\\n",
    "    \"(select a.array_type_name from \"+my_ppty+\".array_type a where a.common_name<>'Arabidopsis')\"+\\\n",
    "    \"group by array_type_name order by array_type_name;\"\n",
    "    \n",
    "    #Pour le nombre de ref biblio\n",
    "    my_reqstat =\"Select count(*) from \"+my_ppty+\".biblio_list;\"\n",
    "    \n",
    "    \n",
    "    #my_req_special=\"select o.organism_name,count(distinct ss.project_id) from \"+my_ppty+\".sample_source ss,\"+my_ppty+\".organism o where ss.project_id in( select project_id from \"+my_ppty+\".project where is_public='yes')  group by o.organism_name;\"\n",
    "    my_req_special=\"select o.organism_name,count(distinct ss.project_id) from \"+my_ppty+\".sample_source ss,\"+my_ppty+\".organism o where ss.project_id in( select project_id from \"+my_ppty+\".project ) and ss.organism_id=o.organism_id group by o.organism_name;\" #where is_public='yes'\n",
    "    \n",
    "    my_req_recherche=\"SELECT * FROM global_search_element('Arabidopsis',param_schemas:=array['chips']);\"\n",
    "    \n",
    "    my_req_recherche1=\"select project_id,Experiment_type,Experiment_name from \"+my_ppty+\".experiment where project_id in (select project_id from \"+my_ppty+\".project where is_public='yes');\"\n",
    "    \n",
    "    #experiement=\"select * from chips.experiment where project_id in (select project_id from chips.project where is_public='yes'));\"\n",
    "    \n",
    "    experiement=\"select * from (select S.project_name,O.project_id,O.experiment_name,O.experiment_type from \"+my_ppty+\".experiment O,\"+my_ppty+\".project S where O.project_id in (select project_id from \"+my_ppty+\".project where is_public='yes')) df limit 100;\"\n",
    "    \n",
    "    \n",
    "    try:\n",
    "        my_req_recherche_av=\"select * from \"+colname_send+\" where (is_public='yes' and project_name like '%\"+value_send+\"%');\"\n",
    "    except:\n",
    "        pass\n",
    "    if element=='my_reqinfo':\n",
    "        requete=my_reqinfo\n",
    "    elif element=='my_querySpecies':\n",
    "        requete=my_querySpecies\n",
    "    elif element=='my_reqstat':\n",
    "        requete=my_reqstat    \n",
    "    elif element=='my_req_special':\n",
    "        requete=my_req_special    \n",
    "    elif element=='my_req_recherche':\n",
    "        requete=my_req_recherche        \n",
    "    elif element=='my_req_recherche_av' and colname_send!=None and value_send!=None:\n",
    "        requete=my_req_recherche_av     \n",
    "    elif element=='my_req_recherche1':\n",
    "        requete=my_req_recherche1\n",
    "    elif element=='experiement':\n",
    "        requete=experiement  \n",
    "    elif (element=='get_project' or element=='get_contact' or element=='my_experience' or 'my_treatment' or 'list_project' or 'type_analysis' or 'type_arrays' or 'all_experiment'):\n",
    "        requete=repertoire_requete[element].replace('schema.',my_ppty+'.')\n",
    "        if value_send!=None:\n",
    "            requete=requete.replace('value_send',value_send)\n",
    "        else:\n",
    "            requete=requete\n",
    "        print(requete)\n",
    "    else:\n",
    "        try:\n",
    "            requete=eval(element)\n",
    "        except:\n",
    "            requete=None\n",
    "            pass\n",
    "    \n",
    "    my_req_special=\"select o.organism_name,count(distinct ss.project_id) from \"+my_ppty+\".sample_source ss,\"+my_ppty+\".organism o where ss.project_id in( select project_id from \"+my_ppty+\".project where is_public='yes') and ss.organism_id=o.organism_id group by o.organism_name;\"\n",
    "    \n",
    "    \n",
    "    \n",
    "    if  element=='my_req_special' and order_ele==None:\n",
    "         requete=my_req_special  \n",
    "    elif element=='my_req_special' and order_ele!=None:    \n",
    "        requete=\"select * from (\"+requete.replace(\";\",')')+' dfg order by dfg.'+order_ele+\";\"\n",
    "    elif  element=='my_querySpecies' and order_ele==None:\n",
    "         requete=my_querySpecies  \n",
    "    elif element=='my_querySpecies' and order_ele!=None:    \n",
    "        requete=\"select * from (\"+requete.replace(\";\",')')+' dfg order by dfg.'+order_ele+\";\"\n",
    "    elif  element=='my_req_recherche1' and order_ele==None:\n",
    "         requete=my_req_recherche1  \n",
    "    elif element=='my_req_recherche1' and order_ele!=None:    \n",
    "        requete=\"select * from (\"+requete.replace(\";\",')')+' dfg order by dfg.'+order_ele+\";\"\n",
    "    if requete!=None:\n",
    "        return read_data_sql(requete,element)    \n",
    "    else:\n",
    "        print(\"error\") "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SELECT project.project_id, sample.organ,sample.sample_id,sample.sample_name, experiment.experiment_name,experiment.experiment_id,experiment.experiment_type, experiment.array_type, project.project_name, project.title,organism.organism_id,organism.organism_name FROM chips.organism,chips.sample_source,chips.experiment, chips.project, chips.sample WHERE project.project_id = experiment.project_id AND   sample.project_id = experiment.project_id AND experiment.experiment_id=sample.experiment_id and sample_source.experiment_id=sample.experiment_id and sample.project_id = sample_source.project_id and sample_source.organism_id = organism.organism_id limit 10;\n"
     ]
    }
   ],
   "source": [
    "data=execution_requete('select_all_tab','','')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{1: {'_key': 1,\n",
       "  '_value': {'array_type': 'CAGE',\n",
       "   'experiment_id': '120',\n",
       "   'experiment_name': 'seedlings_10j_Columbia_WT_vs_ddm1',\n",
       "   'experiment_type': 'gene knock out',\n",
       "   'organ': 'whole plant',\n",
       "   'organism_id': '1',\n",
       "   'organism_name': 'Arabidopsis thaliana',\n",
       "   'project_id': '81',\n",
       "   'project_name': 'CAGE-06_02_ddm1-VIBchips_PRIVATE',\n",
       "   'sample_id': '1071',\n",
       "   'sample_name': 'Col_ddm1_10j',\n",
       "   'title': 'Microarray analysis of transcriptionnal changes induced  by the ddm1 mutation'}},\n",
       " 2: {'_key': 2,\n",
       "  '_value': {'array_type': 'CAGE',\n",
       "   'experiment_id': '120',\n",
       "   'experiment_name': 'seedlings_10j_Columbia_WT_vs_ddm1',\n",
       "   'experiment_type': 'gene knock out',\n",
       "   'organ': 'whole plant',\n",
       "   'organism_id': '1',\n",
       "   'organism_name': 'Arabidopsis thaliana',\n",
       "   'project_id': '81',\n",
       "   'project_name': 'CAGE-06_02_ddm1-VIBchips_PRIVATE',\n",
       "   'sample_id': '1070',\n",
       "   'sample_name': 'Col_WT_10j',\n",
       "   'title': 'Microarray analysis of transcriptionnal changes induced  by the ddm1 mutation'}},\n",
       " 3: {'_key': 3,\n",
       "  '_value': {'array_type': 'CATMA',\n",
       "   'experiment_id': '133',\n",
       "   'experiment_name': 'RfPPR',\n",
       "   'experiment_type': 'wt vs mutant comparison',\n",
       "   'organ': 'flower',\n",
       "   'organism_id': '3',\n",
       "   'organism_name': 'Brassica napus',\n",
       "   'project_id': '85',\n",
       "   'project_name': 'RA05-11_CMSRapeseed',\n",
       "   'sample_id': '1145',\n",
       "   'sample_name': 'PAC_8_2/2_3',\n",
       "   'title': 'Comparing nuclear gene expression in two CMS-systems using microarray technology'}},\n",
       " 4: {'_key': 4,\n",
       "  '_value': {'array_type': 'CATMA',\n",
       "   'experiment_id': '133',\n",
       "   'experiment_name': 'RfPPR',\n",
       "   'experiment_type': 'wt vs mutant comparison',\n",
       "   'organ': 'flower',\n",
       "   'organism_id': '3',\n",
       "   'organism_name': 'Brassica napus',\n",
       "   'project_id': '85',\n",
       "   'project_name': 'RA05-11_CMSRapeseed',\n",
       "   'sample_id': '1146',\n",
       "   'sample_name': 'PPR_8_5/2_1',\n",
       "   'title': 'Comparing nuclear gene expression in two CMS-systems using microarray technology'}},\n",
       " 5: {'_key': 5,\n",
       "  '_value': {'array_type': 'CATMA',\n",
       "   'experiment_id': '133',\n",
       "   'experiment_name': 'RfPPR',\n",
       "   'experiment_type': 'wt vs mutant comparison',\n",
       "   'organ': 'flower',\n",
       "   'organism_id': '3',\n",
       "   'organism_name': 'Brassica napus',\n",
       "   'project_id': '85',\n",
       "   'project_name': 'RA05-11_CMSRapeseed',\n",
       "   'sample_id': '1147',\n",
       "   'sample_name': 'PPR_8_5/2_3',\n",
       "   'title': 'Comparing nuclear gene expression in two CMS-systems using microarray technology'}},\n",
       " 6: {'_key': 6,\n",
       "  '_value': {'array_type': 'CATMA',\n",
       "   'experiment_id': '133',\n",
       "   'experiment_name': 'RfPPR',\n",
       "   'experiment_type': 'wt vs mutant comparison',\n",
       "   'organ': 'flower',\n",
       "   'organism_id': '3',\n",
       "   'organism_name': 'Brassica napus',\n",
       "   'project_id': '85',\n",
       "   'project_name': 'RA05-11_CMSRapeseed',\n",
       "   'sample_id': '1142',\n",
       "   'sample_name': 'CMS_8_4/2_2',\n",
       "   'title': 'Comparing nuclear gene expression in two CMS-systems using microarray technology'}},\n",
       " 7: {'_key': 7,\n",
       "  '_value': {'array_type': 'CATMA',\n",
       "   'experiment_id': '133',\n",
       "   'experiment_name': 'RfPPR',\n",
       "   'experiment_type': 'wt vs mutant comparison',\n",
       "   'organ': 'flower',\n",
       "   'organism_id': '3',\n",
       "   'organism_name': 'Brassica napus',\n",
       "   'project_id': '85',\n",
       "   'project_name': 'RA05-11_CMSRapeseed',\n",
       "   'sample_id': '1143',\n",
       "   'sample_name': 'CMS_8_4/2_3',\n",
       "   'title': 'Comparing nuclear gene expression in two CMS-systems using microarray technology'}},\n",
       " 8: {'_key': 8,\n",
       "  '_value': {'array_type': 'CATMA',\n",
       "   'experiment_id': '133',\n",
       "   'experiment_name': 'RfPPR',\n",
       "   'experiment_type': 'wt vs mutant comparison',\n",
       "   'organ': 'flower',\n",
       "   'organism_id': '3',\n",
       "   'organism_name': 'Brassica napus',\n",
       "   'project_id': '85',\n",
       "   'project_name': 'RA05-11_CMSRapeseed',\n",
       "   'sample_id': '1144',\n",
       "   'sample_name': 'PAC_8_2/2_2',\n",
       "   'title': 'Comparing nuclear gene expression in two CMS-systems using microarray technology'}},\n",
       " 9: {'_key': 9,\n",
       "  '_value': {'array_type': 'Affymetrix',\n",
       "   'experiment_id': '289',\n",
       "   'experiment_name': 'AFFY_seed_kinetic_Wheat',\n",
       "   'experiment_type': 'time course',\n",
       "   'organ': 'seed',\n",
       "   'organism_id': '4',\n",
       "   'organism_name': 'Triticum aestivum',\n",
       "   'project_id': '184',\n",
       "   'project_name': 'AFFY_seed_kinetic_Wheat',\n",
       "   'sample_id': '2912',\n",
       "   'sample_name': 'E_1',\n",
       "   'title': 'Transcriptomic wheat seed'}},\n",
       " 10: {'_key': 10,\n",
       "  '_value': {'array_type': 'Affymetrix',\n",
       "   'experiment_id': '289',\n",
       "   'experiment_name': 'AFFY_seed_kinetic_Wheat',\n",
       "   'experiment_type': 'time course',\n",
       "   'organ': 'seed',\n",
       "   'organism_id': '4',\n",
       "   'organism_name': 'Triticum aestivum',\n",
       "   'project_id': '184',\n",
       "   'project_name': 'AFFY_seed_kinetic_Wheat',\n",
       "   'sample_id': '2911',\n",
       "   'sample_name': 'D_1',\n",
       "   'title': 'Transcriptomic wheat seed'}}}"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "sepcol='#'\n",
    "sep=':'\n",
    "diction={}\n",
    "dictionlev2={}\n",
    "dictionlev3={}\n",
    "for key in data.keys():\n",
    "    level1='project_id:'+data[key]['_value']['project_id']+sepcol+'project_name:'+data[key]['_value']['project_name']\n",
    "    level2='experiment_id:'+data[key]['_value']['experiment_id']+sepcol+'experiment_name:'+data[key]['_value']['experiment_name']+sepcol+'experiment_type:'+data[key]['_value']['experiment_type']+sepcol+'array_type:'+data[key]['_value']['array_type']\n",
    "    level3='organ:'+data[key]['_value']['organ']#'sample_id:'+data[key]['_value']['sample_id']+'sample_name:'+data[key]['_value']['sample_name']+\n",
    "    level4='organism_name:'+data[key]['_value']['organism_name']+sepcol+'organism_id:'+data[key]['_value']['organism_id']\n",
    "    if level1 not in diction.keys():\n",
    "        diction[level1]=[level2]\n",
    "    else:\n",
    "        dat=diction[level1]\n",
    "        dat.append(level2)\n",
    "        diction[level1]=list(set(dat))\n",
    "    if level1+sepcol+level2 not in dictionlev2.keys():\n",
    "        dictionlev2[level1+sepcol+level2]=[level3+sepcol+level4]\n",
    "    else:\n",
    "        dat2=dictionlev2[level1+sepcol+level2]\n",
    "        dat2.append(level3+sepcol+level4)\n",
    "        dictionlev2[level1+sepcol+level2]=list(set(dat2))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'_key':1,'project_id':'85','project_name':'RA05-11_CMSRapeseed','experiment1':{'experiment_id':'133','experiment_name':'RfPPR','experiment_type':'wt vs mutant comparison','array_type':'CATMA','organism1':{'organ':'flower','organism_name':'Brassica napus','organism_id':'3'}}},{'_key':2,'project_id':'81','project_name':'CAGE-06_02_ddm1-VIBchips_PRIVATE','experiment1':{'experiment_id':'120','experiment_name':'seedlings_10j_Columbia_WT_vs_ddm1','experiment_type':'gene knock out','array_type':'CAGE','organism1':{'organ':'whole plant','organism_name':'Arabidopsis thaliana','organism_id':'1'}}},{'_key':3,'project_id':'184','project_name':'AFFY_seed_kinetic_Wheat','experiment1':{'experiment_id':'289','experiment_name':'AFFY_seed_kinetic_Wheat','experiment_type':'time course','array_type':'Affymetrix','organism1':{'organ':'seed','organism_name':'Triticum aestivum','organism_id':'4'}},},\n"
     ]
    }
   ],
   "source": [
    "#dictionlev2\n",
    "text=''\n",
    "cpt=0\n",
    "for key in diction.keys():\n",
    "    cpt=cpt+1\n",
    "    text=text+\"{'_key':\"+str(cpt)+','\n",
    "    for element in key.split('#'):\n",
    "        text=text+\"'\"+element.split(':')[0]+\"':\"+\"'\"+element.split(':')[1]+\"',\"\n",
    "    cpt2=0\n",
    "    for el in diction[key]:\n",
    "        cpt2=cpt2+1\n",
    "        text=text+\"'experiment\"+str(cpt2)+\"':{\"\n",
    "        for val in el.split('#'):\n",
    "            text=text+\"'\"+val.split(':')[0]+\"':'\"+val.split(':')[1]+\"'\"+','\n",
    "        \n",
    "        cpt3=0\n",
    "        for el2 in dictionlev2[key+sepcol+el]:\n",
    "            cpt3=cpt3+1\n",
    "            text=text+\"'organism\"+str(cpt3)+\"':{\"\n",
    "            for val2 in el2.split('#'):\n",
    "                text=text+\"'\"+val2.split(':')[0]+\"':'\"+val2.split(':')[1]+\"'\"+','\n",
    "            text=text+\"}\"\n",
    "        text=text+\"},\"\n",
    "    text=text+\"},\"     \n",
    "text=text.replace(',}}','}}')\n",
    "text=text.replace('},},{','}},{')\n",
    "text=text.replace(\"',}'\",\"'}'\")\n",
    "print(text)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3\n",
      "'project_id':['85','0_1'],'project_name':['RA05-11_CMSRapeseed','0_2'],'experiment_id':'133','experiment_name':'RfPPR','experiment_type':'wt vs mutant comparison','array_type':'CATMA','organ':'flower','organism_name':'Brassica napus','organism_id':'3',\n",
      "3\n",
      "'project_id':['85','0_1'],'project_name':['RA05-11_CMSRapeseed','0_2'],'experiment_id':'133','experiment_name':'RfPPR','experiment_type':'wt vs mutant comparison','array_type':'CATMA','organ':'flower','organism_name':'Brassica napus','organism_id':'3','project_id':['81','0_3'],'project_name':['CAGE-06_02_ddm1-VIBchips_PRIVATE','0_4'],'experiment_id':'120','experiment_name':'seedlings_10j_Columbia_WT_vs_ddm1','experiment_type':'gene knock out','array_type':'CAGE','organ':'whole plant','organism_name':'Arabidopsis thaliana','organism_id':'1',\n",
      "3\n",
      "'project_id':['85','0_1'],'project_name':['RA05-11_CMSRapeseed','0_2'],'experiment_id':'133','experiment_name':'RfPPR','experiment_type':'wt vs mutant comparison','array_type':'CATMA','organ':'flower','organism_name':'Brassica napus','organism_id':'3','project_id':['81','0_3'],'project_name':['CAGE-06_02_ddm1-VIBchips_PRIVATE','0_4'],'experiment_id':'120','experiment_name':'seedlings_10j_Columbia_WT_vs_ddm1','experiment_type':'gene knock out','array_type':'CAGE','organ':'whole plant','organism_name':'Arabidopsis thaliana','organism_id':'1','project_id':['184','0_5'],'project_name':['AFFY_seed_kinetic_Wheat','0_6'],'experiment_id':'289','experiment_name':'AFFY_seed_kinetic_Wheat','experiment_type':'time course','array_type':'Affymetrix','organ':'seed','organism_name':'Triticum aestivum','organism_id':'4',\n"
     ]
    }
   ],
   "source": [
    "#dictionlev2\n",
    "text=''\n",
    "cpt,cpt1=0,0\n",
    "x=0\n",
    "y=0\n",
    "for key in diction.keys():\n",
    "    cpt_special_col=0\n",
    "    cpt=cpt+1\n",
    "\n",
    "    for element in key.split('#'):\n",
    "        cpt1=cpt1+1\n",
    "        x=cpt1\n",
    "        text=text+\"'\"+element.split(':')[0]+\"':\"+\"['\"+element.split(':')[1]+\"','\"+str(y)+'_'+str(x)+\"'],\"\n",
    "    for el in diction[key]:\n",
    "        for val in el.split('#'):\n",
    "            text=text+\"'\"+val.split(':')[0]+\"':'\"+val.split(':')[1]+\"'\"+','\n",
    "            #print(text)\n",
    "        for el2 in dictionlev2[key+sepcol+el]:\n",
    "            for val2 in el2.split('#'):\n",
    "                text=text+\"'\"+val2.split(':')[0]+\"':'\"+val2.split(':')[1]+\"'\"+','\n",
    "                cpt_special_col=cpt_special_col+1\n",
    "    print(cpt_special_col)\n",
    "    print(text)\n",
    "    #print(text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "444"
      ]
     },
     "execution_count": 63,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.4.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
