CREATE TABLE array_type (
array_type_id integer NULL,
array_type_name VARCHAR (100) NULL,
common_name VARCHAR (500) NOT NULL,
platform_name VARCHAR (100) NOT NULL,
platform_type VARCHAR (50) NOT NULL,
surface_type VARCHAR (30) NOT NULL,
coating_type VARCHAR (30) NOT NULL,
nature_attachment VARCHAR (30) NOT NULL,
strand VARCHAR (20) NOT NULL,
slide_dim VARCHAR (10) NOT NULL,
metablock_nb smallint NOT NULL,
metacol_nb smallint NOT NULL,
metarow_nb smallint NOT NULL,
spot_nb integer NOT NULL,
col_nb smallint NOT NULL,
row_nb smallint NOT NULL,
avg_spot_dim smallint NOT NULL,
spotting_plate smallint NOT NULL,
arrayer_in_file VARCHAR (100) NOT NULL,
arrayer_out_file VARCHAR (100) NOT NULL,
geo_platform VARCHAR (20) NOT NULL,
note VARCHAR (4000) NOT NULL,
user_id integer NOT NULL,
submission_date date NOT NULL,
distribution VARCHAR (20) NOT NULL,
manufacture_website VARCHAR (100) NOT NULL,
array_probe_file VARCHAR (100) NOT NULL );
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (16,"poplar","Poplar","Affymetrix","in situ oligonucleotide","None","unknown","None","None","None","None","None","None",61413,"None","None","None","None","None","Poplar.cdf","GPL4359","P. Trichocarpa and P. TremulaxTremuloides ",7,"2008-05-20","commercial","http://www.affymetrix.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (22,"Human GenomeU133A 2.0","Human","Affymetrix","in situ oligonucleotide","None","None","None","None","None","None","None","None",22000,"None","None","None","None","None","HG-U133A_2.cdf","GPL571","Analyzes the expression level of 18,400 transcripts and variants, including 14,500 well-characterized human genesComprised of more than 22,000 probe sets and 500,000 distinct oligonucleotide features.",7,"2010-05-20","None","None","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (14,"Gallus gallus","Gallus","Affymetrix","in situ oligonucleotide","None","unknown","None","None","None","None","None","None",38392,"None","None","None","None","None","Chicken.cdf","GPL3213","Number of gallus gallus transcripts: 32773; Number of avian transcripts: 684 ; hybridization controls, poly-A controls and housekeeping genes",7,"2007-04-23","commercial","http://www.affymetrix.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (18,"rose","Rose","Affymetrix","in situ oligonucleotide","None","unknown","None","None","None","None","None","None",4869,"None","None","None","None","None","Rosa_1a520434F.cdf","GPL6746","custom arrays with classical affymetrix controls",7,"2008-06-24","commercial","http://www.affymetrix.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (11,"maize","maize","Affymetrix","in situ oligonucleotide","None","unknown","None","None","None","None","None","None",17552,"None","None","None","None","None","Maize.cdf","GPL4032","Number of probe sets Zea mays: 17555; number of transcripts ~14850; hybridization controls: BioB, BioC, BioD from E. coli and cre from P1 Bacteriophage; poly-A controls: dap, Lys, phe, thr, trp from B. subtilis; housekeeping controls genes: GAPDH, Actin, Cyclophilin1, Ubiquitin and 18S rRNA.",7,"2007-03-12","commercial","http://www.affymetrix.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (12,"rice","rice","Affymetrix","in situ oligonucleotide","None","unknown","None","None","None","None","None","None",55515,"None","None","None","None","None","Rice.cdf","GPL2025","Number of indica transcrits:1260; Number of japonica transcrits:48564; hybridization controls: BioB, BioC, BioD from E. coli and cre from P1 Bacteriophage; poly-A controls: dap, Lys, phe, thr, trp from B. subtilis; housekeeping controls genes: GAPDH, Actin, Cyclophilin1, Ubiquitin and 18S rRNA.",7,"2007-03-13","commercial","http://www.affymetrix.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (15,"Vitis","Vitis","Affymetrix","in situ oligonucleotide","None","unknown","None","None","None","None","None","None",16000,"None","None","None","None","None","Vitis.cdf","GPL1320","16 probe pair per probe set interogate 14000 transcrits of V. vinifera and 1700 transcrits of other vitis species. array format:100",7,"2007-12-11","commercial","http://www.affymetrix.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (23,"Cotton","Cotton","Affymetrix","in situ oligonucleotide","None","None","None","None","None","None","None","None",23977,"None","None","None","None","None","Cotton.cdf","GPL8672","Probe sets: 23977Transcripts: 21854Hybridization controls: bioB, bioC, bioD, crePolyA controls: dap, lys, phe, thr, trpHousekeeping controls genes: sucrose synthase, actin, and polyubiquitin for Gossypiumhirsutum and Gossypium raimondiiHybridization volume130ÂµL",216,"2010-06-03","None","None","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (13,"Medicago","Medicago","Affymetrix","in situ oligonucleotide","None","None","None","None","None","None","None","None",61200,"None","None","None","None","None","Medicago.cdf","GPL4652","this arrays contains 48116 transcripts of M. truncatula; 1850 transcripts of M. sativa and 8226 transcripts of S. meliloti. Hybridization controls: bioB, bioC and bioD from E.coli and cre from P1bacteriophage.",8,"2014-02-03","commercial","http://www.affymetrix.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (21,"SUNFLOWER","Sunflower","Affymetrix","in situ oligonucleotide","None","None","None","None","None","None","None","None",94691,"None","None","None","None","None","SYNG010s520477.cdf","GPL10568","Number of probe sets  SUNFLOWER: 94691; hybridization controls: BioB, BioC, BioD from E. coli and cre from P1 Bacteriophage; poly-A controls: dap, Lys, phe, thr, trp from B. subtilis; housekeeping controls genes: GAPDH, Actin, Cyclophilin1, Ubiquitin and 18S rRNA.",7,"2010-06-23","None","None","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (37,"121115_Psativum_SB_expr","Pea","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",12,"None","None",270000,"None","None","None","None","None","121115_Psativum_SB_expr.ndf","GPL17462","- 2 oligo/target sequence except for the 323 miRNA which have 1 oligo/sequence - Sens: Reverse (+ Reverse and Forward sens for miRNA)-all probes are triplicate-40795 Unigen including 18 control",7,"2013-07-17","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (38,"Medgene -1.0stv1","Medicago","Affymetrix","in situ oligonucleotide","None","None","None","None","None","None","None","None",654305,"None","None","None","None","None","MedGene-1_0-st.1.0_pgfclf.CDFRME","GPL18240","Measure expression, across the entire gene, with higher resolution and accuracy than with classical 3-biased microarray solutions.654,305 probes and22 probes/gene andtargeted 38,144 genes",7,"2014-02-04","None","None","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (31,"Brachypodium","Brachypodium","Affymetrix","in situ oligonucleotide","None","None","None","None","None","None","None","None",53263,"None","None","None","None","None","BradiAR1b520742.cdf","GPL15863","Whole-Transcript Expression Analysis. AffymetrixÂ® Brachypodium Gene 1.0 ST Array.",7,"2012-07-30","None","None","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (35,"090918_Vitus_exp_HX12","Vitis","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",12,"None","None",145868,"None","None","None","None","None","090918_Vitus_exp.ndf","GPL17460","145868 probes / chamber.Several different probes / gene.",7,"2013-07-17","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (36,"100718_Populus_135K","Poplar","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",12,"None","None",135000,"None","None","None","None","None","100718_Populus_135K_EXP.ndf","GPL17461 ","146170 probes corresponding to 49475 UniGen",7,"2013-07-17","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (34,"110121_Taestivum-HD12","Wheat","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",12,"None","None",145804,"None","None","None","None","None","110121_Taestivum_EP_exp.ndf","GPL17441","145804 oligos/chamber; 3 probes/Unigen.",7,"2013-07-17","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (20,"HD2.3X","Arabidopsis","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",3,"None","None",711320,"None","None","None","None","None","090507_URGV_AT7_WG_CGH.ndf","GPL11005","Tiling array HD2.3X. ChIP-chip design of Arabidopsis 7.0 whole genome, with 175 bp spacing.  This design is a refinement of AT7v2 with 3 chambers of 711320 spots on a single slide.",133,"2010-10-01","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (10,"wheat","Wheat","Affymetrix","in situ oligonucleotide","None","unknown","None","None","None","None","None","None",61290,"None","None","None","None","None","wheat.cdf","GPL3802","note",7,"2007-03-12","commercial","http://www.affymetrix.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (2,"ATH1","Arabidopsis","Affymetrix","in situ oligonucleotide","None","unknown","None","None","None","None","None","None",251078,"None","None","None","None","None","Arab.cdf","GPL198","note",7,"2007-03-12","commercial","http://www.affymetrix.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (9,"CAGE_Microgrid","Arabidopsis","CAGE","RT-PCR","glass","miror","covalent","double strand","75 x 25 mm",48,4,12,1165824,22,23,100,384,"None","CATMA_23K_4.e.gal","None","note",10,"2006-12-13","non-commercial","http://urgv.evry.inra.fr/CATdb","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (24,"AT7v1.1","Arabidopsis","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",1,"None","None",389648,"None","None","None","None","None","071102_URGV_AT7_WG_ChIP01.ndf","GPL10919","Tiling array NimbleGen AT7v1.1. ChIP-chip design of Arabidopsis 7.0 whole genome, with 175 bp spacing.Array with one chamber but 2 arrays used for one sample (AT7v1.1 and v1.2).",8,"2010-09-20","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (40,"CATMAv7_4PLEX","Arabidopsis","Agilent","in situ oligonucleotide","glass","aminosilane","None","single strand","None",4,"None","None",149916,"None","None","None","None","None","047069_D_20130204_ED.gal","GPL18904","73196 probes: 34836 probes no_gene single + 38360 probes gene in triplicate.",8,"2015-11-10","custom-commercial","http://www.home.agilent.com/","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (32,"121018_Medicago_trun","121018_Medicago_trun","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",12,"None","None",249087,"None","None","None","None","None","121018_Mtrun_SB_expr.ndf","GPL17428","Design:-Sens: Reverse -Triplicate-1 oligo/target sequence-       83029 x3 probes",335,"2016-08-01","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (33,"HD2.3X_REV","Arabidopsis","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",3,"None","None",711320,"None","None","None","None","None","090715_URGV_AT7_WG_REV_CGH.ndf","GPL17057","Tiling array HD2.3X. ChIP-chip design of Arabidopsis 7.0 whole genome, with 175 bp spacing.  This design is a refinement of AT7v2 with 3 chambers of 711320 spots on a single slide and all oligo are in REVERSE design.",7,"2013-04-24","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (17,"SAP","Arabidopsis","SAP","RT-PCR","glass","aminosilane","covalent","double strand","75 x 25 mm",192,8,24,5308416,12,12,90,1536,"None","SAP.gps","None","A.thaliana genes were predicted with Eugene software (Foissac S. et al., Nucleic Acids Res., 2003). Each promoter amplicon 3  end was delimited by the nucleotide preceeding the translation initiation codon in the corresponding gene model. The amplified fragments also include the corresponding 5  untranslated region. The amplicon 5  end was located at the boundary between the targeted intergenic region and the adjacent upstream open reading frame (ORF), or ca. 2.0 kb upstream of the translation initiation codon.Two rounds of PCR were performed and PCR products were finally purified and dissolved in 50% DMSO in TE at concentrations ranging between 50-200ng/ul and re-arrayed in 1536 microtiter plates. The pins are washed in 0.1% SDS at 50 degre Celsius then three times in bi-distilled water. Spotter settings are 45-55% humidity, and a pitch = 180 um. The room temperature is set between 19 and 21 Â°s Celsius and below 40 % humidity. The microtiter plates are placed in the biobank. The four trays are cleaned and 100-120 slides placed on them (30 slides per tray). The spotting lasts 25 hours. The slides are stored in the dark in a dry-cabinet for 3 days and UV-cross-linked at 300 mJ. After crosslinking, the slides are stored in the dark in a dry-cabinet.This array contains 24576 Gene Specific Tags (GST).",1,"2008-05-29","non-commercial","http://urgv.evry.inra.fr/CATdb","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (5,"CHROMO4_1","Arabidopsis","Chromochip","RT-PCR","glass","aminosilane","ionic","double strand","75 x 25 mm",3,8,8,27648,12,12,100,1536,"Bio_bank_CHROMO4_1.txt","CHROMO4_1.gal","GPL10892","The entire TIGR4 Arabidopsis thaliana ec. Col chromosome 4 assembly was used to design a genomic tiling microarray. This ~18.6 Mb sequence represented on the microarray starts downstream of the nucleolar organizing region (NOR) comprising the 5Â terminus of the chromosome. Notably, it contains several megabases of pericentromeric heterochromatin. In the microarray design, annotation was intentionally ignored, thus ensuring that repeated and unique sequences were equally represented. PCR primer pairs were selected using the Primer3 software. Probe selection was first accomplished by BLASTN analysis of sequential 100-bp windows of sequence along chromosome 4 against the whole genome to discriminate between unique and repeated DNA. Whenever a window gave at least one hit with 85% identity elsewhere in the genome, that window was marked as repeated, or unique otherwise. This information was then used to define repeated and unique segments of at least 300bp along the chromosome, as follows. Repeat segments were composed of successive repeat windows, possibly interrupted by series of contiguous unique windows spanning less than 300bp in each case. Conversely, unique segments were composed of successive unique windows, never interrupted by more than 300bp of contiguous repeat windows. Series of alternate unique and repeated windows were treated as repeat segments. Within each repeated or unique segment, PCR primer pairs were then selected using the Primer3 software and a routine that ensured maximum coverage of the segment and limited overlap between successive probes. Briefly, primer pairs were designed from both ends of the segment, considering sequential 1.2 kb windows, amplicon sizes ranging from 850 bp to 1.2 kb (optimum 1 kb) and primers (19-23 nt each, optimum 21 nt, 40-60% GC) with similar melting temperatures in the 57Â°CÂ­63Â°C range (optimum 60Â°C). Each new 1.2 kb window was positioned 50 bp within the end of the preceding amplicon. Successive amplicons could therefore overlap by up to 50 bp or be separated by up to 250 bp. When no primer pair was found in a given window, window size was increased to 1.4 kb, and if still unsuccessful, amplicons were finally designed using the forward and reverse primers of the upstream and downstream amplicons, respectively. When the last (central) window of a segment was less than 1.2kb but more than 300bp, primer pair selection was carried out by considering the actual size of that window plus 50bp on either side. Using this approach, 21,405 amplicons were designed with an average size of ~950bp. An additional 356 amplicons were designed that cover 36 genes of interest and neighboring sequences located on the other 4 chromosomes. Most amplicons were produced using BAC clones as templates, and all amplicons were verified by gel electropheresis.",8,"2010-09-08","non-commercial","http://urgv.evry.inra.fr/CATdb","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (25,"AT7v1.2","Arabidopsis","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",1,"None","None",389648,"None","None","None","None","None","071102_URGV_AT7_WG_ChIP02.ndf","GPL10920","Tiling array NimbleGen AT7v1.2. ChIP-chip design of Arabidopsis 7.0 whole genome, with 175 bp spacing.Array with one chamber but 2 arrays used for one sample (AT7v1.1 and v1.2).",8,"2010-09-20","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (28,"CATMAv6-HD12","Arabidopsis","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",12,"None","None",73228,"None","None","None","None","None","100701_Athal_CATMA6_SB_exp.ndf","GPL15719","The CATMAv6-HD12 array contains 73228 oligos in forward and reverse strands tagging TAIRv8 gene prediction (31227 oligos including 476 Chloro/mito probes) and some specific EUGENE genes (1289). In addition, we including oligos corresponding to repeat_element (5352), miRNA (658) and 36 human controls oligos.",8,"2012-06-20","custom-commercial","http://www.nimblegen.com","CATMAv6-HD12_07122011.txt");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (3,"CATMA_2","Arabidopsis","CATMA","RT-PCR","glass","aminosilane","covalent","double strand","75 x 25 mm",3,8,8,27648,12,12,100,1536,"Biol_bank_CATMA2.txt","CATMA2.gal","GPL5336","A.thaliana genes were predicted with Eugene software (Foissac S. et al., Nucleic Acids Res., 2003) and gene-specific sequence tags were automatically designed (Thareau et al. Bioinformatics, 2003 Nov 22). Two rounds of PCR were performed and PCR products were finally purified and dissolved in 50% DMSO in TE at concentrations ranging between 50-200ng/ul and re-arrayed in 1536 microtiter plates. The pins are washed in 0.1% SDS at 50 degre Celsius then three times in bi-distilled water. Spotter settings are 45-55% humidity, and a pitch = 180 um. The room temperature is set between 19 and 21 Â°s Celsius and below 40 % humidity. The microtiter plates are placed in the biobank. The four trays are cleaned and 100-120 slides placed on them (30 slides per tray). The spotting lasts 25 hours. The slides are stored in the dark in a dry-cabinet for 3 days and UV-cross-linked at 300 mJ. After crosslinking, the slides are stored in the dark in a dry-cabinet.This array contains 24576 Gene Specific Tags (GST).",5,"2007-06-01","non-commercial","http://urgv.evry.inra.fr/CATdb","CATMA_2_07122011.txt");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (1,"CATMA_2.1","Arabidopsis","CATMA","RT-PCR","glass","aminosilane","covalent","double strand","75 x 25 mm",3,8,8,27648,12,12,100,1536,"Biol_bank_CATMA2_1.txt","CATMA-2_1.gal","GPL5337","A.thaliana genes were predicted with Eugene software (Foissac S. et al., Nucleic Acids Res., 2003) and gene-specific sequence tags were automatically designed (Thareau et al. Bioinformatics, 2003 Nov 22). Two rounds of PCR were performed and PCR products were finally purified and dissolved in 50% DMSO in TE at concentrations ranging between 50-200ng/ul and re-arrayed in 1536 microtiter plates. The pins are washed in 0.1% SDS at 50 degre Celsius then three times in bi-distilled water. Spotter settings are 45-55% humidity, and a pitch = 180 um. The room temperature is set between 19 and 21 Â°s Celsius and below 40 % humidity. The microtiter plates are placed in the biobank. The four trays are cleaned and 100-120 slides placed on them (30 slides per tray). The spotting lasts 25 hours. The slides are stored in the dark in a dry-cabinet for 3 days and UV-cross-linked at 300 mJ. After crosslinking, the slides are stored in the dark in a dry-cabinet.This array design contains : - 24576 Gene Specific Tags (GST)- 615 Chloroplastic and Mitochondrial Tilling array probes",5,"2007-06-01","non-commercial","http://urgv.evry.inra.fr/CATdb","CATMA_2.1_07122011.txt");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (27,"AT7v2.2","Arabidopsis","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",1,"None","None",390304,"None","None","None","None","None","080222_URGV_AT7_WG_ChIP02.ndf","GPL10918","Tiling array AT7v2.2. ChIP-chip design of Arabidopsis 7.0 whole genome, with 175 bp spacing.  This design is a refinement of AT7v1  by replacing the existing chrM and chrC probes with a straight tiling of those chromosomes.  Probes split between the 2 designs, with every-other probe in each array.",133,"2010-09-15","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (4,"CATMA_2.2","Arabidopsis","CATMA","RT-PCR","glass","aminosilane","covalent","double strand","75 x 25 mm",3,8,8,27648,12,12,100,1536,"Biol_bank_CATMA2_2.txt","CATMA2_2.gal","GPL4346","A.thaliana genes were predicted with Eugene software (Foissac S. et al., Nucleic Acids Res., 2003) and gene-specific sequence tags were automatically designed (Thareau et al. Bioinformatics, 2003 Nov 22). Two rounds of PCR were performed and PCR products were finally purified and dissolved in 50% DMSO in TE at concentrations ranging between 50-200ng/ul and re-arrayed in 1536 microtiter plates. The pins are washed in 0.1% SDS at 50 degre Celsius then three times in bi-distilled water. Spotter settings are 45-55% humidity, and a pitch = 180 um. The room temperature is set between 19 and 21 Â°s Celsius and below 40 % humidity. The microtiter plates are placed in the biobank. The four trays are cleaned and 100-120 slides placed on them (30 slides per tray). The spotting lasts 25 hours. The slides are stored in the dark in a dry-cabinet for 3 days and UV-cross-linked at 300 mJ. After crosslinking, the slides are stored in the dark in a dry-cabinet. This array design contains: - 24576 Gene Specific Tags (GST)- 615 Chloroplastic and Mitochondrial Tilling array probes- 44 Non Protein Coding (NPC)",5,"2007-02-09","non-commercial","http://urgv.evry.inra.fr/CATdb","CATMA_2.2_07122011.txt");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (26,"AT7v2.1","Arabidopsis","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",1,"None","None",390304,"None","None","None","None","None","080222_URGV_AT7_WG_ChIP01.ndf","GPL10911","Tiling array AT7v2.1. ChIP-chip design of Arabidopsis 7.0 whole genome, with 175 bp spacing.  This design is a refinement of AT7v1  by replacing the existing chrM and chrC probes with a straight tiling of those chromosomes.  Probes split between the 2 designs, with every-other probe in each array.",133,"2010-09-15","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (8,"CAGE_Lucidea","Arabidopsis","CAGE","RT-PCR","glass","miror","covalent","double strand","75 x 25 mm",24,2,12,568512,47,21,120,384,"None","Cage_23k3_avec_noms.gal","None","note",10,"2006-12-13","non-commercial","http://urgv.evry.inra.fr/CATdb","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (6,"CHROMO4_2","Arabidopsis","Chromochip","RT-PCR","glass","aminosilane","ionic","double strand","75 x 25 mm",3,8,8,27648,12,12,100,1536,"Bio_bank_CHROMO4_2.txt","CHROMO4_2.gal","GPL10772","The entire TIGR4 Arabidopsis thaliana ec. Col chromosome 4 assembly was used to design a genomic tiling microarray. This ~18.6 Mb sequence represented on the microarray starts downstream of the nucleolar organizing region (NOR) comprising the 5Â terminus of the chromosome. Notably, it contains several megabases of pericentromeric heterochromatin. In the microarray design, annotation was intentionally ignored, thus ensuring that repeated and unique sequences were equally represented. PCR primer pairs were selected using the Primer3 software. Probe selection was first accomplished by BLASTN analysis of sequential 100-bp windows of sequence along chromosome 4 against the whole genome to discriminate between unique and repeated DNA. Whenever a window gave at least one hit with 85% identity elsewhere in the genome, that window was marked as repeated, or unique otherwise. This information was then used to define repeated and unique segments of at least 300bp along the chromosome, as follows. Repeat segments were composed of successive repeat windows, possibly interrupted by series of contiguous unique windows spanning less than 300bp in each case. Conversely, unique segments were composed of successive unique windows, never interrupted by more than 300bp of contiguous repeat windows. Series of alternate unique and repeated windows were treated as repeat segments. Within each repeated or unique segment, PCR primer pairs were then selected using the Primer3 software and a routine that ensured maximum coverage of the segment and limited overlap between successive probes. Briefly, primer pairs were designed from both ends of the segment, considering sequential 1.2 kb windows, amplicon sizes ranging from 850 bp to 1.2 kb (optimum 1 kb) and primers (19-23 nt each, optimum 21 nt, 40-60% GC) with similar melting temperatures in the 57Â°CÂ­63Â°C range (optimum 60Â°C). Each new 1.2 kb window was positioned 50 bp within the end of the preceding amplicon. Successive amplicons could therefore overlap by up to 50 bp or be separated by up to 250 bp. When no primer pair was found in a given window, window size was increased to 1.4 kb, and if still unsuccessful, amplicons were finally designed using the forward and reverse primers of the upstream and downstream amplicons, respectively. When the last (central) window of a segment was less than 1.2kb but more than 300bp, primer pair selection was carried out by considering the actual size of that window plus 50bp on either side. Using this approach, 21,405 amplicons were designed with an average size of ~950bp. An additional 356 amplicons were designed that cover 36 genes of interest and neighboring sequences located on the other 4 chromosomes. Most amplicons were produced using BAC clones as templates, and all amplicons were verified by gel electropheresis.",8,"2010-08-10","non-commercial","http://urgv.evry.inra.fr/CATdb","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (7,"CATMA_2.3","Arabidopsis","CATMA","RT-PCR","glass","aminosilane","covalent","double strand","75 x 25 mm",3,8,8,27648,12,12,100,1536,"Biol_bank_CATMA2_3.txt","CATMA2_3.gal","GPL6403","A.thaliana genes were predicted with Eugene software (Foissac S. et al., Nucleic Acids Res., 2003) and gene-specific sequence tags were automatically designed (Thareau et al. Bioinformatics, 2003 Nov 22). Two rounds of PCR were performed and PCR products were finally purified and dissolved in 50% DMSO in TE at concentrations ranging between 50-200ng/ul and re-arrayed in 1536 microtiter plates. The pins are washed in 0.1% SDS at 50 degre Celsius then three times in bi-distilled water. Spotter settings are 45-55% humidity, and a pitch = 180 um. The room temperature is set between 19 and 21 Â°s Celsius and below 40 % humidity. The microtiter plates are placed in the biobank. The four trays are cleaned and 100-120 slides placed on them (30 slides per tray). The spotting lasts 25 hours. The slides are stored in the dark in a dry-cabinet for 3 days and UV-cross-linked at 300 mJ. After crosslinking, the slides are stored in the dark in a dry-cabinet.This array design contains: - 24576 Gene Specific Tags (GST) - 44 Non Protein Coding (NPC)- 1217 microRNA precursors.",1,"2008-01-23","non-commercial","http://urgv.evry.inra.fr/CATdb","CATMA_2.3_07122011.txt");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (19,"CATMA_5","Arabidopsis","CATMA","RT-PCR","glass","aminosilane","covalent","double strand","75 x 25 mm",3,8,8,37632,14,14,100,1536,"None","CATMA5.gal","GPL9553","A.thaliana genes were predicted with Eugene software (Foissac S. et al., Nucleic Acids Res., 2003) and gene-specific sequence tags were automatically designed (Thareau et al. Bioinformatics, 2003 Nov 22). Two rounds of PCR were performed and PCR products were finally purified and dissolved in 50% DMSO in TE at concentrations ranging between 50-200ng/ul and re-arrayed in 1536 microtiter plates. The pins are washed in 0.1% SDS at 50 degre Celsius then three times in bi-distilled water. Spotter settings are 45-55% humidity, and a pitch = 180 um. The room temperature is set between 19 and 21 Â°s Celsius and below 40 % humidity. The microtiter plates are placed in the biobank. The four trays are cleaned and 100-120 slides placed on them (30 slides per tray). The spotting lasts 25 hours. The slides are stored in the dark in a dry-cabinet for 3 days and UV-cross-linked at 300 mJ. After crosslinking, the slides are stored in the dark in a dry-cabinet.This array design contains: - 32626 Gene Specific Tags (GST) - 44 Non Protein Coding (NPC)- 1263 microRNA precursors.",5,"2009-11-10","non-commercial","http://urgv.evry.inra.fr/CATdb","CATMA_5_07122011.txt");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (30,"CATMAv6.2-HD12","Arabidopsis","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",12,"None","None",219684,"None","None","None","None","None","120228_Athal_CATMA62_AA_exp.ndf","GPL15719","73228 primers in triplicate corresponding to 38383 with an associated annotation (30834 probes corresponding to TAIRv8 annotation (including 476 probes of mitochondrial and chloroplast genes) + 1289 probes corresponding to EUGENE software predictions. Moreover, we included 5352 probes corresponding to reapeat elements, 658 probes for miRNA/MIR, 342 probes for other RNAs (rRNA,tRNA, snRNA, soRNA) and finally 36 controls) + 34845 primers which is the reverse of previous one.",8,"2013-04-08","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (29,"CATMAv6.1-HD12","Arabidopsis","NimbleGen","in situ oligonucleotide","glass","aminosilane","None","single strand","None",12,"None","None",135000,"None","None","None","None","None","111220_Athal_SB_exp.ndf","GPL15719","Each chamber contains 38392 primers : 30834 primers coming from TAIR8 annotation genes (including 476 mito/chloro genes primers) + 1289 primers coming from EUGENE software specific prediction. Moreover, we have put 5352 primers coming from repeat-element, 658 primers miRNA/MIR, 342 otherRNA (rRNA,tRNA, snRNA, soRNA) and 36 controls. Each primer is design in Forward sense and in triplicate in one chamber.",8,"2014-08-19","custom-commercial","http://www.nimblegen.com","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (39,"4PLEX_TOMATO","4PLEX_TOMATO","Agilent","in situ oligonucleotide","glass","aminosilane","None","single strand","None",4,"None","None",67934,"None","None","None","None","None","050012_D_20130607_corrED.gal","GPL20224","67826 Probes designed in genome and duplicated: 33913 forward and 33913 reverse (1 oligo by sequence) + 108  probes corresponding to control in triplicate. ",8,"2015-05-21","custom-commercial","http://www.home.agilent.com/","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (43,"4PLEX_BRACHY","4PLEX_BRACHY","Agilent","in situ oligonucleotide","glass","aminosilane","None","single strand","None",4,"None","None",126604,"None","None","None","None","None","066852_D_20140513_19-10-16_9h13.gal","None","- Strand:Forward- 2 oligos/target sequence cDNA, with technical duplicate- Controls and spikeNumber of sequences/target: 31651 forward probes (x2 because 2 probes/target) in technical duplicate.",8,"2016-11-03","custom-commercial","http://www.home.agilent.com/","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (41,"4PLEX_PHYSCO","4PLEX_PHYSCO","Agilent","in situ oligonucleotide","glass","aminosilane","None","single strand","None",4,"None","None",160574,"None","None","None","None","None","066853_D_20140513_ED.gal","GPL21755","32104 probes Forward en Triplicat technique et 32104 probes Reverse en duplicat technique (1 oligo by target sequence) + 54 probes corresponding to control in triplicate.",335,"2016-04-22","custom-commercial","http://www.home.agilent.com/","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (42,"4PLEX_MAIZE","4PLEX_MAIZE","Agilent","in situ oligonucleotide","glass","aminosilane","None","single strand","None",4,"None","None",157348,"None","None","None","None","None","064491_D_20140214_ED.gal","GPL22405","78416 forward for cDNA in duplicate + 154 probes miRNA in triplicate + 18 controls in triplicate, so 157348 probes in total",335,"2016-10-10","custom-commercial","http://www.home.agilent.com/","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (44,"4PLEX_BARLEY","4PLEX_BARLEY","Agilent","in situ oligonucleotide","glass","aminosilane","None","single strand","None",4,"None","None",46637,"None","None","None","None","None","021623_D_20170601.gal","GPL14927","Total Number of Features:45220 Number of Agilent Controls:1417 ",335,"2018-07-16","custom-commercial","http://www.home.agilent.com/","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (45,"4plex_Brassica","4plex_Brassica","Agilent","in situ oligonucleotide","glass","aminosilane","None","single strand","None",4,"None","None",67417,"None","None","None","None","None","076686_D_20150708.gal","GPL25499","Forward and Reverse, 1 oligo/target sequence unigene (forward and reverse): 3 technicals replicats for forward and 2 technicals replicats for reverse.Number of sequences/target: 33695 probes sens in triplicat + 33695 probes antisens in duplicat + 27 CATMA controls in duplicat.",335,"2018-08-29","custom-commercial","http://www.home.agilent.com/","None");
INSERT INTO array_type (array_type_id,array_type_name,common_name,platform_name,platform_type,surface_type,coating_type,nature_attachment,strand,slide_dim,metablock_nb,metacol_nb,metarow_nb,spot_nb,col_nb,row_nb,avg_spot_dim,spotting_plate,arrayer_in_file,arrayer_out_file,geo_platform,note,user_id,submission_date,distribution,manufacture_website,array_probe_file) VALUES (46,"4PLEX_PEA","4PLEX_PEA","Agilent","in situ oligonucleotide","glass","aminosilane","None","single strand","None",4,"None","None",81624,"None","None","None","None","None","064021_D_20140128_corrED.gal","GPL25688","Number of sequences/target:40454 probes x2 + 36 controls + 18 CATMA Forward controls x3 + 323 miRNA x2 (Forward and Reverse).",335,"2018-10-18","custom-commercial","http://www.home.agilent.com/","None");
