CREATE TABLE experiment (
experiment_id integer NULL,
experiment_idname VARCHAR (50) NULL,
experiment_name VARCHAR (100) NULL,
experiment_type VARCHAR (100) NULL,
experiment_factors VARCHAR (100) NULL,
note VARCHAR (4000) NOT NULL,
array_type VARCHAR (20) NULL,
protocol_id integer NOT NULL,
project_id integer NOT NULL,
user_id integer NOT NULL,
submission_date date NOT NULL,
repository_db VARCHAR (30) NOT NULL,
repository_access VARCHAR (20) NOT NULL,
analysis_type VARCHAR (20) NULL );
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (554,"P378E554","12PLEX_PEA_2013_03_ROOTS","treated vs untreated comparison","drought,time,tissue type","Pea plants (genotype Cameor) were subjected to a moderate water stress at the growth cycle for a period of 15 days and were rewatered during 7 days. Samples were collected from Well Watered (WW) plants  and from Water-Stressed (WS)  at the end of the drought period (point S) and after 7 days of rewatering (point R). Samples named roots consisted of the roots without nodules. Samples named nod consisted of nodules detached from the roots. Each sample consited of a mix of 18 individual plants and 4 repetitions per condition were carried out.","NimbleGen",0,378,238,"2014-07-29","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (553,"P377E553","Identification of the miss-regulated genes in mkk3 under ABA treatment.","gene knock out,treated vs untreated comparison"," compound","Voir entente préalable du projet.","Agilent",0,377,335,"2014-06-13","GEO","GSE59155","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (540,"P366E540","Transcriptome and translatome of Arabidopsis WT seeds according to dormancy","time course","time","At harvest seeds are dormant. They stay dormant if they are stored at -20°C  (D) and become non-dormant (ND) if they are stored 3 weeks at +20°C. Arabidopsis dormant seeds do not germinate at 25°C in darkness while non-dormant seeds do. Total RNA and polysomal RNA (polysomal fractions  purified on sucrose gradients) were extracted from imbibed seeds for 16h or 24h at 25°C in darkness (3 biological replicates).Transcriptome and translatome are compared for  Dormant vs Non-Dormant for 16h and 24 imbibition. In silico comparison will allow to compare transcriptome and translatome for each point and type of seeds and to compare the time points (16 vs 24h) for each type of sample.","NimbleGen",10311,366,0,"2014-01-15","GEO","GSE61809","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (0,"P0E0","stem developement","time course","developmental stage","transcription frofiling of stem developement in Arabidopsis thaliana.time course with 3 points: 2cm, 10 cm and 25cm (maturation)","CATMA",290,0,5,"2006-12-19","GEO","GSE8094","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (341,"P224E341","Infection with Salmonella or Pseudomonas or E. coli.","treated vs untreated comparison","inoculation","Arabidopsis thalians Col-0 seedlings were infected for 2 hours with ; a) Salmonella typhimurium strain 14028s, b) Pseudomonas syringea DC3000 or c) Escherichia coli DH5A","CATMA",10134,224,14,"2010-03-19","GEO","GSE20996","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (5,"P0E5","CD","normal vs disease comparison","genotype","1 biological repeatCD vs Sauvage","CATMA",283,0,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (538,"P365E538","Acclim","dose response","temperature","Plants from 163AV have been grown in greenhouse during 2 weeks, then transferred in climate chamber : one half at 4°C and the others at 0°C. Plants stayed 8 days in these conditions (12h light, 4°C or 0°C). Samples are collected at the end of this period.","NimbleGen",10310,365,0,"2013-12-23","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (541,"P367E541","mRNA and small RNA sequencing","normal vs transgenic comparaison","developmental stage,genotype,tissue type","mRNA seq and sRNA seq on wild-type Col, rtl2 and dcl234 mutants, and 35S-RTL1, 35S-RTL2 and 35S-DCL1-R overexpressers.","Illumina",0,367,238,"2014-01-17","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (542,"P368E542","RNAdapt","dose response,ecotype comparison,time course","dose,time,tissue type","Plants (Columbia er 105, or Landsberg erecta) were grown one week on media with Phosphate (+P) and then transfered on  media with (+P) or without (-P = P starvation) for 15 minutes, 1h or 2h. Root apices were harvested and total RNA extracted for RNA sequencing.","Illumina",0,368,0,"2014-01-29","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (558,"P381E558","4 mutants","gene knock out,genotype comparaison","genotype,light","Plants Col-0, mips1, sup62 and sup62 mips1 were grown for 3 weeks in Short-Day condition and subsequently transfered in Long-Day condition. Rosette leaves were sampled after 4 days in Long-Day condition.","NimbleGen",10313,381,0,"2014-09-05","GEO","GSE67010","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (559,"P382E559","ABAseed","gene knock out,genomic comparison","genotype","Transcriptome comparison of  developing seeds harvested at 14 DAP from two ABA-deficient mutants (nced2 nced5 nced9 and nced2 nced5 nced6 nced9)  and wild type (Columbia-0)","NimbleGen",10314,382,0,"2014-09-10","GEO","GSE68088","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (103,"P69E103","Modification of response to salicylic acid by wortmannin treatment","dose response,treated vs untreated comparison","compound,dose","Cell suspension culture of Arabidopsis thaliana (ecotype Columbia) was treated by 250 uM salicylic acid and by two different concentrations of wortmannin (1 uM and 30 uM) for 4 hours.","CATMA",361,69,5,"2007-06-01","GEO","GSE7495","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (549,"P374E549","12PLEX_PEA_2013_02_F","treated vs untreated comparison","developmental stage,drought,organism part,tissue type","Pea plants (genotype Cameor) were subjected to a moderate water stress at the beggining of the seed filling period (12 Days After Pollination) of the second flowering node for a period of 8 days. Samples were collected from Well Watered (WW) plants at the beginning of the stress imposition (point A, T=0), and from Water-Stressed (WS) and WW control plants at the end of the drought period (point B, T=+8). Samples named SEED consisted of  seeds from the pod of the second flowering node (seed-WW-A, seed-WW-B and Seed-WS-B). Samples named LEAF consisted of the leaves and stem sections from the two vegetative nodes  below the first flowering node (leaf-WW-A, Leaf-WW-B and Leaf-WS-B). Each sample consited of a pool of 3 individual plants and 4 repetitions per condition were carried out.","NimbleGen",0,374,238,"2014-05-27","GEO","GSE93630","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (525,"P352E525","12PLEX-WHEAT_2012-05","genotype comparaison,treated vs untreated comparison","developmental stage,genotype,temperature","The effect of the temperature on the expression of genes along with  the effect of genotype","NimbleGen",10297,352,174,"2013-04-29","GEO","GSE100642","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (555,"P378E555","12PLEX_PEA_2013_03_NODULES","treated vs untreated comparison","drought,time,tissue type","Pea plants (genotype Cameor) were subjected to a moderate water stress at the growth cycle for a period of 15 days and were rewatered during 7 days. Samples were collected from Well Watered (WW) plants  and from Water-Stressed (WS)  at the end of the drought period (point S) and after 7 days of rewatering (point R). Samples named roots consisted of the roots without nodules. Samples named nod consisted of nodules detached from the roots. Each sample consited of a mix of 18 individual plants and 4 repetitions per condition were carried out.","NimbleGen",0,378,238,"2014-08-04","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (26,"P19E26","SADE-CATMA","treated vs untreated comparison","dose","gamma rays irradiated vs non irradiated 4 days old seedlings of Col0","CATMA",296,19,167,"2010-02-03","GEO","GSE7981","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (543,"P369E543","VarCo","ecotype comparison","genotype and ploidy level","We have compared the meiotic transcriptome of two Brassica napus varieties (Darmor-bzh and Yudal), using both euploid (AACC) and allohaploid (AC) plants and a RNA-Seq approach. Three biological and two technical replicates were used for each genotype (thus twelve plants and 24 samples). All samples were bar-coded, pooled, processed and sequenced in the same lane so that one replicate of every plant was sequenced in every lane (4 lanes in total).","Illumina",0,369,0,"2014-01-29","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (551,"P375E551","BIOADAPT_NoStressWall_2011","treated vs untreated comparison","organism part","3 weeks old Brachypodium (Bd21) are submitted to an osmoticum stress ( PEG - 30mM) during 1 day or 4 days in liquid media. Leaves, stems and roots are harvested for transcriptional analysis using affymetrix microarrays.","Affymetrix",0,375,376,"2014-05-28","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (380,"P251E380","H3K4me3","chip-chip","ip/input","H3K4me3 ChIP-chip","Chromochip",0,251,133,"2010-07-09","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (487,"P316E487","Plant RABT n°1","treated vs untreated comparison","dose","Determine the incidence of RNA marking by the RABT method Plantation on the plants transcriptome.","NimbleGen",10274,316,0,"2012-07-27","GEO","GSE60532","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (73,"P48E73","Comparisons Wt-mutants 1 and 3","gene knock out","genotype","Col-0, dcl2-1, dcl3-1 and rdr2-1 were grown for 12-day on MS solid medium ,seedlings were then transferred in MS liquid medium and harvested 2 days after.","CATMA",378,48,0,"2008-02-05","GEO","GSE10352","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (488,"P317E488","Complex I","gene knock out","genotype","direct comparison between col-0 (wild type) and 4 mutants, each mutants having different levels of residual complex I. Three biological replicates were processed","NimbleGen",10275,317,0,"2012-07-27","GEO","GSE60533","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (381,"P251E381","H3K9me3","chip-chip","ip/input","H3K9me3 ChIP-chip","Chromochip",0,251,133,"2010-07-09","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (99,"P65E99","Analysis of an Arabidopsis Kdo transferase mutant","gene knock out","genotype","Comparison of RNA levels between WT and a KdoT mutant","CATMA",372,65,95,"2008-02-27","ArrayExpress","E-MEXP798","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (466,"P295E466","CRR4_DYW1_Col0","genotype comparaison","genetic background","CRR4 and DYW1  are two PPR mutants. They have the same post-transcriptonnel molecular defect and their phenotype is very close. We will try to determine here if their transcriptome is similar or not.","NimbleGen",0,295,7,"2011-08-04","GEO","GSE60534","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (417,"P260E417","CGH_ColvsC24_WG","cgh","genetic background","Comparative genome hybridization between Arabidopsis thaliana accessions Col-0 and C24.","NimbleGen",0,260,133,"2010-09-28","GEO","GSE24831","CGH");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (98,"P48E98","Flg22 treatment & mutants 2","treated vs untreated comparison","compound","Col-0, dcl4-1, rdr6-1 were grown for 12-day on MS solid medium ,seedlings were then transferred in MS liquid medium 2 days after and treated with 10uM flagellin active or inactive. Samples were harvested at 30 min after treatment.","CATMA",381,48,0,"2008-02-01","GEO","GSE10355","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (560,"P383E560","INASEEDarray","gene knock out","developmental stage,genotype","Transcriptome comparison of ale1 and gso1/gso2  mutants compared to Col-0 seeds at two developmental stages (globular and heart)","Agilent",10315,383,0,"2014-09-11","GEO","GSE68048","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (418,"P260E418","CGH_ColvsCvi_WG","cgh","genetic background","Comparative genome hybridization between Arabidopsis thaliana accessions Col-0 and CVi.","NimbleGen",0,260,133,"2010-09-28","GEO","GSE24835","CGH");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (149,"P96E149","Mutant vs WT","gene knock out","genotype","Comparison of drv1 mutant against wild type in stems","CATMA",377,96,1,"2008-03-11","GEO","GSE7421","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (550,"P374E550","12PLEX_PEA_2013_02_G","treated vs untreated comparison","developmental stage,drought,organism part,tissue type","Pea plants (genotype Cameor) were subjected to a moderate water stress at the beggining of the seed filling period (12 Days After Pollination) of the second flowering node for a period of 8 days. Samples were collected from Well Watered (WW) plants at the beginning of the stress imposition (point A, T=0), and from Water-Stressed (WS) and WW control plants at the end of the drought period (point B, T=+8). Samples named SEED consisted of  seeds from the pod of the second flowering node (seed-WW-A, seed-WW-B and Seed-WS-B). Samples named LEAF consisted of the leaves and stem sections from the two vegetative nodes  below the first flowering node (leaf-WW-A, Leaf-WW-B and Leaf-WS-B). Each sample consited of a pool of 3 individual plants and 4 repetitions per condition were carried out.","NimbleGen",0,374,238,"2014-05-27","GEO","GSE97237","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (384,"P251E384","H3K36me3","chip-chip","ip/input","H3K36me3","Chromochip",0,251,133,"2010-07-09","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (128,"P83E128","Ecotypes comparison (Col0 and Ler0)","ecotype comparison","developmental stage,strain,tissue type","Compare the transcriptome of ColO versus Ler0 for different tissues and developmental stages (seedlings 7 days, shoot 14 days, flowers, leafs 35 days)","CATMA",304,83,95,"2008-02-27","ArrayExpress","E-MEXP800","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (114,"P78E114","Stifenia","treated vs untreated comparison","compound","Stifenia sprayed and/or Fusarium infection. ","CATMA",341,78,1,"2008-03-07","GEO","GSE10713","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (133,"P85E133","RfPPR","wt vs mutant comparison","developmental stage,genetic background","Flower buds from three B. napus lines (Pactol, CMS, Rfo PPR) at one developmental stage (stage 8).","CATMA",344,85,5,"2006-12-20","GEO","GSE8820","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (408,"P260E408","H3K27me3_Cvi","chip-chip","ip/input","H3K27me3 profiling in Cvi seedlings","NimbleGen",0,260,133,"2010-09-28","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (28,"P19E28","ATM-ROOT","treated vs untreated comparison","dose","Gamma rays irradiated vs non irradiated 3 or 4-days old seedlings of Col0","CATMA",295,19,167,"2010-02-03","GEO","GSE7980","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (173,"P114E173","Maternal/embryonic ABA-1","gene knock out,treated vs untreated comparison","compound,genotype","Comparison of expression in seeds (15 days after pollination) of aba2-2 mutant to wild type (Col-0), aba2 (female) x Col-0 (male) crossing, and aba2-2 sprayed with ABA (100 micro M, twice a week).","CATMA",433,114,95,"2008-03-03","GEO","GSE10674","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (159,"P66E159","ChIP in silencing suppressor plants or in miRNA plants","chip-chip","ip/ip","How are the histone modifications and the chromatin structure in miRNA mutants or in silencing suppressor plants?","Chromochip",10282,66,95,"2007-02-19","GEO","GSE24692,GSE26739","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (302,"P197E302","RNAi Tor time course","normal vs rnai mutant comparaison","genotype","Modification of translational and transcriptional profiles in Tor inducible RNAi mutants and identification of TOR targets.","CATMA",10093,197,0,"2009-09-11","GEO","GSE25701","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (517,"P346E517","Vegetative and reproductive leaves","time course,treated vs untreated comparison","time,tissue type,traitment","Analysis of expression in Medicago truncatula reproductive leaves in untreated and nitrogen deficient plants during remobilization process between beginning of flowering, pod filling and the end of pod filling.","NimbleGen",0,346,335,"2013-04-19","GEO","GSE109521","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (281,"P177E281","Thelungiella","genotype comparaison","genotype,tissue type","flower/leaf transcriptomic comparison in Arabidopsis and Thellungiella.","CATMA",10072,177,0,"2010-01-08","GEO","GSE19740","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (436,"P269E436","fonctional caracterization of VvMYB82","wt vs mutant comparison","genotype","To compare wt with a surexpressor of VvMYB82 (Line A311 = 35s::VvMYB)and wt with a loss of fonction mutant AtMYB82","CATMA",10216,269,216,"2010-12-13","GEO","GSE30381","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (188,"P127E188","H3K9me2","chip-chip","ip/input","H3K9me2 ChIP-chip ","Chromochip",0,251,133,"2007-10-08","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (212,"P146E212","ZIP9_target_genes","gene knock in (transgenic),gene knock out","developmental stage,genotype","three genotypes were grown in MS/2, 0,03% glicose (starvation) for 7 days under constant dim light. Three days dormancy break.","CATMA",10022,146,95,"2008-04-11","GEO","GSE11140","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (215,"P125E215","CGH : Comparative Genomic Hybridization","genotype and ecotype comparison","genotype","Several factors affect today a more extended use of microarrays for research and as a diagnosis tool. These factors are the experimental cost, the reproducibility of the measurements and the format of the analyses. This project aims at bringing solutions in these three domains by developing new applications such as Comparative Genomic Hybridization (GGH) or Promoter Arrays to bring the plant microarray beyond transcriptome analyses. Altogether, these approaches will allow us to construct an optimized diagnostic tool based on 96 well microplate microarrays. This multidisciplinary project bring together biologists, chemists, physicists and mathematicians to develop innovative solutions for future application of DNA microarrays.","CATMA",10025,125,5,"2008-04-23","GEO","None","CGH");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (224,"P125E224","ChIP to see the Role of the MET2  methyltransferase on methylation and gene expression","genotype and ecotype comparison","ip/ip,organism part","Comparison of promoter methylation pattern and transcription  in roots and aerial parts of one week-old plants from two lines (WT and MET2 overexpressor)","SAP",10034,125,95,"2008-05-29","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (223,"P125E223","ChIP for identification of target genes of ABI3","genotype and ecotype comparison","ip/input","ChIP with anti-ABI3 antibody was performed on fixed chromatin, isolated from seeds of A.th. About 12 dap. Nonprecipitated chromatin, i.e. input, served for control.","CATMA",10033,125,95,"2008-05-29","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (419,"P251E419","5mC_GW","chip-chip","ip/input","Dna methylation profiling in Col-0 seedlings","NimbleGen",0,251,133,"2010-09-28","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (416,"P260E416","CGH_ColvsC24_Chr4","cgh,chip-chip","genetic background","Comparative genome hybridization between Arabidopsis thaliana accessions Col-0 and C24","Chromochip",0,260,133,"2010-09-28","GEO","GSE24494","CGH");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (33,"P22E33","AGO","gene knock out","genotype","Transcription analisys of 2 different AGO1 mutants ","CATMA",328,22,5,"2006-12-20","GEO","GSE7989","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (34,"P23E34","Oncogene_effect","gene knock in (transgenic)","time","Effects of the activation of the oncogene 6b on the transcriptome","CATMA",350,23,5,"2006-12-20","GEO","GSE7993","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (197,"P135E197","GEMSUP","gene knock in (transgenic)","genotype","Transgenic lines of Arabidopsis overexpressing three different geminiviral silencing suppressors: C2, C4 and V2 were grown in adition to a control and their total RNA was extracted in order to study their genetic expressions profiles.","CATMA",10004,135,0,"2008-01-11","GEO","GSE24475","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (257,"P160E257","LAR","normal vs disease comparison","inoculation","Plants were treated either with PstDC3000 (avrRpm1)or MgCl2 (control plants). The samples were studied at 3 points of the infection cinetics of the LAR phenomenon : 6h, 24h and 48h. 12 biological repetitions were pooled for this experiment. ","CATMA",10045,160,1,"2008-10-29","GEO","GSE13390","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (261,"P160E261","HR","normal vs disease comparison","inoculation","We want to analyse the transcriptome of Arabidopsis thaliana developping a Hypersensitive Reaction (HR). Le goal is to identify the genes involved in LAR.To achieve this, we used Col0 leaf tissues developping a HR reaction after inoculation of the  avirulent strain of  PstDC3000 carrying the gene avrRpm1.","CATMA",10049,160,1,"2008-10-29","GEO","GSE13389","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (27,"P19E27","ATM time course","time course","dose","which Arabidopsis thaliana genes are induced by ionising radiations ? time course: 0.75h, 1.5h, 3h, 5hdose: Gamma rays irradiated vs non irradiated 4-days old seedlings of Col0","CATMA",295,19,167,"2010-02-03","GEO","GSE7979","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (420,"P251E420","H3K36me3_GW","chip-chip","ip/input","H3K36me3 profiling in Col-0 seedlings","NimbleGen",0,251,133,"2010-09-28","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (65,"P44E65","PLP2 Bc 1","normal vs rnai mutant comparaison,time course,treated vs untreated comparison","disease state,genotype","2500 spores of Botrytis cinerea were pipetted on 4 infection sites per adult leaf. Leaf material was harvested at 0, 24 and 48 hours. 3 plant genotypes were used (Col-0) : PLP2- silenced (RNAi), control, PLP2-overexpressors.","CATMA",393,44,0,"2009-04-29","GEO","GSE15880","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (35,"P23E35","Dex-GVG","gene knock in (transgenic)","time","Observation of the effect of Dex-GFP expression on the transcriptome (control)","CATMA",350,23,5,"2006-12-20","GEO","GSE8062","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (37,"P25E37","Potyvirus","normal vs disease comparison","disease state","In this experiment, Arabidopsis plants infected by a virus, Tobacco etch virus (TEV), a potyvirus, were compared with healthy plants to identify genes for which the expression is modified by the viral infection. Analysis of both inoculated leaves and upper young leaves were performed 7 days after the inoculation with the virus (or with only buffer for the healthy plants). ","CATMA",321,25,5,"2006-12-20","GEO","GSE8875","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (61,"P29E61","Nickel_Leaf","gene knock in (transgenic),organ comparison","dose,genotype,organism part","After germination, plants were grown hydroponically for 6 weeks in classical nutrient solution (Marques et al, New Phytol, 2004, vol 164, pp 289-95) and then treated for 48 hours with 50 uM NiSO4 or with no nickel added for control.","CATMA",355,29,5,"2006-12-20","GEO","GSE7938","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (421,"P251E421","H3K27me1_GW","chip-chip","ip/input","H3K27me1 profiling in Col-0 seedlings","NimbleGen",0,251,133,"2010-09-28","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (146,"P94E146","TFL2","chip-chip","none","Chromatin Immunoprecipitation from a WT line expressing a TFL2:HA tagged protein. Comparison of the IP fraction to the INPUT (total genome).","Chromochip",10000,94,133,"2006-11-10","None","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (226,"P154E226","AFFY_POPSEC_Nancy_Roots_poplar ","treated vs untreated comparison","dose,genotype,stress,time","Two poplar clones, Soligo (S) and Carpacio (C) were submitted to 4 treatments: control, mild water deficit, moderate water deficit (12-day long for both) and early-drought stress (about 36-h long). Growth and physiology was characterised on a batch of plants and samples collected on another batch of plants. Four to eight root apices (1 cm-long) were collected on each individual tree. Total RNAs were extracted from all roots for each tree individually. Two pools of 3 (or 2) individuals were made using equimolar ratio. A pool is considered as one biological replicate and corresponds to one Affimetrix slide. The two biological replicates originate from the same experiment.","Affymetrix",0,154,7,"2009-10-12","GEO","GSE17223","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (310,"P203E310","Quinolinate","treated vs untreated comparison,wt vs mutant comparison","cell line","Comparison between the mutant or the control after a quinolinate treatment","CATMA",10106,203,14,"2009-10-27","GEO","GSE19084","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (298,"P193E298","Edelfosine and wortmannin effect at 22°C and 4°C","treated vs untreated comparison","compound,temperature","Experiments were carried out with Columbia suspension cells. Cells were treated or not with edelfosine (PLC inhibitor) or wortmannin (PI4-kinase inhibitor). These agents were added 15 min before cold stress. ARN were extracted 4h after stress. Controls were made at 22°C. ","CATMA",10087,193,216,"2010-01-12","GEO","GSE19850","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (299,"P194E299","upregulation of transcripts in the absence of PRC2 repressory complex","gene knock out","genotype","Comparison of transcriptome between two biological replicates of fie mutants in comparison with two biological replicates of Col WT. ","CATMA",10113,194,216,"2010-01-12","GEO","GSE19851","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (350,"P228E350","ZF1 experiment","normal vs transgenic comparaison","none","Plants were grown on soil in controlled environment under LD (16 h light/8 h dark) and the rosette leaves (8-leaf stage seedlings) were collected for RNA preparation","CATMA",10151,228,256,"2010-04-19","GEO","GSE21646","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (335,"P219E335","AU09-05_Hd2B","wt vs mutant comparison","genotype","Analysis of the mutant Hd2B","CATMA",10139,219,224,"2010-03-22","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (336,"P219E336","AU09-05_Hd2Bflg","wt vs mutant comparison","genotype","Transcriptomic analysis of Hd2b after flg22 treatment","CATMA",10140,219,224,"2010-03-22","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (2,"P1E2","Isoxaben treatment","treated vs untreated comparison","genotype","1- 45 hours, transfer to medium  +/-isoxaben, harvest at 48 hours: comparison 48 hours + /- isoxaben.2- 52 hours, transfer on + /-isoxaben, harvest 55 hours: comparison 55 hours + /-isoxaben (after transition: inhibition synthesis of cellulose, no inhibition of cellular elongation).","CATMA",325,1,167,"2010-03-17","GEO","GSE7985","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (1,"P1E1","Time course","time course","genotype","1- Time course from 45h until 55h after seed-imbibition, comparisons respectively between: 45h - 48h, 48h - 52h, 52h - 55h","CATMA",325,1,167,"2010-03-17","GEO","GSE7986","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (217,"P1E217","plant germination","time course","age","Analysis of genes expression during germination","CATMA",10135,1,14,"2010-03-19","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (167,"P109E167","Analysis of npc48 overexpressing plants","gene knock in (transgenic)","none","We have lines overexpressing npc48 that shhowed a leaf morphological phenotype (serrated leaves). The proportion of lines havin gthe phenotype is around 20% of total transformants. We wanted to compare the transcriiptome of npc48-overexpressing lines with and without phenotype together with a transformation control for an additional control. ","CATMA",431,109,95,"2008-03-03","GEO","GSE10679","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (422,"P251E422","H3_GW","chip-chip","ip/input","H3 profiling in Col-0 seedlings","NimbleGen",0,251,133,"2010-09-28","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (382,"P251E382","H3K27me3","chip-chip","ip/input","H3K27me3 ChIP-chip","Chromochip",0,251,133,"2010-07-09","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (462,"P293E462","Septante project","normal vs transgenic comparaison","genotype","Transgenic lines producing respectively octopine and mannopine and the WT line were grown in hydroponic tanks (containing nutritive solution) under short day conditions (8h light, 24°C). Whole plant aged of one month were harvested and frozen in nitrogen liquid. The plants were ground with a mortar and pestle and RNA extraction was performed with the RNeasy extraction kit (QIAGEN). The RNA concentration estimation was performed on a NANODrop spectrophotometer (Nanodrop).","CATMA",10243,293,256,"2011-07-26","GEO","GSE30975","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (423,"P251E423","H3K27me3_roots_GW","chip-chip","ip/input","H3K27me3 profiling in Col-0 roots","NimbleGen",0,251,133,"2010-09-28","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (424,"P251E424","H3K4me3_roots_GW","chip-chip","ip/input","H3K4me3 profiling in Col-0 roots.","NimbleGen",0,251,133,"2010-09-28","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (402,"P251E402","H2Bub","chip-chip","ip/input","Epigenomic mapping of H2Bub","NimbleGen",0,251,133,"2010-09-14","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (383,"P251E383","H3K27me2","chip-chip","ip/input","H3K27me2 ChIP-chip","Chromochip",0,251,133,"2010-07-09","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (385,"P251E385","H3K56Ac","chip-chip","ip/input","H3K56Ac ChIP-chip","Chromochip",0,251,133,"2010-07-09","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (386,"P251E386","H4K20me1","chip-chip","ip/input","H4K20me1 ChIP_chip","Chromochip",0,251,133,"2010-07-09","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (407,"P260E407","H3K27me3_ColxCvi","chip-chip","ip/input","H3K27me3 profiling in ColxCvi hybrids","NimbleGen",0,260,133,"2010-09-28","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (409,"P260E409","H3K27me3_Col(Cvi)","chip-chip","ip/input","H3K27me3 profiling in Col","NimbleGen",0,260,133,"2010-09-28","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (410,"P260E410","H3K4me2_ColxCvi","chip-chip","ip/input","H3K4me2 profiling in ColxCvi hybrids","NimbleGen",0,260,133,"2010-09-28","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (411,"P260E411","H3K4me2_Cvi","chip-chip","ip/input","H3K4me2 profiling in Cvi","NimbleGen",0,260,133,"2010-09-28","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (412,"P260E412","H3K4me2_Col(Cvi)","chip-chip","ip/input","H3K4me2 profiling in Col-0","NimbleGen",0,260,133,"2010-09-28","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (413,"P260E413","H3K27me3_ColxC24","chip-chip","ip/input","H3K27me3 profiling in ColxC24 hybrids","NimbleGen",0,260,133,"2010-09-28","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (414,"P260E414","H3K27me3_C24","chip-chip","ip/input","H3K27me3 profiling in C24","NimbleGen",0,260,133,"2010-09-28","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (415,"P260E415","H3K27me3_Col(C24)","chip-chip","ip/input","H3K27me3 profiling in Col","NimbleGen",0,260,133,"2010-09-28","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (426,"P260E426","H3K4me2_ColxC24_Chr4","chip-chip","ip/input","H3K4me2 profiling in ColxC24 hybrids","Chromochip",0,260,133,"2010-10-01","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (427,"P260E427","H3K4me2_Col_Chr4","chip-chip","ip/input","H3K4me2 profiling in Col","Chromochip",0,260,133,"2010-10-01","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (428,"P260E428","H3K4me2_C24_Chr4","chip-chip","ip/input","H3K4me2 profiling in C24","Chromochip",0,260,133,"2010-10-01","GEO","None","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (403,"P251E403","H3K27me3_GW","chip-chip","ip/input","Epigenomic mapping H3K27me3","NimbleGen",0,251,133,"2010-09-15","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (387,"P251E387","H3","chip-chip","ip/input","H3 ChIP-chip","Chromochip",0,251,133,"2010-07-09","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (379,"P251E379","H3K4me2","chip-chip","ip/input","H3K4me2 ChIP-chip","Chromochip",0,251,133,"2010-07-09","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (356,"P234E356","Kinetics of gene expression after light exposure in hub1-3 mutant compared to wild-type","genotype comparaison,time course","genotype,light,time","RNA from 5-day-old wild-type and hub1-3plants in 3 conditions were used for hybridization: 5 days of darkness, 5 days of darkness + 1 hour of light and 5 days of darkness + 6 hours of light. ","CATMA",10229,234,256,"2010-05-12","GEO","GSE21922","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (404,"P251E404","H3K4me3_GW","chip-chip","ip/ip","Epigenomic mapping of H3K4me3","NimbleGen",0,251,133,"2010-09-15","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (405,"P251E405","H3K4me2_GW","chip-chip","ip/input","Epigenomic mapping of H3K4me2","NimbleGen",0,251,133,"2010-09-15","GEO","GSE24710","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (398,"P257E398","H3K27me3_WT_Col_seedlings_DB","chip-chip","ip/input","Genome-wide epigenomic mapping","NimbleGen",0,257,133,"2010-09-14","GEO","GSE24163","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (400,"P257E400","H3K4me3_WT_Col_seedlings_DB","chip-chip","ip/input","Genome-wide epigenomic mapping","NimbleGen",0,257,133,"2010-09-14","GEO","GSE24163","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (397,"P257E397","H3K27me3_fie_Col_seedlings_DB","chip-chip","ip/input","Genome-wide epigenomic mapping","NimbleGen",0,257,133,"2010-09-14","GEO","GSE24163","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (399,"P257E399","H3K4me3_fie_Col_seedlings_DB","chip-chip","ip/input","Genome-wide epigenomic mapping","NimbleGen",0,257,133,"2010-09-14","GEO","GSE24163","ChIP-chip");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (434,"P267E434","AFFY_Strigolactone_Sunflower","treated vs untreated comparison"," compound","The experiment consisted of 2 repeats of four 12-day-old-plantlets of 6 sunflower genotype SF193 (INRA code: XRQ),  SF326 (INRA code: PSC8),  SF056 (INRA code: FU),  SF306 (INRA code: PAZ2),  SF302 (INRA code: PAC2),  SF268 (INRA code: RHA266),  grown in growth chamber conditions and submitted to a 24-hour-treatment of 100 nM strigolactone analogue rac-GR24 (Chiralix, Nijmegen, Netherland) or not.Growth conditions were 14h light at 23°C and 10h night at 20°C under fluorescent bulbs.Plants were grown in hydroponic boxes containing 20 litres of aerated liquid culture medium (as described in Massonneau et al., 2001 Planta).The entire root systems were harvested 4 hours after light onset and frozen immediately in liquid nitrogen.","Affymetrix",0,267,7,"2010-12-03","GEO","GSE25888","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (52,"P37E52","siliques","treated vs untreated comparison","compound","16 DAP siliques were harvested from Col 0 plants grown on 10 or 50 mM nitrate or from G4 3 mutant grown on 10 mM nitrate in growth chambers.","CATMA",329,37,5,"2009-04-03","ArrayExpress","E-MEXP-447","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (319,"P208E319","mutants versus wt","gene knock in (transgenic)","genotype","Transcriptome comparison between mutants and wild-type plants L1 in shoot of  seedlings.","CATMA",10116,208,0,"2009-12-23","GEO","GSE26413","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (125,"P81E125","Flowers_Columbia_WT_vs_ddm1_1","gene knock out","genotype","comparison of amplified polyA+RNA extracts obtained from wild type and ddm1 flowers","CAGE",310,81,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (108,"P18E108","comparison_Cul4_As_light","gene knock out,normal vs antisens mutant comparison","age,developmental stage,genotype","The seeds have been sown every two days on MS 255 (Duschefa) without 1% sugar. These  plates stayed 6 days in the dark at 4 °s. Then they were transfered to the light for 6 more days. ","CATMA",356,18,5,"2006-12-20","GEO","GSE36655","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (105,"P71E105","RS05-15_Iron","time course","compound,time","Arabidopsis cell culture was treated with 300 uM iron-citrate. Samples were collected before (0), and 5, 15, 30 and 60 minutes after iron addition. A repetition was performed for 2 points of the time course: before and 30 min after iron addition. Samples are designed as follow: Exp 1: 0(1), 5, 15, 30(1) et 60. Exp 2: 0(2) et 30(2)","CATMA",342,71,95,"2008-02-27","ArrayExpress","E-MEXP799","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (9,"P3E9","polyphenol","wt vs mutant comparison","genotype","comparison mutant cytP450 vs control WS","CATMA",289,3,167,"2009-06-24","ArrayExpress","E-MEXP198","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (175,"P116E175","AFFY_Aba_Ath1","wt vs mutant comparison","genotype","comparison of four suppressor mutants (g-radiation induced) to wild type and the ABA deficient mutant aba3-1","Affymetrix",0,116,7,"2009-10-12","GEO","GSE7961","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (142,"P90E142","Chip-Chip_GCN5","wt vs mutant comparison","genotype","xxxxxxxxxxx","CATMA",0,90,1,"2008-09-02","ArrayExpress","E-MEXP-1233","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (371,"P247E371","Salmonella_14028S","normal vs disease comparison","infection","Plants were grown during 14 days in solid medium and were transfered to liquid medium during 16 h. at 9 am samonella was added to a final concentration of OD600=0.2","CATMA",10256,247,224,"2010-07-02","GEO","GSE23790","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (393,"P225E393","mpk6","gene knock out","genotype","Comparision between mpk6 and col-0 seedlings","CATMA",10252,225,224,"2010-07-26","GEO","GSE23191","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (85,"P54E85","carbonic anhydrase (1 or 2) simple or double mutants","gene knock out","genotype","CA1 (At3g01500) and CA2 (At5g14740) T-DNA insertional mutant lines, the double (CA1+CA2) mutant and wild type Arabidopsis seeds were sown in soil in a phytotron. Leaves were harvested 40 days later for RNA extraction","CATMA",394,54,167,"2007-11-16","GEO","GSE36625","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (134,"P86E134","Cadmium-miRNA","treated vs untreated comparison","time","After 5 days of grown in a fresh medium (5% PCV at day0), CdCl2 was added to tested cells (Cad) to a final concentration of 200uM. Nothing was added to control cells (Tem). After 12 and 24 hours of growth +/- cadmium, cells were harvested and frozen in liquid nitrogen.","CATMA",375,86,0,"2006-07-19","GEO","GSE36627","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (107,"P18E107","comparison_Cul4_As_dark_light","gene knock out,normal vs antisens mutant comparison","age,developmental stage,genotype","The seeds have been sown every two days on MS 255 (Duschefa) without adding sugar. These  plates stayed 2 days in the dark at 4 °s. Then they got a 8 hours light induction ( light induction of germination) and then back to the dark for 5 more days. ","CATMA",356,18,5,"2006-12-20","GEO","GSE36652","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (347,"P226E347","AFFY_genomic_poplar","gain of fuction epimutation,genomic comparison","genotype","Two poplar cultivars, Soligo (S) and Carpacio (C) were grown in controlled conditions. Mature leaves were collected and genomic DNA was extracted from leaves in CTAB buffer. gDNA was fragmented with DNAse1. DNA fragments were labelled with Biotin N6-ddATP and hybridized on Affymetrix poplar genome array. Two technical replicates per genotype were performed.","Affymetrix",0,226,7,"2010-04-14","GEO","GSE21334","CGH");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (483,"P312E483","MYB52-2 T-DNA ","normal vs transgenic comparaison","genotype","Plants grew in short-day growth chamber until they reached 20 cm height. Ten centimeter-stem from the rosette were collected. Samples are  pools of 8 wild-type (Col0) or (atmyb52) mutant plants. Two biological repeats were conducted.","CATMA",10270,312,0,"2012-07-24","GEO","GSE43647","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (493,"P322E493","AU12-02_DEX-TAp46","normal vs rnai mutant comparaison","time","Tap46 overexpressing lines were grown for 10 days in MS (Murashige and Skoog) media and then transferred to new MS media containing ethanol (-DEX) or 10 mM DEX (+DEX).  At 1 day and 3 days after transfer, seedlings were harvested and total RNA was extracted from them. -Samples were composed of three independant biological replicates.DEX:dexamethasone","CATMA",0,322,329,"2012-12-19","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (496,"P324E496","AFFY_ATH_2012_04","gene knock out","genotype","- Seeds were sterilized in bleach, washed with ethanol, dried, dispersed on Murashige and Skoog (MS) 0.5X containing 2% sucrose and kept 5 days at 4°C.- After 12 days of growth under a cool white fluorescent light (120 &#956;mol m-2 s-1) and long day conditions (16h of light) in a growth chamber, the plants are harvested and stored at -80°C.-The RNA are extracted with the RNA Plant NucleoSpin (Machery-Nagel), quantified with Nanodrop and visualised on agarose gel. The samples are stored at -80°C.","Affymetrix",0,324,238,"2013-01-24","GEO","GSE43728","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (498,"P326E498","AFFY_ATH_2012_05","gene knock out","genotype","Seeds were stratified for 3 days at 4ºC in the dark. Afterwards, seeds were surface sterilized, sown onto MS media and grown at 23ºC, with 100 µE light condition, in long days (16h L/8h D). Each MS plate was dived in four, corresponding to a genotype (Col, AB, SIZ1 or SAB), and 10 seeds were sown per genotype. After 10 days, seedlings were collected and frozen in liquid nitrogen. Each replicate was made of ~40 seedlings coming from 4 different MS plates. Frozen plant material was grounded in an eppendorf tube using a mini pestle. RNA was extracted using RNeasy Plant Mini kit (QIAGEN cat.74904), following the manufacturer instructions (QIAGEN, Hilden, Germany). The RNA samples were DNase treated (Recombinant DNase I, Takara Biotechnology, Dalian, China) and subsequently cleaned by column using RNeasy Plant Mini kit (QIAGEN, Hilden, Germany).","Affymetrix",0,326,238,"2013-01-29","GEO","GSE43883","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (512,"P341E512","Comparison after 4 and 52h","normal vs transgenic comparaison,time course","genotype,time","Plants of four different genotypes (43A10.6, 44B1.3, H1 P3.2 and Col-0) were grown for ten days on MS/2 plates with 1% sucrose under 80µE light in long-day conditions (16 hours of light, 8 hours of dark). Plants were treated at 10:30 in the morning (Time 0; 10:30 is 3 and a half hours after the onset of the light period) by flooding with 30uM dexamethasone (in aqueous solution with 0,3 % DMSO and 0,033 % Silwet L-77) for 5 minutes . Samples were taken at 4 hrs and 52 hrs after T0. Aerial parts of the seedlings were rapidly harvested and frozen in liquid nitrogen.","NimbleGen",10288,341,0,"2013-04-12","GEO","GSE62656","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (513,"P342E513","cdt1 mutants and MCM2 over-expressors","day lenght,normal vs transgenic comparaison","genotype,light","Wild-type (Col0), MCM2 over-expressers (MCM2OE), hemizygous cdt1a/+ mutants expressing the truncated version of CDT1 (sh10.3) and homozygous cdt1 null mutants complemented with the truncated version of CDT1 (sh10.1) were grow under short and long day conditions. All genotypes were compared to wild-type plants grown in the same conditions.","NimbleGen",10289,342,0,"2013-04-15","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (30,"P20E30","Nematode-1","normal vs disease comparison","disease state","Time course of the host response to the nematod infection","CATMA",286,20,5,"2007-01-31","ArrayExpress","E-MEXP-233","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (31,"P21E31","Endothelium","gene knock out","developmental stage,genotype","Comparisons between wild type seeds and seed that have no endothelium, thus no tannin producing cells. This phenotype is obtained by specifically expressing the Barnase gene under the BANYULS promoter. ","CATMA",388,21,5,"2006-12-20","GEO","GSE8063","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (279,"P175E279","ADT08-01_upp","wt vs mutant comparison","genotype","WT/upp comaparison at the beginning of the development alteration (5d for Col0/upp-1, 6d for WS/upp-2), and 2 days later (7d for Col0/upp-1, 8d for WS/upp-2)","CATMA",10066,175,223,"2010-03-31","GEO","GSE21148","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (3,"P0E3","EBL87","normal vs disease comparison,treated vs untreated comparison","genotype","EBL87 vs WSEBL87+UV vs WS+UVEBL87 vs EBL87+UVWS vs S+UV","CATMA",442,0,0,"2007-08-30","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (4,"P0E4","DML6","normal vs disease comparison,treated vs untreated comparison","genotype","DML6 vs SDML6+UV vs S+UVS vs S+UVDML6 vs DML6+UV","CATMA",441,0,0,"2007-08-30","GEO","GSE8095","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (6,"P2E6","Comparison","normal vs disease comparison","genotype","2 technical repeats:WS Cul3a vs Mutant Cul3a","CATMA",280,2,5,"2006-12-19","ArrayExpress","E-MEXP-175","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (7,"P0E7","BX","normal vs disease comparison","organism part","comparison mutant BXHH higher part vs control WSHH higher partcomparison mutant BXHB lower part vs control WSHB lower part ","CATMA",267,0,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (8,"P0E8","CCR","normal vs disease comparison","organism part","comparison mutant CCRHH higher part of stem vs control WSHH higher part of stemcomparison mutant CCRHB lower part of stem vs control WSHB lower part of stem","CATMA",267,0,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (365,"P243E365","embryogenesis col0","tissue comparison","tissue type","compare embryogenic areas of a callus with undifferenciate area in the same callus","CATMA",10159,243,216,"2010-06-15","GEO","GSE30425 ","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (10,"P4E10","comparison TAF WS","wt vs mutant comparison","genotype","T-DNA insertions in the HAF2 gene are characterised. The mutation of the gene induced production of pale young leaves. Genetic analysis showed that HAF2 is required for both phytochrome and cryptochrome signalling. Accordingly, light-inducible gene expression and CAB2 promoter activity were affected by the mutation. The present transcriptome analysis showed that the mutation affected about 9% of Arabidopsis genes in 14 day-old leaves.","CATMA",292,4,5,"2007-01-25","ArrayExpress","E-MEXP-178","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (11,"P5E11","msil","wt vs mutant comparison","genotype","control S4 vs mutant msil2technical repeat","CATMA",326,5,5,"2006-12-20","GEO","GSE7988","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (12,"P6E12","comparison mutant HCA vs WS","wt vs mutant comparison","genotype","comparison control versus mutant high cambial activity.3 biological & technical repeats.","CATMA",282,6,5,"2006-12-19","ArrayExpress","E-MEXP-379","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (13,"P7E13","Circadian cycle","time course","genotype","The  first experiments will deal with the wild reference plant (WS) . The RNAm of leaves would be harvested according to day and night hours. Some harvesting will be done  every 3 hours  during a 24H period (so 10 different points ). This correspond to a total of 20 slides ( 2 samples and 1 swap per point). This account have been done according to the informations obtained from JJ Leguay in dec 2002","CATMA",266,7,5,"2007-02-12","GEO","GSE7975","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (14,"P8E14","Comparison WS vs DRN","wt vs mutant comparison","genotype","2 first leaves of mgo3-2 mutant or wild-type plants (WS ecotype) were harvested from plants grown during 2 weeks in vitro on MS medium in long days conditions. 3 repeats have been done for each samples.","CATMA",292,8,5,"2007-01-25","ArrayExpress","E-MEXP-196","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (15,"P9E15","Comparison Ws vs DE and DY","wt vs mutant comparison","genotype","Wild-type (WS) and two T-DNA mutant lines for the variable subunit of ferredoxin:thioredoxin reductase ( DY  and  DE  from INRA of Versailles  collection) were compared","CATMA",294,9,5,"2006-12-19","GEO","GSE7978","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (16,"P10E16","Cadmium effect on roots and leaves","dose response,time course","tissue type","Effects of two cadmium concentrations on leaves and roots at three different times","CATMA",315,10,5,"2006-12-19","ArrayExpress","E-MEXP-416","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (17,"P11E17","PPR45vsWTabis","normal vs rnai mutant comparaison","genotype","Comparaison between WT and a PPR s mutant- 1 technical replicate","Affymetrix","None",11,7,"2004-10-21","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (18,"P12E18","comparison WS vs CRB (Cul3a)","wt vs mutant comparison","genotype","Comparaison of mutant AtCul3a (T-DNA mutant of Versailles collection) and WS.","CATMA",281,12,5,"2006-12-19","GEO","GSE7974","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (19,"P12E19","comparison Col0 vs 003D","wt vs mutant comparison","genotype","Comparaison of AtCul3b (mutant of Gabi collection) and Col0.","CATMA",281,12,5,"2006-12-19","GEO","GSE7975","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (20,"P13E20","Comparison WS vs HAF2_GCN5","wt vs mutant comparison","genotype","mRNA extraxted from 72h germinating seed of the mutants gcn5 and haf2 were used for comparison with Ws seed at the same satge.2 repetitions techniques + 1 repetition biologique.","CATMA",292,13,5,"2007-01-18","GEO","GSE7977","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (21,"P14E21","Phospholipases 4°C","treated vs untreated comparison","compound","To answer this question we chose a pharmacological approach. We decided to use molecules that are going to inhibit PLC or PLD. For PLC we use U73122 and its inactive analog as a control; for PLD we used a primary alcohol (ethanol) and a tertiary alcohol (tert-butanol) as a control. We therefore extracted RNA after the following treatments:4°C22°C4°C + 0.9% primary-ethanol4°C + 09% tertiary-butanol4°C +  60 micromolaire U731224°C +  60 micromolaire U73343The exposure at 4°C was 4 hours.","CATMA",351,14,5,"2006-12-20","ArrayExpress","E-MEXP-259","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (22,"P15E22","OPEN STOMATA2","wt vs mutant comparison","genotype","Ler vs OST (Open STomata mutant),Ler drought vs OST droughtLer vs Ler droughtOST vs OST drought","CATMA",287,15,5,"2007-01-23","GEO","GSE7982","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (23,"P16E23","TOR","treated vs untreated comparison,wt vs mutant comparison","compound,genotype","Mutant vs wild type on different nitrogen conditions (10mM=control and 100uM)","CATMA",327,16,5,"2006-12-20","GEO","GSE7987","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (24,"P17E24","Iron response","time course","compound,time","Time course experiment, star design and 3 biological repeats.","CATMA",320,17,5,"2006-12-29","GEO","GSE7983","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (36,"P24E36","NTR","gene knock out,wt vs mutant comparison","genotype","Comparison of the transcript profile of the ntra ntrb knock-out mutant vs Col-0 wild-type ecotype.Seeds of the wild-type Col-0 ecotype and of the ntra ntrb homozygote double mutant were sowed in soil. Three replicates of lines were made (Col-0 1, 2 and 3 ntra ntrb 1, 2 and 3) : plants were put at different places of the greenhouse and grown at different times. Germination was synchronized by 4 days at 4°C and plants were grown during 17 days, until they reached a 6 rosette leaves stage. Entire plants (30 plants per sample) were harvested, frozen in liquid nitrogen and stored at -80°C until extraction.ntra:At2g17420, ntrb:At4g35460","CATMA",353,24,167,"2010-03-18","GEO","GSE7998","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (457,"P289E457","Analysis of Arabidopsis transcriptome in response to b-cyclocitral treatment","treated vs untreated comparison"," compound","Analysis of Arabidopsis transcriptome in response to b-cyclocitral treatment.","CATMA",10236,289,0,"2011-06-27","GEO","GSE33963","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (38,"P26E38","Myb","gene knock in (transgenic)","genotype","Xylogenesis is a fundamental developmental process that is specific of vascular plants. It allows the formation of xylem, also called wood in trees, a complex tridimensional tissue composed of different cell types. This process occurs through the control of fundamental cell mechanisms like cell division and differentiation, secondary cell wall synthesis, lignin deposition and programmed cell death.Xylogenesis is controlled spatially and temporally by specific genetic programs that involve hundreds of genes. For instance, lignin biosynthetic genes, CAD and CCR, are specifically expressed during xylogenesis through MYB transcription factor binding sites, a process that seems to be common to all vascular plants.We have cloned two xylem specific MYB transcription factors, EgMYB1 et EgMYB2, in Eucalyptus. Interestingly, they are able to bind MYB consensus sequences of CAD and CCR promoters in vitro and to modulate CAD and CCR expression in vivo. When overexpressed in Arabidopsis or tobacco, they affect xylem structure by changing cell wall structure and quality.To follow expression changes of Arabidopsis genes in transgenic plants overexpressing Eg MYB1 and EgMYB2 should help us to find out which genes might be target of those transcription factors. This should help us to decipher the actual role of those two MYBs in xylogenesis, two new members of a large family of transcription factors in plants.","CATMA",354,26,5,"2006-12-20","GEO","GSE7996","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (39,"P27E39","Comparison WT-set mutants  ","normal vs rnai mutant comparaison","genotype","comparison of the transcriptoms from mutants N5,N6 and double-mutant N5 N6 named X56 ","CATMA",358,27,5,"2006-12-20","ArrayExpress","E-MEXP-302","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (41,"P29E41","Nickel_Root","gene knock in (transgenic)","dose,genotype,organism part","After germination, plants were grown hydroponically for 6 weeks in classical nutrient solution (Marques et al, New Phytol, 2004, vol 164, pp 289-95) and then treated for 48 hours with 50 uM NiSO4 or with no nickel added for control.","CATMA",355,29,5,"2006-12-20","GEO","GSE7940","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (42,"P30E42","CDT1","normal vs rnai mutant comparaison","genotype","Comparison of 3 independant RNAi lines","CATMA",357,30,1,"2008-03-11","GEO","GSE8698","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (43,"P32E43","CCME","gene knock in (transgenic)","genotype","The homozygote arabidopsis line 9S9 overproducing AtCCME-HIS (At3g1790), a mitochondrial heme chaperon involved in c-type cytochrome maturation  will be compared with the corresponding wild type (Wassilievskjia). ","CATMA",352,32,5,"2006-12-20","GEO","GSE7995","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (44,"P32E44","Antimycine A","treated vs untreated comparison","compound","Presence of an inhibitor of the cytochrome pathway (antimycine A). This treatment induces the expression of a gene coding for an alternative  oxydase (AOX1a) and one of the two genes coding for apocytochrome c (CYTc-a). ","CATMA",352,32,5,"2006-12-20","GEO","GSE7994","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (343,"P225E343","Bak1 bkk1 flg22","treated vs untreated comparison","flg22","2 weeks Bak1bkk1 plants treated with flg22 1h 1 microM","CATMA",10142,225,224,"2010-03-25","GEO","GSE23184","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (47,"P34E47","Isogenic lines","near isogenic lines (hif)","genotype","The lines HIF404-Sha & HIF404-Bay were grown on 3 mM nitrate in growth chamber according to Loudet et al . 2002 TAG 104 : 1173-84 until phenotypique yellowing symptoms differed between them. Plants were collected 49 days after sowing.","CATMA",334,34,5,"2006-12-20","ArrayExpress","E-MEXP-387","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (48,"P34E48","Sugars and leaf senescence","treated vs untreated comparison","compound","Title: Comparaison of sugar effect on the leaf senescence progress using three RIL from the Bay-0 x shahdara propulation that exhibited differential leaf yellowing symptoms. The recombinant inbred lines RIL310 (hypersenescing and early-senescing), RIL232 (early--senescing) et RIL045 (late-senescing) were grown in petri dishes (4,7 mM Nitrogen, with (LNG) or without (LN) glucose 2% until some difference of Fv/Fm trait can be observed (Wingler et al. 2004, New Phytol, 161 : 781-789). Growth experiments were performed in A. Wingler lab, University College of London.","CATMA",334,34,5,"2006-12-20","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (49,"P35E49","ddm1 loop Col and Ler (Seedlings) ","genotype and ecotype comparison","genetic background,genotype","Loop design :Comparison of gene expression in seedlings (1 biological replicat) between WT and the ddm1 mutant, for Col and Ler ecotype. And comparison of gene expression between Ler and Col ecotype, for the WT genotype and for the ddm1 genotype.","Chromochip",10000,35,10,"2005-10-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (126,"P81E126","Flowers_Columbia_WT_vs_ddm1_2","gene knock out","genotype","comparison of amplified polyA+RNA extracts obtained from wild type and ddm1 flowers","CAGE",312,81,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (50,"P35E50","ddm1 loop Col and Ler (Flowers) ","genotype and ecotype comparison","genetic background,genotype","Loop design :Comparison of gene expression in flowers (1 biological replicat) between WT and the ddm1 mutant, for Col and Ler ecotype. And comparison of gene expression between Ler and Col ecotype, for the WT genotype and for the ddm1 geneotype.","Chromochip",10000,35,10,"2005-10-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (222,"P124E222","Gene profiling of Turnip Crinkle Virus (TCV) siRNA in mutants at generation G1 and G11","gene knock out,treated vs untreated comparison","genotype","Col-0, dcl2/4, dcl3/4 and dcl2/3/4 were grown until rosette state and inoculated with TCV. Ten days post-inoculation the leaves were harvested. The same  was made at generation 1 (G1) and 11 (G11).","CATMA",10031,124,95,"2008-05-20","GEO","GSE34659","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (199,"P113E199","Arabidopsis transcriptome microarray from virus infected leafs.","normal vs disease comparison","developmental stage,disease state","mRNA and microRNA expression profile in virus-infected leafs of Arabidopsis.","CATMA",440,113,0,"2009-04-06","GEO","GSE15557","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (55,"P38E55","Water stress","gene knock in (transgenic),treated vs untreated comparison","genotype,stress","Overexpression of the hahb4 gene vs vector transformed control","CATMA",359,38,5,"2006-12-20","GEO","GSE8878","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (56,"P39E56","Nt-Kis","gene knock in (transgenic)","developmental stage","35S NtKIS (CDK inhibitor) versus Col0","CATMA",360,39,1,"2008-03-11","GEO","GSE8697","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (57,"P40E57","time course H2O2 traitement","treated vs untreated comparison","compound","Arabidopsis cells (5 days after subculturing) were cultured at 21°C, 8 h photoperiod under agitation. H2O2 2.5 uM was added to cells 5 days after subculturing and 2 hours after the beginning of the light period. Treated and control cells were collected 15 min, 1 h, 5 h and 12 h after treatment. ","CATMA",330,40,5,"2006-12-20","GEO","GSE7941","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (58,"P41E58","ACC-treated root vs. control root","treated vs untreated comparison","age,compound,time,tissue type","Arabidopsis plants are grown for 5 days on MS medium and then we transfer them to a medium supplemented with ACC (5uM) or without ACC. After 3 h of growth, we isolate the RNA of the roots.","CATMA",335,41,5,"2006-12-20","GEO","GSE7935","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (59,"P14E59","Phospholipases 22°C","treated vs untreated comparison","cell line","These experiments are control experiments, where we want to investigate the effect of LC and PLD inhibitors at 22°C. ARN were extracted from cells at 22°C that had been treated for 90 min with 60 uM U73122, 60 uM U73343, 0.9% (v/v) ethanol or 0.9% (v/v) tertiary butanol.  We compared the conditions control vs. 60 uM U73343; 60 uM U73122 vs. 60 uM U73343 control vs. 0.9% (v/v) tertiary butanol and 0.9% (v/v) ethanol vs. 0.9% (v/v) tertiary butanol.","CATMA",351,14,5,"2006-12-20","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (60,"P42E60","Nin7","gene knock out","dose,genotype","Plants were harvested during the first 3 hours of light after 42 growth on 0.2mM and 6mM nitrate in hydroponic condition (short day, 170 uE). 42 DAYS of growth","CATMA",333,42,5,"2007-02-09","GEO","GSE7932","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (156,"P102E156","AFFY_pathogen_rice","normal vs disease comparison,time course,treated vs untreated comparison","disease state","Two-week old rice plants (cultivar Nipponbare) were treated with either Magnaporthe grisea (virulent isolate FR13) spore suspension in gelatine or gelatine alone. Two time points were taken (3 and 4 days post inoculation- dpi). Disease symptoms were not visible at 3 dpi whereas they were at 4 dpi. Two biological repeats were done.","Affymetrix",0,102,7,"2008-01-31","GEO","GSE7256","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (186,"P126E186","Drought and salt stress response","gene knock out,treated vs untreated comparison","genotype,stress","Individual plants were grown in Fertiss clods in a culture chamber. 3 treatments were made to the plants : control, drought and salt.","CATMA",444,126,95,"2008-02-05","GEO","GSE10384","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (205,"P105E205","Treatment g-butyrobetaine WT Col.0","treated vs untreated comparison","compound","Seedlings WT are growing in a independent way on media MS. After seven days of culture, seedling are harvested and a pull is formed from the three prelevements. Extractions of RNA will be realised on this pull. We proceed in a identical way for the test sample. Seedlings are cultivated on media MS containing butyrobétaïne 1 mM. After seven days, seedlings are harvested and extractions of RNA will be realised on this second pull. All this experiment is repeated one more time independently.","CATMA",10012,105,95,"2008-03-03","GEO","GSE10677","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (63,"P43E63","Profiling of the PGPR induced systemic resistance (ISR)","normal vs rnai mutant comparaison,treated vs untreated comparison","developmental stage","Seeds were sawn on 0.8% (W/V) agar mineral medium (see below). 4 days after storage in the dark at 4°C, seedling were cultivated 6 days in a growth chamber (16 h daily, 20-22°C) and then transferred on soil inoculated or not with 107 cfu.g-1 of Bradyrhizobium strain ORS278. Three weeks later, 3 leaves per plant were infiltrated with a suspension of Pseudomonas syringae pv. tomato (2.105 cfu.ml-1) or with MgSO4 10 mM alone for control plants. Infiltrated leaves were collected 24h later.","CATMA",332,43,5,"2006-12-20","GEO","GSE7990","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (320,"P197E320","Overexpression of pten2a","gene knock in (transgenic)","developmental stage","Comparaison control versus overexpressor. The overexpressor has a male phenotype of sterility.","CATMA",10117,197,0,"2010-01-06","GEO","GSE24902","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (95,"P44E95","PLP2 Botrytis cinerea 2","gene knock in (transgenic),normal vs rnai mutant comparaison,time course","disease state,genotype","5000 spores of Botrytis were pipetted on 4 infection sites per ault leaf. Leaf material was harvested at 0,1,2,3 days later.3 plant genotypes were used (Col-0 ecotype): siPLP2 (RNAi-silenced),pBIN (empty pBIN-transformed),PLP2OE (PLP2-overexpressors).","CATMA",393,44,65,"2009-05-06","GEO","GSE15972","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (204,"P141E204","Growth on MS of quadruple and GPL2:VP16","gene knock in (transgenic),gene knock out","age,developmental stage,genotype,strain,temperature,time","sterilized seeds were sown on MS medium and grown for 14 days (16h light/8h dark) under 100 mocroEinstein. No stratification of seeds.","CATMA",10263,141,149,"2008-02-14","GEO","GSE36732","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (67,"P45E67","Effect of atrazine and sucrose on Arabidopsis transcriptome","treated vs untreated comparison","compound","Seeds of Arabidopsis thaliana (ecotype Columbia) were surface-sterilized inbayrochlore/ethanol (1:1, v/v), rinsed in absolute ethanol and dried overnight. Germination and growth were carried out under axenic conditions in square Petri dishes. After seeds were sowed, Petri dishes were placed at 4°C for 48 h in order to break dormancy and homogenize germination, and then transferred to a control growth chamber at 22°C under a 16-h light period regime at 4500 lux for 4 weeks. Growth medium consisted of 0.8% (w/v) agar in 1x Murashige and Skoog (MS) basal salt mix (Sigma, St. Louis, MO, USA) adjusted to pH 5.7. After 4 weeks of cultivation on vertical plates plantlets were transferred on fresh medium complemented or not with atrazine 10 uM and with sucrose 80mM or mannitol 80 mM which were directly added during preparation of agar-MS media prior to sterilisation. Atrazine was sterilized by microfiltration through 0.2 um cellulose acetate filters (Polylabo, Strasbourg, France) and then axenically added to melted agar-MS medium prior to pouring into Petri dishes. Then plantlets were harvested after 24 hours of transfer and extracted for RNA.","CATMA",362,45,0,"2007-04-11","GEO","GSE7942","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (68,"P46E68","SGS2_and_3","gene knock in (transgenic)","genotype","Transcriptome of rdr6 and sgs3 mutants will be compared to the transcriptome of wild-type plants in flower and leaves. Further-more the comparison between transcriptome of rdr6-1 and sgs3-1 mutant alleles impaired in PTGS and development (juvenile-to-adult transition) and transcriptome of rdr6-5 and sgs3-3 alleles impaired only in PTGS will be done for flowers and leaves.","CATMA",337,46,5,"2006-12-20","GEO","GSE7936","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (180,"P121E180","DELLA-regulation of salt stress responses","dose response,gene knock out,time course","genotype","Aim was to determine downstream target of DELLA proteins involved in salt stress tolerance. Wt, ga1-3, penta seeds were sterilized, sown on MS agar plates then put for stratification for 3 days at 4°C. Plates were placed in growth cabinet for 9 days. Seedlings were transferred to 24 well-plates, with 2 seedlings per well (0.5 ml MS liquid per well). Plates were placed in the same condition for 3 days. Finally, NaCl were added (final concentration 200 mM), except for the control. The salt treatment was applied for 30 min and 1h. Treatment was stopped by freezing in liquid nitrogen.","CATMA",10037,121,163,"2008-06-23","GEO","GSE8556","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (72,"P48E72","Comparisons Wt-mutants 2 ","gene knock out","genotype","Ler, dcl1-9, hen1-1 and hasty were grown for 12-day on MS solid medium ,seedlings were then transferred in MS liquid medium and harvested 2 days after.","CATMA",378,48,0,"2008-02-01","GEO","GSE10353","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (174,"P115E174","ABA4/1","gene knock in (transgenic),gene knock out","genotype","Comparison of different genotypes to wild type Col-0: aba4-3 mutant, two independent 35S-ABA4 transgenic lines and two independent 35S-NCED6 transgenic lines.","CATMA",427,115,0,"2008-02-01","GEO","GSE10357","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (74,"P49E74","0-nitrogen","wt vs mutant comparison","compound","plants were cultivated in hydroponie condition (light 16h / dark 8h, temperature 22°C)in 3 nitrogen medium conditions : 1mM NO3, nitrogen deficient medium (5 days) and then feed in 1mM or 4mM NH4. After 6 weeks of culture, leaves from 3 plants (6.20-6.30 boyes stage) were sampled for each treatment then pooled.","CATMA",10136,49,14,"2010-03-19","ArrayExpress","E-MEXP-446","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (198,"P130E198","per8::VIP micro-array","treated vs untreated comparison","cell line,gene induction","estradiol-induced expression of VIP1-YFP and VIP1_D-YFP; 3h estradiol treatment. First comparison: non-induced versus induced. Second comparison: VIP1 lines versus YFP control line","CATMA",10006,130,14,"2008-10-07","GEO","GSE13094","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (330,"P215E330","oct07","gene knock out","genotype","Mutants impaired for the DNA methylation maintenance pathway and for the RNA interference machinery were compared to mutants simultenously impaired for both machineries to estimate its impact on genome activity.","CATMA",10128,215,0,"2010-01-19","GEO","GSE26843","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (219,"P150E219","AFFY_CCR_Maize","gene knock in (transgenic)","genotype","2 wild type and 2 CCR mutants were compared. Plants were grown in greenhouse condition and harvested at 7-8 leaf stages. ","Affymetrix",0,150,7,"2008-06-09","GEO","GSE11531","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (76,"P49E76","NH4","wt vs mutant comparison","compound","plants were cultivated in hydroponie condition (light 16h / dark 8h, temperature 22°C)in 3 nitrogen medium conditions : 1mM NO3, nitrogen deficient medium (5 days) and then feed in 1mM or 4mM NH4. After 6 weeks of culture, leaves from 3 plants (6.20-6.30 boyes stage) were sampled for each treatment then pooled.","CATMA",10137,49,14,"2010-03-19","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (168,"P50E168","DGPP - 2 ","treated vs untreated comparison","compound","5ml of 3 days-old suspension cells were incubated with ABA or DGPP 18:1 for 3hours under the conditions of culture. DGPP was emulsified by sonication for 1min 4 times at 4°C in 1ml of culture medium then added to 4ml of suspension cells. Cells were filtrated under vacuum, frozen in liquid nitrogen and RNA extracted. DGPP is from Avanti Polar Lipids.","CATMA",428,50,0,"2008-09-26","GEO","GSE12938","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (191,"P48E191","Flagellin experiment in dcl2-dcl3-dcl4 mutant background","treated vs untreated comparison","compound","What are the genes (including primary miRNA transcripts) that are differentially regulated upon flg22 treatment?14-day old seedlings (Col-0 and dcl2-dcl3-dcl4) treated with flg22 (flagellin- derived peptide).","CATMA",451,48,0,"2008-02-01","GEO","GSE10354","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (79,"P7E79","Double mutant SBE","gene knock out","genotype","Biological question : what is the impact of the absence of both branching enzymes II on the expression of the genes involved in carbon metabolism in the Arabidopsis leaves?","CATMA",291,7,5,"2006-12-19","GEO","GSE7976","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (80,"P51E80","NO","treated vs untreated comparison","compound","Germination of surface sterilized seeds was performed on solid MS/2 medium. After 2 weeks, the young plantlets were placed on sand for an additional 3 week period in a controlled environment (8 h photoperiod of 300 umol m-2 s-1, 22°C, 70% relative humidity) . Plants of similar rosette diameters were then transfered to an hydroponic culture using the nutritive solution described in [Gravot et al. 2004 FEBS Letters 561:22-28] for an acclimation of 3 days. L-NAME and cadmium treatments were respectively started 1 and 2 hours after the start of illumination and root tissues were collected 24 h later and immediatly frozen in liquid nitrogen before RNA extraction. Treatments were :1 : control2 : L-NAME 5 mM3 : Cd 30 uM4 : L-NAME 5mM + Cd 30 uM5 : Cd 15 uMFor each condition, 3 independent biological repetitions were conducted. RNA were extracted separatly and checked for quality before pooling together.","CATMA",365,51,5,"2006-12-20","GEO","GSE7945","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (216,"P113E216","Arabidopsis transcriptome microarray from virus infected roots.","normal vs disease comparison","developmental stage,disease state,tissue type","Arabidopsis miRNA/transcriptome microarray from virus infected roots of plants.","CATMA",10028,113,0,"2009-04-06","GEO","GSE15558","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (158,"P104E158","atnar2.1-1","dose response,gene knock out","compound,dose,genotype","Plants were grown for 41 days in hydroponics in media containing 6 mM nitrate and then transferred to 6 or 0.2 mM nitrate media for 24h ( short days, irradiation 150 uE). Plants were harvested on day 42 during the first 2 hours of light.","CATMA",10020,104,1,"2008-03-11","GEO","GSE10705","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (129,"P83E129"," Impact of SINE RNA ","gene knock in (transgenic)","developmental stage,genotype,tissue type","Analyse the consequence of the expression of a SINE RNA in Arabidopsis thaliana.","CATMA",305,83,95,"2008-02-27","ArrayExpress","E-MEXP800","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (86,"P55E86","2 colors","organ comparison","organism part","Comparison of 2 colors and 3 colors hybridization and development of new normalization methods.","CATMA",449,55,65,"2007-10-31","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (87,"P55E87","3 colors & organs","organ comparison","organism part","Comparison of 2 colors and 3 colors hybridization and development of new normalization methods.","CATMA",450,55,65,"2007-10-31","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (88,"P57E88","all comparison","gene knock out","genotype","seeds were sown in vitro and plantlets (Col0 wild type ,and  n1, n2, n3, x123, sdg8, vip4,sdg25 and sdg26 mutants)were cultivated during 6 days (1.02 boyes stadge)","CATMA",426,57,167,"2007-07-25","GEO","GSE8427","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (89,"P58E89","Nitrogen starvation and re-supply","time course","tissue type","WS Arabidopsis ecotype were grown on 6mM nitrate as sole nitrogen source during 35 days under short days . At T0, plants were then starved for nitrate for 10 days and  root and shoot samples were harvested separately 2 and 10 days after treatment (T2, T10). Then,  nitrate (6 mM) was re-supplied for 1 and 24 hours (T+1, T+24).","CATMA",293,58,5,"2006-12-19","GEO","GSE8031","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (90,"P59E90","Overexpressors","treated vs untreated comparison","compound","Identification of target genes (direct and undirect) by activation of an inducible form (GR fusion) of a transcription factor in a constitutively active form (VP16 fusion)","CATMA",425,59,1,"2008-09-29","GEO","GSE12913","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (91,"P60E91","Erwinia","normal vs disease comparison","disease state,time","Identification of Arabidopsis genes regulataed by Erwinia amylovora and of a subset of Arabiddopsis genes regulated by the type three secretion system of Erwinia amylovora","CATMA",336,60,5,"2006-12-20","GEO","GSE8701","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (92,"P56E92","Analysis of Atnuc-L1 mutant line","gene knock out","genotype","transcriptome analysis of the Atnuc-L1 mutant line","CATMA",364,56,5,"2006-12-20","GEO","GSE7943","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (93,"P61E93","Rhodococcus response","normal vs disease comparison,time course","disease state,time","Arabidopsis Thaliana plants were infected with Rhodococcus fascians D188 in 1.05 stage and at two timepoints after infection (7, 14 and 24 days post infection) samples were harvested for Microarray hybridisations.As a control Arabidopsis Thaliana plants were infected with the non virulent strain Rhodococcus fascians D188-5.","CATMA",303,61,65,"2007-01-08","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (171,"P112E171","TSN_FJ","gene knock in (transgenic),gene knock out","genotype,stress","Comparison of a tsn1 tsn2 double mutant line (A10.9.3- or A13.3.11-) and an isogenic line complemented by TSN2 (A10.9.5)or TSN1 (A13.3.2)","CATMA",10067,112,0,"2009-04-06","GEO","GSE15559","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (130,"P83E130"," Impact of the gcn2 mutation","gene knock out","developmental stage,genotype,tissue type","Impact of the gcn2 mutation on the Arabidopsis thaliana transcriptome. GCN2 is a kinase potentially regulated by the SINE non-coding RNA","CATMA",304,83,95,"2008-02-27","ArrayExpress","E-MEXP800","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (104,"P70E104","Preliminary experiment:repetability","genotype and ecotype comparison","developmental stage,genotype,organism part","The third experiment is a comparison of the transcriptome of two parents (Bay0,Sha) and two RILs (104, 140) in loop experimental design.Plants are grown in individual pots on a tray.Flowers were tagged on the flowering day using colored tape. Siliques were harvested 10 days after flowering, freezed in liquid nitrogene and stored at -80°C until RNA extraction.","CATMA",391,70,7,"2009-05-28","GEO","GSE16241","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (150,"P97E150","Time course Uranyl nitrate response","organ comparison,time course,treated vs untreated comparison","compound,organism part,time","Plants are grown on sand and transfert in hydroponic culture  during 2 days and then expose or not to 50uM uranyl nitrate at pH 4.5 in water or only to water at pH 4.5. Roots and leaves were collected independently after 2h, 6h and 30h of treament. ","CATMA",10019,97,1,"2008-06-18","GEO","GSE11797","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (322,"P124E322","Endogenous Hairpins-IR71 targets","gene knock out","genotype","Lines IR71 KO n°3 and Col-0 were grown on soil until bolting, Flower buds were then harvested and RNA extracted with Qiagen RNeasy Plant mini kit. RNA was stored at -20°C.","CATMA",10120,124,0,"2010-01-08","GEO","GSE34653","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (54,"P37E54","soaked seeds","treated vs untreated comparison","seed imbibition","Freshly harvested C10, C50 or G4 3 seeds obtained from mother plants grown respectively with 10 mM nitrate or 50 mM nitrate in growth chambers were sown on 0.5% Litex agarose plates and imbibed for 6 hrs in a culture chamber (at 25°C, 100uE m-2 sec-1 at 70% relative humidity).","CATMA",329,37,5,"2009-04-03","ArrayExpress","E-MEXP-447","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (177,"P119E177","DspA/E of Erwinia amylovora","wt vs mutant comparison","strain","Regulation of the Arabidopsis transcriptome by the type three effector DspA/E of Erwinia amylovora.5-week old Arabidopsis plants were leaf infiltrated with Erwinia amylovora wild-type (wt), type three secretion mutant (sec) or dspA/E mutant (dspA/E) strains.","CATMA",347,119,0,"2008-02-01","GEO","GSE8436","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (206,"P118E206","Potyvirus effect","normal vs disease comparison","strain","Arabidopsis thaliana ecotype Ler was inoculated with three different Potyviruses. About 4 weeks after sowing, the 6 expanded leaves plants were inoculated with the different viruses or were mock-inoculated. Seven days after inoculation, inoculated leaves were collected, RNA was extracted and virus infection controlled. Three biological repeats have been done and two dye-swaps.","CATMA",10015,118,1,"2008-03-07","GEO","GSE10707","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (101,"P67E101","Transcriptome Analysis of prc1 or  the1 and prc1_the1","gene knock out","genotype","We have isolated two loci THE1 and THE2, which are identified by mutations that partially suppress the dark-grown hypocotyl phenotype in a mutant background for cellulose synthase catalytic subunit CESA6/PROCUSTE1. THE1 encodes a receptor kinase and may play a role in this feedback signalling process. To identify genes that are regulated by THE1 and THE2, we compared the transcript profiles of 5 day-old dark-grown seedlings of the1-1/prc1-1 with prc1-1 ; the1-3/prc1-8 with prc1-8 ; prc1-8 or the1-3 with WS and the2-1/prc1-1 with prc1-1.","CATMA",340,67,5,"2006-12-20","GEO","GSE7937","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (164,"P70E164","Seed development","time course","developmental stage","This experiment is a time course performed to choose the best harvesting point to maximise the number of detectable transcripts (8, 10, 12 and 14 dap).","CATMA",392,70,7,"2009-05-28","GEO","GSE16242","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (106,"P72E106","Leaf_Flower-bud","organ comparison","tissue type","Genes expressed in flowers and leaves","CATMA",385,72,5,"2006-12-19","ArrayExpress","E-MEXP-94","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (148,"P96E148","Overexpressor vs WT","gene knock in (transgenic)","genotype","Comparison of DRV1 overexpressor against wild type in stems","CATMA",377,96,1,"2008-03-11","GEO","GSE7420","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (187,"P70E187","Embryo seed development","time course","developmental stage","Analysis of the expression of small RNAs precursors in the seed development of Arabidopsis Thaliana.","CATMA",446,70,65,"2007-10-10","GEO","GSE28311","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (308,"P201E308","miRNA_Nitrogen","time course","tissue type","To discover miRNA implied following a deficiency out of nitrogen and after reinduction.","CATMA",10201,201,0,"2009-10-27","GEO","GSE25010","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (431,"P264E431","Comparison of transcriptomes in mad mutants","genotype comparaison","genotype","Seeds of GFP171.1 (parental line), mad1, mad2, mad3, mad6 and dcl1-12 were sterilized and germinated on Murashige/Skoog medium with 0.9% agar. Plates were stratified at 4C in the dark for 4 days. The plates were then transferred to a growth cabinet at 21C under a 16h light/8h darkness light regime, and the seedlings were harvested 18 days after transfer to the growth cabinet.","CATMA",10202,264,256,"2010-11-26","GEO","GSE30477","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (110,"P74E110","Seed_maturation","gene knock out","developmental stage","Comparaison mutant/wt in time course","CATMA",309,74,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (111,"P75E111","Pollen","ecotype comparison","genetic background","RNAs of Pollen from Columbia and WS were compared","CATMA",276,75,5,"2006-12-19","GEO","GSE6910","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (163,"P70E163","Seed specific genes","organ comparison","tissue type","This experiment aimed to identify the genes expressed in the seeds vs siliques (easiest to harvest), and to know if the expression pattern of the whole silique could provide a valuable information.","CATMA",392,70,7,"2009-05-28","GEO","GSE16244","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (113,"P77E113","PLD","gene knock out,treated vs untreated comparison","genotype,stress,temperature","A double mutant PLDzeta1,2 was obtained in a Columbia background. 12-day-old Arabidopsis seedlings, wt or double mutant, were treated with either NaCl (200 mM), mannitol (400 mM) or with 4°C temperature during 3 h. 3 biological replicates were pooled.","CATMA",373,77,1,"2006-05-29","GEO","GSE9459","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (115,"P73E115","FWA-Pollen","gene knock out","genotype","Compare gene expression between WT pollen and mutant pollen","CATMA",302,73,1,"2008-03-11","GEO","GSE10717","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (182,"P123E182","AtMIPS mutant/photoperiod","gene knock out","light","Wild type Colombia and MIPS mutant (Salk 023626) plants are grown in short days (8h light) during 27 days and then transfered in long days (16h light) during 4 days. J4JC correspond to 27+4=31 days plants in short days.","CATMA",10102,123,167,"2009-10-12","GEO","GSE15123","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (351,"P229E351","TAG_Forward","organ comparison","organism part","transcriptomic profiling to flowing buds vs germinating seeds; leaf_4F vs leaf_2F; stem vs root and seeds10dap vs mature_leaf. All with one biological repetition and in dye-swap.","NimbleGen",0,229,7,"2010-05-03","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (252,"P125E252","Multiplexage :  triple-target hybridization microarray experiment","genotype and ecotype comparison","genotype,sources of nitrogen ,time","Triple-target hybridization microarray experiment with doble factors comparison : two mutants and its wild-types at different times.The aim of this part is  to reduce the slides, the cost and to optimize the transcriptome datas with a third dye.","CATMA",10040,125,65,"2008-09-03","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (264,"P163E264","AFFY_herbicide_vulpin","treated vs untreated comparison","genotype","Comparison using heterologous hybridisation onto a wheat whole-genome microarray of transcriptome of three pools of black-grass plants obtained 2h30 after herbicide spraying at field rate. The three pools correspond to:· Sensitive plants (killed by herbicide).· Moderately resistant plants (growth impaired by herbicide but plants still alive)· Resistant plants (growth unimpaired by herbicide)","Affymetrix",0,163,7,"2009-05-28","GEO","GSE15864","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (119,"P81E119","seedlings_8j_Columbia_WT_vs_ddm1","gene knock out","genotype","comparison of amplified polyA+RNA extracts obtained from wild tupe nad ddm1 seedlings","CAGE",312,81,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (120,"P81E120","seedlings_10j_Columbia_WT_vs_ddm1","gene knock out","genotype","comparison of amplified polyA+RNA extracts obtained from wild tupe nad ddm1 seedlings","CAGE",312,81,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (121,"P81E121","seedlings_10j_Ler_WT_vs_ddm1","gene knock out","genotype","comparison of amplified polyA+RNA extracts obtained from wild type and ddm1 seedlings","CAGE",312,81,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (122,"P81E122","seedlings_8j_Ler_WT_vs_ddm1","gene knock out","genotype","comparison of amplified polyA+RNA extracts obtained from wild type and ddm1 seedlings","CAGE",310,81,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (123,"P81E123","Flowers_Ler_WT_vs_ddm1_1","gene knock out","genotype","comparison of amplified polyA+RNA extracts obtained from wild type and ddm1 flowers","CAGE",310,81,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (124,"P81E124","Flowers_Ler_WT_vs_ddm1_2","gene knock out","genotype","comparison of amplified polyA+RNA extracts obtained from wild type and ddm1 flowers","CAGE",310,81,5,"2006-12-19","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (132,"P85E132","CMS4:19 and At Chr3","wt vs mutant comparison","developmental stage,genetic background","Flower buds from four different B. napus lines (fertile cv Hanna, CMS-line 4:19, restored line 46, cv Hanna with the additon of A. thaliana chr 3) at two developmental stages (stages 0-5 and stage 8).","CATMA",344,85,5,"2006-12-20","GEO","GSE8819","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (214,"P148E214","Erwinia chrysanthemi treatments","normal vs disease comparison","inoculation","Comparison of 4 transcriptoms : Wt Arabidopsis ecotype infected by Wt Erwinia chrysanthemi strain, bos1 Arabidopsis mutant infected by Wt Erwinia chrysanthemi strain, Wt Arabidopsis ecotype infected by BOS1-non inducing pelBC Erwinia chrysanthemi mutant, Wt Arabidopsis ecotype buffer inoculated. Experiments were repeated 3 times.","CATMA",10024,148,1,"2008-06-18","GEO","GSE11793","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (136,"P88E136","Translatome-Cd","transcribed vs translated,treated vs untreated comparison","compound,rna type","Comparison between translated RNA (polysomes) and total RNA or non translated RNA (monosomes) in cadmium stress conditions.","CATMA",387,88,5,"2006-12-19","GEO","GSE8061","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (367,"P239E367","Primary Floral Stem 20cm","normal vs transgenic comparaison","age,genotype,tissue type","Each mutant compare to wild type, samples only primary floral stem of 20cm","CATMA",10231,239,256,"2010-06-17","GEO","GSE22517","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (109,"P73E109","FWA-leaves","gene knock out","genotype","Compare gene expression between WT leaves and hypomethylated leaves  ","CATMA",302,73,1,"2008-03-11","GEO","GSE10724","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (83,"P53E83","Cyt_Aux_Balance","dose response","compound","Different hormone concentrations were used to grow the cells","CATMA",271,53,1,"2008-03-11","GEO","GSE10725","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (141,"P89E141","NRT2.4","gene knock out","genotype","Mutants have no phenotype","CATMA",389,89,5,"2006-12-20","GEO","GSE7933","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (395,"P255E395","mpk4/MPK4 GOF vs mpk4/MPK4 vs Col0","genotype comparaison","genotype","In order to study of the genes controlled by MPK4 pathway we generate mpk4 complemented lines expressing active MPK4 (MPK4 GOF). For the transcriptome analysis we compare the MPK4 GOF to mpk4/MPK4 (complemented mpk4 mutant with a wt MPK4) and to Col0.","CATMA",10230,255,256,"2010-08-16","GEO","GSE23664","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (342,"P225E342","Col-0 flg22","treated vs untreated comparison","flg22"," 2 weeks Col-0 plants treated 1h with flg22 1 microM","CATMA",10141,225,224,"2010-03-25","GEO","GSE23186","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (147,"P95E147","tetraploid","genotype and ecotype comparison","genotype","transcriptome changes in synthetized tetraploids compared to their diploid progenitors ","Affymetrix",10000,95,7,"2006-11-14","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (135,"P87E135","MKK2_mkk2_and_WRKY19_cold_time_course","time course,wt vs mutant comparison","genotype,temperature,time","seedlings were put to 4 ° comparing wild type ColO versus mkk2 knockouts, MKK2 overexpressors and wrky19 knockouts.","CATMA",279,87,1,"2008-03-07","GEO","GSE10763","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (155,"P59E155","crc1 mutant","gene knock out","genotype","Comparison between the wild type Ler and crc1 mutant.","CATMA",425,59,1,"2008-09-29","GEO","GSE12914","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (51,"P36E51","Club root response","normal vs disease comparison,time course","disease state,time","The objective of the study was to identify the genes up and down-regulated during the interaction of P. brassicae with a partial resistant genotype of Arabidopsis. The transcriptome pattern of inoculated and non-inoculated plants of Arabidopsis was compared at three time points after inoculation (24 hours, 48 hours and one-week ai).Sixty plants were studied by comparison (inoculated-non inoculated) and by kinetic point. RNAs of the 30 inoculated plants and the RNAs of the 30 non-inoculated plants were pooled to eliminate variability between individual plants and thus to target for genes for which differences in expression were linked to the inoculation by the pathogen. Five comparisons have been performed: inoculated-non inoculated at 24 hours ai; inoculated-non inoculated 48 hours ai; inoculated-non inoculated one-week ai; inoculated 24h ai-inoculated 48h ai; inoculated 48h ai- inoculated one-week ai. The entire experiment was repeated (biological repetition) and each comparison was performed in two independent hybridisations (technical repetition).","CATMA",322,36,65,"2008-02-12","GEO","GSE8877","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (165,"P70E165","eQTL analysis (random pair design)","genotype and ecotype comparison","genotype","Comparison of the developping seed transcriptome in 160 Bay0/Sha Recombinant Inbred Lines (RILs) at 10 days after pollinisation. ","CATMA",391,70,7,"2009-06-05","GEO","GSE16393","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (200,"P136E200","Trancript profiling in Arabidopsis GDH mutant.","wt vs mutant comparison","genotype,organism part","Comparison of transcriptome in leaves, stems and roots in wild type versus triple GDH (1, 2, 3)-defficient mutant and in wild type versus GDH3- defficient mutant.","CATMA",10008,136,14,"2009-01-28","GEO","GSE14614","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (143,"P91E143","wat1-1","gene knock out,organ comparison","genotype,organism part","wat1-1 (N509.9) mutant vs wild type in 2 different stages, with biological repetition","CATMA",10057,91,167,"2009-02-20","GEO","GSE10716","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (153,"P101E153","AFFY_Bt2_maize","gene knock out","developmental stage,organism part,strain","At 35 DAP whole kernels (pericarp + endosperm + embryo) without glumes of green house grown ears of heterozygous (+/bt2-H2328), self-pollinated plants were visually divided into pools of phenotypically normal looking kernels (small indentation, slightly smaller than mutant kernels, genotype +/+ or +/bt2-H2328) and pools of phenotypically mutant kernels (plump, round kernels, slightly larger than normal kernels, genotype bt2-H2328/bt2-H2328). Pools consisted of 4 kernels. 3 different ears were used for a biological duplicate.","Affymetrix",0,101,7,"2008-01-31","GEO","GSE7030","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (118,"P80E118","Transcriptome of hag4 root","gene knock out","genotype","hag4 mutant (with a insertion in HAG4 gene encoding a Histone Acetyl Transferase) and wild-type ecotype (Ws) were grown during 15 days, in vitro. RNA were extrated from roots of seedlings. Each sample (ws or hag4) corresponds to a pool of  3 independant cultures and harvesting. ","CATMA",370,80,1,"2008-06-16","ArrayExpress","E-MEXP-792","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (162,"P107E162","identification of SA responsive genes dependent on PLD activation","treated vs untreated comparison","cell line","The experiment is done on Arabidopsis suspension cells, ecotype Columbia : Research of phospholipase D (PLD) activity implication in the response to SA. The use of primary alcohol, like N-butanol, makes possible to derive PLD activity towards the production of phosphatidylalcohol with the detriment of phosphatidic acid, the product of the PLD. Thus, in the presence of n-butanol, the response of the genes to SA dependent on phosphatidic acid will see their response disturbed. On the contrary, in the presence of tertiary butanol, the response of the genes should not be disturbed, tertiary alcohols not acting on the PLD.","CATMA",432,107,95,"2007-11-27","GEO","GSE9695","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (392,"P225E392","mpk3","wt vs mutant comparison","genotype","Comparisions between mpk3 and col-0. 2 weeks seedlings.","CATMA",10251,225,224,"2010-07-26","GEO","GSE23187","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (284,"P180E284","AFFY_POPSEC_Nancy_Stomata_poplar ","treated vs untreated comparison","dose,genotype,stress,tissue type","Two poplar clones, Soligo (S) and Carpacio (C) were submitted to 4 treatments: control, mild water deficit, moderate water deficit (12-day long for both) and early-drought stress (about 36-h long). Growth and physiology was characterised on a batch of plants and samples collected on another batch of plants. Five hundred stomata (one thousand five hundreds guard cells) were laser microdissected in mature leaves of each tree. Stomata from 3 trees (1500) were pooled for total RNAs extraction. A pool is considered as one biological replicate and corresponds to one Affimetrix slide. The two biological replicates originate from the same experiment.","Affymetrix",0,180,7,"2009-10-12","GEO","GSE17219","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (270,"P169E270","circad_pop1","time course","time","Global  transcriptome analysis of tension wood  in poplar over a 24h period. Yound differentiating xylem was harvested on the tension wood side of 6 month-old poplar trees grown in greenhouse conditions. The samples were harvested at 6 hour intervals throughout the day. The bioinformatic analysis will permit the identification of genes expressed in tension wood and  that cycle with a nycthemeral oscillation pattern.","Affymetrix",0,169,7,"2009-10-12","GEO","GSE16459","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (172,"P113E172","Arabidopsis transcriptome microarray from virus infected plants.","normal vs disease comparison","developmental stage,disease state,tissue type","Arabidopsis transcriptome microarray from virus infected plants.","CATMA",440,113,0,"2009-04-06","GEO","GSE15562","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (166,"P108E166","methotrexate treatment","time course,treated vs untreated comparison","compound,stress,time","Treatment of cells of Arabidopsis by methotrexate which is an inhibitor of the synthesis of the folate. Consequently, the pool of folates presents a  strong reduction. The goal of this project is to look at the variations of expression in the Arabidopsis genome and specifically  in some genes implied in the metabolism of the folate.","CATMA",10013,108,95,"2008-03-03","GEO","GSE10675","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (161,"P105E161","carnitine effect","treated vs untreated comparison","compound","Our objective is the description of the existence of an intracellular transport depend on the carnitine at Arabidopsis thaliana. For that, we planned the study of the response of the transcriptome in the presence of exogenic L-carnitine. Seedlings WT are cultivated on medium MS, 6 independent times. At 144h, the seedlings are collected and pools are formed with 3 samples. We proceed in the same way for the seedlings which are cultivated on medium MS containing L-carnitine 5mM.","CATMA",439,105,95,"2008-03-03","GEO","GSE10676","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (112,"P76E112","atnrt2.7","gene knock out","dose,genotype,organism part","Plants were harvested after 32 days for plants under high nitrogen nutrition and 42 days for low nitrogen nutrition. Plantes were grown hydroponic condition (short day, 170 uE).","CATMA",343,76,1,"2008-06-16","ArrayExpress","E-MEXP-801","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (196,"P134E196","AFFY_fertility2_chicken","normal vs disease comparison","developmental stage,genotype,tissue type","We use 4 different hen lines-one line of laying hens with 3 different samplesthe just ovulated oocyte, the oocyte collected 24 hours before ovulation (F1 stage), and granulosa cells collected at the F1 stage. We could compare different tissue and developmental stages.- one line of hen with rapid growth speed- two lines of laying hensFor the 3 last lines we used animals with different fertility levels. We collected the oocyte of the largest follicle before ovulation (F1). The aim of the study is to identify genes involved in fertility or early embryo mortality. ","Affymetrix",0,134,7,"2008-01-31","GEO","GSE10231","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (282,"P178E282","cytb1-1","gene knock out","genotype","We want to check which possible changes occur in gene expression by knocking out the AtCytb1 gene.","CATMA",10074,178,0,"2010-01-08","GEO","GSE19739","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (170,"P111E170","Drought","treated vs untreated comparison","stress","Drought-stressed plants were compared to non-stressed plants, using total, polysomal or non-polysomal RNAs.","CATMA",10050,111,5,"2008-10-29","GEO","GSE13366","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (355,"P233E355","Nitrate induction - nlp mutants","gene knock out,time course","genotype","10 days old seedling grown in liquid culture on 3mM nitrate wer starvec for N for 3 days and the kinetic for the resupply of nitrate was studied during a short kinetic (0,5, 10,20 minutes).","CATMA",10283,233,256,"2010-05-05","GEO","GSE39261","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (169,"P110E169","AFFY_fertility_chicken","normal vs disease comparison","developmental stage,genotype,tissue type","We use 4 different hen lines.one line of laying hens with 3 different samplesthe just ovulated oocyte, the oocyte collected 24 hours before ovulation (F1 stage), and granulosa cells collected at the F1 stage. We could compare different tissue and developmental stages.one line of hen with rapid growth speed:two lines of laying hens.For the 3 last lines we used animals with different fertility levels. We collected the oocyte of the largest follicle before ovulation (F1). The aim of the study is to identify genes involved in fertility or early embryo mortality. ","Affymetrix",0,110,7,"2008-01-31","GEO","GSE7805","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (84,"P48E84","Flg22 treatment & mutants","time course,total transcriptome knowledge","compound","What are the genes (incuding microRNA precursors) that are differentially regulated upon flg22 treatment?14-day old seedlings treated with flg22 (flagellin- derived peptide).","CATMA",381,48,0,"2008-02-05","GEO","GSE10383","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (178,"P57E178","sdg25 and sdg26","gene knock out","genotype","seeds were sown in vitro and plantlets (Col0 wild type ,and  n1, n2, n3, x123, sdg8, vip4,sdg25 and sdg26 mutants)were cultivated during 6 days (1.02 boyes stadge)","CATMA",426,57,167,"2007-07-25","GEO","GSE8429","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (321,"P185E321","DRN/DRN-L expressing cells vs non-epressing cells","genotype comparaison","cell type","In primordia timing (pt) mutant background DRN::erGFP transgenic cell culture (somatic embryos suitable to regenerate plants) has been protoplasted and protoplasts have been sorted for GFP + and GFP- cells. Total RNA was extracted and sent to Evry for comparison (samples DRN a,b,c).  DRN-L::erGFP transgenic cell culture has been protoplasted and protoplasts have been sorted for GFP-positive and GFP-negative cells ( samples DRN-L-2-1 and DRN-L-3-1).","CATMA",10119,185,0,"2010-01-08","GEO","GSE19799","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (127,"P82E127","vdac1-p26","gene knock out,treated vs untreated comparison","genotype","Arabidopsis lines (WT, vdac1.1, p26.1, vdac1.1-p26.1) were grown in vertical position on ABIS medium (Lanquar et al., 2006). One week-old seedlings were transferred on a fresh ABIS medium containing either 0 or 30 mM salicylic acid. After 2 hours, the all seedlings were harvested. Three replicates were done; total RNAs from these 3 experiments were mixed together for the transcriptomic analysis.","CATMA",371,82,1,"2008-06-16","ArrayExpress","E-MEXP-795","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (260,"P138E260","MKK2EE and mkkkc5 mutants","wt vs mutant comparison","genotype","Comparison between the over-expressed mutant of the MKK2 gene and the KO mutant for the MKKKC5 gene versus the wild type.","CATMA",10048,138,14,"2009-01-28","GEO","GSE14444","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (208,"P142E208","PLC_cold_plant","treated vs untreated comparison","compound","To answer this question we chose a pharmacological approach. We decided to use molecules that are going to inhibit PLC.","CATMA",10018,142,14,"2009-01-14","GEO","GSE13295","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (366,"P239E366","Primary Floral Stem 40cm","normal vs transgenic comparaison","age,genotype,tissue type","Each mutant compare to wild type, samples only primary floral stem of 40cm","CATMA",10232,239,256,"2010-06-17","GEO","GSE22518","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (179,"P120E179","miR398 mutant analysis","gene knock out","genotype","Plants were grown on soil in culture chambers under the following conditions, a day/night cycle of 16/8 hours was applied, light intensity was around 100-150 umol m-2 s-1, temperature day/night was 20/15°C. Plants were harvested 17 days later and RNAs were then extracted. The experiment was repeated twice.","CATMA",434,120,95,"2008-03-03","GEO","GSE10672","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (185,"P52E185","GCN5_2","gene knock in (transgenic)","genotype","Two gcn5 mutants were grown on MS during 10 days (1.03 boyes stadge) : gcn5-1 and gcn5-2. Each mutant was compared to the wild type (Ws)","CATMA",443,52,1,"2008-09-02","ArrayExpress","E-MEXP-1241","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (193,"P132E193","Cells","dose response","dose","3 doses of sugar (Carbon) and 3 doses of nitrogen were applied to cells : C : 7,5g, 15g, 30g; and N : 25mM, 2.5mM and 0.25mM","CATMA",10001,132,1,"2007-12-11","GEO","GSE36623","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (314,"P206E314","AFFY_EN_Maize","genotype comparaison","developmental stage,genotype","Comparison of 2 maize lines :Cm484: good digestibilityF98902: bad digestibility) , at the following developmental stade 2: male flowering","Affymetrix",0,206,5,"2009-11-02","GEO","GSE18857","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (192,"P131E192","Sphingo-1","treated vs untreated comparison","temperature","7ml of 5 days-old cells suspensions were incubated in presence of different sphingolipid pathway inhibitors, 30 min to 2 hours depending in the coumpound (all were resuspended in DMSO and control weew done with DMSO). Then a 30 min cold choc was applied before cells were harvested and frozen in cold nitrogen. RNA were then extracted.    FB1 and DMS were from Alexis , Myr from Cayman, TSP  from matreya.","CATMA",10262,131,0,"2007-11-27","GEO","GSE36659","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (294,"P188E294","AFFY-AGRO-BI-Medicago","growth in nitrogen-sufficient (s) vs. nitrogen-limited (l) conditions","genotype,organism part","The plant root systems of plants were split into two parts, each one being installed in a separate compartment. For the S treatment, one part was supplied with 10 mM NH4NO3 while the other part was supplied with a nitrogen-free nutrient solution. For the L treatment, one part was supplied 0.5mM NO3- and the other part was supplied with the nitrogen free solution. Eight biological materials (designated AL, AS, SL, SS, IL, IS, NIL, NIS), with three biological repeats for each, were collected and analyzed. The effects of the S and L conditions were investigated on wild-type A17 (AL vs. AS) and sunn-2 mutant plants (SL vs. SS); one set of A17 plants was inoculated with Sinorhizobium meliloti (IL vs. IS), harvested at four days post inoculation and compared to non-inoculated plants (NIL vs. NIS).","Affymetrix",0,188,7,"2009-10-12","GEO","GSE17253","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (250,"P99E250","Amirim_Yehudya_part_3_1","genotype and ecotype comparison","stress","same experiment as Amirim_Yehudya part1, but on leaves and roots of young plants: 4-5 leaves stage growth","Affymetrix",0,99,136,"2008-07-24","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (207,"P118E207","Arabidopsis ecotypes","normal vs disease comparison","inoculation","Four different Arabidopsis ecotypes were inoculated with one Potyviruse. About 4 weeks after sowing, the 6 expanded leaves plants were inoculated with the different viruses or were mock-inoculated. Seven days after inoculation, inoculated leaves were collected, RNA was extracted and virus infection controlled. RNA fron infected plants was then used for microarrays hybridization. Three biological repeat have been done and two dye-swap.","CATMA",10016,118,1,"2008-03-07","GEO","GSE10711","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (181,"P122E181","WT/MYB77","gene knock in (transgenic),gene knock out","genotype","Plants were grown in vitro on 1/2 MS medium, 3% sucrose, 0.8% agar for 21 days under balance day cycle (12 hours daylight at 22°C; 12 hours dark at 20°C) in climate chamber. The whole plants were collected for RNAs extraction.","CATMA",435,122,95,"2008-03-03","GEO","GSE10673","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (301,"P196E301","Silencing Mutants","genotype comparaison","genotype","Lines IR71 KO n°3, smd8#25, sde3 (Col-0), sde5 (Col-0), Suc-SUL(Col-0) and Col-0 were grown for 11 days on MS solid medium, seedlings were then transferred in MS liquid medium and harvested 2.5 days after.","CATMA",10092,196,0,"2009-09-10","GEO","GSE23915","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (71,"P48E71","Flg22 time course","time course,treated vs untreated comparison","compound","14-day old seedlings (Col-0 and fls2 KO) treated with 10uM of flg22 (flagellin derived peptide) over a one hour timecourse.","CATMA",383,48,0,"2008-02-07","GEO","GSE10418","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (190,"P129E190","NaCl","treated vs untreated comparison","genotype","Arabidopsis seedlings have been treated with 150 mM NaCl in orden to determine small RNAs specifically induced by this stress.","CATMA",447,129,0,"2008-02-07","GEO","GSE10420","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (154,"P36E154","Clubroot CATMA-2","organ comparison","disease state","The objective was to identify the genes up and down-regulated between susceptible and partially resistant plants. The Arabidopsis accession Bur-0 shows partial resistance against the isolate eH of P. brassicae, whereas it is susceptible against the isolate e2. The transcriptome patterns of Bur-0 plants inoculated either by isolate e2 or isolate eH were compared at one week after inoculation. A similar comparison was done on roots. The entire experiment was repeated (biological repetition) and each comparison was performed in two independent hybridisations (technical repetition).","CATMA",322,36,0,"2008-02-07","GEO","GSE10426","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (210,"P144E210","rewatering","treated vs untreated comparison","stress","The experimental design consisted in a differential water supply. Experiments were done on tilted trees to induce tension wood formation. Trees were regularly irrigated at full capacitance until July 18, 2007, the beginning of the experiment. Two sets of plants were designed. For each set, irrigation was withheld until July 25, 2007. At this time, one of the two sets (treated-sample) was re-irrigated at full capacitance during 14hrs (WDR) while the other one was maintained under water deficit (WD). Samples were collected after the 14hrs for all trees.","Affymetrix",0,144,7,"2009-06-05","GEO","GSE16398","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (78,"P48E78","Agrobacterium treatment","normal vs disease comparison,organ comparison","developmental stage,disease state,tissue type","Col-0 plants were infected with wt Agrobacterium tumefaciens (strain A208) by stabbing the emerging stems. 18 days later, growing tumors were harvested as were the non-infected part of the stabbed-stem.In parallel, Col-0 seeds were germinated on Callus Induction Media (CIM) and 18 days later, calli were harvested.","CATMA",379,48,0,"2008-02-01","GEO","GSE10350","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (201,"P137E201","SQE1","gene knock out","genotype","Arabidopsis microarray from Ler ecotype and the sqe-1 mutant.","CATMA",10009,137,0,"2008-02-01","GEO","GSE10349","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (195,"P33E195","Meiocyte transcriptome","organ comparison","cell type,tissue type","comparison of total RNAs from isolated meiocytes with those isolated from root tips, and those from leaves. Comparison of transcriptomes of normal stamen (ecotype columbia) with root tips and leaves. Comparison of transcriptomes of normal stamens  (ecotype L. erecta) with stamens from sporocyteless mutant (L. erecta).","CATMA",10002,33,0,"2008-02-01","GEO","GSE10229","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (82,"P48E82","Comparisons Wt-mutants 1","gene knock out","genotype","What are the genes (including miRNA precursors) that are differentially regulated in a set of tasiRNA mutants?Col-0, dcl4-1, rdr6-1 were grown for 12-day on MS solid medium ,seedlings were then transferred in MS liquid medium and harvested 2 days after.","CATMA",378,48,0,"2008-02-01","GEO","GSE10351","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (359,"P236E359","mpk4 salt","treated vs untreated comparison","nacl","2 weeks seedling were treated with 150 mM NaCl during 1 h.","CATMA",10157,236,224,"2010-05-13","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (94,"P62E94","Drought stress","gene knock out,treated vs untreated comparison","genotype","Wild type and mutant arabidopsis plants were grown in soil for 6 weeks under short days. Each lot of plants is shared into 2 groups, one of which was watered normally (arr) whislt the other half was not watered for a week (sech).","CATMA",386,62,0,"2008-02-07","GEO","GSE10427","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (251,"P99E251","Ldn_Git_part3_2","genotype and ecotype comparison","stress","Drought stress experiment with 2 cultivated durum wheat Langdon (LDN) and Gitit (GIT).","Affymetrix",0,99,136,"2008-07-25","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (81,"P52E81","GCN5","gene knock out","genotype","gcn5, hd1 ou double mutant gcn5hd1 were grown on MS during 10 days (1.03 boyes stadge). Different genes show an opposed expression profil in gcn5 compared to hd1. For these genes, the double mutant might show an expression similar to the wild type. ","CATMA",366,52,167,"2009-03-06","ArrayExpress","E-MEXP-1233","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (344,"P225E344","mpk3 flg22","treated vs untreated comparison","flg22","2 weeks mpk3 plants treated 1 h with 1 microM flg22","CATMA",10143,225,224,"2010-03-25","GEO","GSE23188","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (253,"P158E253","mpk","gene knock out","genotype","1) mpk5 (WS genetic background) versus wt-WS 2) mpk11 (Col0 genetic background) versus wt-Col03) mpk10 (Col0 genetic background) versus wt-Col0Seeds were surface sterilized and grown in MS agar medium (3% sucrose) for 11 days in long day conditions.Entire seedling were harvested and deep frozen in liquid nitrogen.","CATMA",10039,158,14,"2009-11-18","GEO","GSE18749","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (317,"P207E317","abi4","wt vs mutant comparison"," compound","Abi4 is regulator pleotropic that interview in the metabolism glucose","CATMA",10110,207,14,"2009-11-18","GEO","GSE26044","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (295,"P189E295","AFFY_root-dvt-Nitrogen_Medicago","dose response,wt vs mutant comparison","dose,genotype","Three successive experiments in growth chamber were performed on two different genotypes of Medicago truncatula. Each experiment constituted a biological replicate. For each of them, after 4 days of seed cold-treatment followed by 4 days of germination, individual plantlets were transferred onto hydroponic culture tanks containing vigorously aerated nutrient solution. Each tank contained 6 plantlets of J5 and 6 plantlets of the mutant. On one shelf of the growth chamber, nutrient solution in the tanks was supplemented by 1mM of KNO3; on the other shelf, the level of KNO3 in the solution was of 10mM. Nutrient solution was renewed every week. Roots of the two genotypes were harvested separately but at the same time, 10 days after the plantlets transfer onto hydroponic culture tanks. ","Affymetrix",0,189,7,"2009-10-12","GEO","GSE18318","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (376,"P248E376","UPRT overexpression ","normal vs transgenic comparaison","genotype","Transcriptomic analysis of the overexpression of URPT","CATMA",10214,248,0,"2010-07-07","GEO","GSE26847","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (211,"P145E211","Ler vs pop2-1: GABA over-accumulation effects","gene knock out,time course","compound,genotype,time","Designed experiment consisted in comparison of transcriptomes of Arabidopsis thaliana Landsberg erecta ecotype and its mutant pop2-1 (impaired in GABA transaminase activity) during a kinetic of endogenous GABA accumulation. For this purpose, 10-day-old plants grown on half strength Hoagland s agar medium were transferred to agar plates supplemented with 1 mM GABA. We isolated RNA from plants treated for 0, 1 and 4 days. Treatments were made in duplicate.","CATMA",10046,145,0,"2008-10-21","GEO","GSE11129","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (213,"P147E213","Transcriptome analysis of fry2 mutants","gene knock out","genotype","The Arabidopsis 14-5sep mutant was isolated as overexpressing a NII::LUC construct fusing a tobacco nitrite reductase gene with the luciferase reporter gene. This mutant overexpressed also the endogenous Arabidopsis nitrite reductase gene. Map-based cloning showed that the 14-5 sep mutant was affected in the FRY2 gene. Transciptome analysis was performed on both the 14-5sep and the fry2-1 mutants and their respective controls (either the C24 ecotype containing the RD29A::LUC transgene for fry2-1 or the Col0 accession containing the NII::LUC transgene for the 14-5sep mutant) to analyze which genes are differentially expressed in the mutants versus control plants.","CATMA",10023,147,95,"2008-04-15","GEO","GSE11174","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (227,"P155E227","AFFY_floral_Rose","time course","developmental stage,genotype","This project aims to find in rose genes involved in flowering control (floral initiation and recurrent flowering).First, the floral initiation will be observed in 3 genotypes. Then we will check if same genes are regulated within genotypes for this process.Concerning recurrent blooming, we will compare flower bud versus vegetative buds in non-recurrent conditions and finally bud from non-recurrent and recurrent genotypes.","Affymetrix",0,155,7,"2009-10-12","GEO","GSE17370","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (220,"P151E220","comparative analysis of CATMA  chips  with a MET2 overexpression line","organ comparison,wt vs mutant comparison","genotype,organism part","Comparison of transcription in roots and aerial parts of one week-old plants from two lines (WT and MET2 overexpressor)","CATMA",10032,151,95,"2008-05-28","GEO","GSE11563","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (157,"P103E157","AFFY_Nitrogen_Medicago","treatment variations","organism part,sources of nitrogen ","The root system of non-nodulated (NO3- and NH4+) or nodulated (N2) plants is split into two unequal parts and each one is installed in a separate compartment. For the S treatement, the major root part is supplied with NH4NO3 10 mM whereas the minor part is supplied with either NO3- 1mM, NH4+ 1mM or N2. For the C treatement, the major root part is supplied with nitrogen-free nutrient solution whereas the minor part is supplied with either NO3- 1mM, NH4plus 1mM or N2. Each treatement is four days long. Samples of roots of six biological types (NO3S, NO3C, NH4S, NH4C, N2S and N2C) were collected. Two biological repeats per biological types have been analyzed. The effect of the S and C treatments were investigated for each N sources by comparing Affymetrix transcriptomes (NO3C vs NO3S, NH4C vs NH4S,  N2C vs N2S).","Affymetrix",0,103,7,"2008-01-31","GEO","GSE9818","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (221,"P152E221","AFFY_Phloem_ATh1","gene knock in (transgenic),gene knock out","genetic background","Transcriptome map of a mutant  (versus WT and overexpressor) impaired in phloem differentiation and vascular patterning-","Affymetrix",0,152,7,"2009-10-12","GEO","GSE11532","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (312,"P205E312","ISD-Starch","gene knock out","genotype","comparison between several mutant of the starch biosynthesis and the WT, at the end of the day and at the end of the night ","CATMA",10107,205,14,"2009-10-28","GEO","GSE21793","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (96,"P63E96","Seaweed extract and hydroponically grown Arabidopsis plants","treated vs untreated comparison","tissue type","Columbia Arabidopsis ecotype were grown hydroponically on 1 mM NH4NO3 as sole nitrogen source during 35 days under short days. Plants leaves were then sprayed with a seaweed extract or with water (control). Root and shoot samples were harvested separately 1 hour and 7 days after this spraying.","CATMA",307,63,1,"2008-06-16","ArrayExpress","E-MEXP-573","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (373,"P247E373","Salmonella_prgH","normal vs disease comparison","infection","Plants were grown during 14 days in solid medium and were transfered to liquid medium during 16 h. at 9 am samonella was added to a final concentration of OD600=0.2","CATMA",10257,247,224,"2010-07-02","GEO","GSE23791","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (445,"P243E445","embryogenesis ws","tissue comparison","tissue type","To compare embryogenic areas of a callus with undifferenciate area in the same callus","CATMA",10219,243,216,"2011-03-02","GEO","GSE30383","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (345,"P225E345","mpk4 flg22","treated vs untreated comparison","flg22","2 weeks mpk4 plants treated & h with 1 microM ","CATMA",10144,225,224,"2010-03-25","GEO","GSE23189","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (259,"P161E259","Dark and Light","treated vs untreated comparison","compound,genotype","Three genotypes: Two lines of Arabidopsis transformed to overexpress QPRT, an enzyme involved in NAD synthesis (NadC4.11 and NadC15.3) and a third control line transformed with an empty vector (162). Leaf discs from the two lines of  the QPRT+ plants were incubated with quinolinate, the QPRT substrate, to boost leaf NAD.  Discs from the empty vector control were incubated on a buffer withour quinolinate. The experiment was performed in the light and the dark. The aim is to compare the QPRT+ quinolinate samples with the empty vector samples, both for the sample series generated in the light and that generated in the dark.","CATMA",10103,161,0,"2009-10-26","GEO","GSE15599","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (225,"P125E225","Udo_Functional characterization of AtMYB44 transcription factor","genotype and ecotype comparison","genotype","Transcriptome analysis of Arabidopsis wild type plants and MYB44 knock out mutant under light stress condition","CATMA",10036,125,95,"2008-05-30","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (254,"P158E254","BB","gene knock out","genotype","Transcriptomic analysis of the double mutant bak1-4 bkk1-1 to compare its gene expression profile with that of the single mutants atmekk1 and atmpk4 and elucidate their involvement in the brassinolide signalling.","CATMA",10041,158,14,"2009-11-18","GEO","GSE18748","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (46,"P33E46","Meiocytes","total transcriptome knowledge","tissue type","Comparison of meiocyte transcriptome versus leaf transcriptome.","CATMA",338,33,0,"2008-10-01","GEO","GSE13000","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (131,"P84E131","Hypocotyls","organ comparison","developmental stage","Seeds of A. thaliana were sowed on culture medium poured in Magenta boxes. They were allowed to germinate. Plantlets were in vitro for 5 or 11 days and harvested for RNA extraction.","CATMA",368,84,0,"2010-01-05","ArrayExpress","GSE14648","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (144,"P92E144","Urea","treated vs untreated comparison","compound,tissue type","Columbia Arabidopsis ecotype were grown hydroponically on 0.5 mM NH4NO3 as sole nitrogen source during 35 days under short days. Plants were then placed on 3 nutrient solutions supplemented, either with 1 mM NH4NO3, or with 0.5 mM NH4NO3 + 0.5 mM Urea, or with 1 mM Urea. Root and shoot samples were harvested separately 7 days after these different nitrogen treatments","CATMA",10042,92,1,"2008-09-23","GEO","GSE9258","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (117,"P79E117","ESR1-DRN","normal vs transgenic comparaison","compound","Identification of the target genes of the ESR1/DRN TF using dexamethasone induction","CATMA",275,79,0,"2008-09-24","GEO","GSE12919","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (184,"P125E184","Multiplexage : single-target hybridization microarray experiment ","genotype and ecotype comparison","none","Single-target hybridization microarray experiment to perform the statitical analysis for three-colours microarrays.","CATMA",448,125,65,"2007-10-30","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (70,"P47E70","Stages 1.02-1.04 and 1.06","normal vs transgenic comparaison","dose,genotype,organism part","Seeds were surface sterilized, chilled at 4°C for 4 days, and then germinated and grown on square Petri plates containing sterilized Murashige and Skoog (MS) medium with 1% sucrose. Such stock plates were arranged vertically in plastic racks and placed into growth chamber. After 5 days, plants were transferred carefully onto fresh MS medium big round plates. On each plate, 60 Wild Type (WT) plantlets were transferred on the half right of the plate, and 60 transgenic plantlets (B, E or T lines) were transferred on the half left of the plate. Plates were arranged horizontally and placed into growth chamber.Experiment 1 : 5 plates containing WT0A control plants and B0A transgenic plants.Experiment 2 : -5 plates containing WT1 control plants and B transgenic plants.                        -5 plates containing WT2 control plants and E transgenic plants.                         -5 plates containing WT3 control plants and T transgenic plants.Plants were harvested after 7 days (experiment 1) or 12 days (experiment 2), and WT roots or transgenic roots were pooled and conserved at -80°C.","CATMA",301,47,1,"2008-06-16","ArrayExpress","E-MEXP-404","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (102,"P68E102","LIF","gene knock in (transgenic),organ comparison","age,developmental stage","comparison to wild type of four mutants (LIF1, LIF2, LIF1LIF2, LHP1) : stage in vitro Leaf1.04 / In vivo Leaf1.04 / flower.","CATMA",339,68,0,"2008-10-01","ArrayExpress","E-MEXP-802","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (140,"P87E140","Mutants","gene knock out","genotype","Several mutants in MKK pathway were tested against the wild type.","CATMA",278,87,5,"2006-12-19","GEO","GSE10764","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (77,"P50E77","DGPP","treated vs untreated comparison","compound","Five ml of 3 days-old suspension cells was incubated with ABA or lipids for 3 h under the conditions of culture. Lipids were emulsified by sonication for 1 mi, four times, at 4°C, in one ml of culture medium then added to 4 ml suspension cells. Cells were filtrated under vacuum, frozen in liquid nitrogen and RNA extracted. Dioleoyl PA and dioctanoyl PA are from Sigma, dioleoyl DGPP and dioctanoyl DGPP are from Avanti Polar Lipids. ","CATMA",428,50,0,"2008-10-02","GEO","GSE13019","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (231,"P157E231","SYN","genotype and ecotype comparison","genotype","transcriptomic changes in synthetic hexaploid derived from a cross between a synthetic tetraploid and a natural diploid","Affymetrix",0,157,136,"2008-07-21","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (248,"P99E248","Amirim_Yehudya_part1","genotype and ecotype comparison","stress","Drought stress related changes on wild wheat expression profiles, Amirim and Yehudya","Affymetrix",0,99,136,"2008-07-23","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (249,"P99E249","Ldn_Git_Pool_part2","genotype and ecotype comparison","stress","Drought stress experiment with 2 cultivated durum wheat Langdon (LDN) and Gitit (GIT) and theirs derived pools of RILs.","Affymetrix",0,99,136,"2008-07-24","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (304,"P190E304"," AFFY_PetalDvt_Lyon_Rose","scented vs non scented flowers","genotype","In this study, we used a microarray approach to compare the transcriptome of a scented rose flower (PF) versus non scented rose flower (RF). The objective is to identify genes which expression correlates with flower opening and scent emission. Samples (petal tissues) were collected at the same time early in the afternoon. Total RNA was extracted using the Plant RNA kit (Macherey Nagel), and then used to hybridize Rosa-Affymetrix microarrays.","Affymetrix",0,190,7,"2009-10-12","GEO","GSE18357","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (374,"P247E374","invA_vs_14028S","normal vs disease comparison","infection","Plants were grown during 14 days in solid medium and were transfered to liquid medium during 16 h. at 9 am samonella was added to a final concentration of OD600=0.2","CATMA",10254,247,224,"2010-07-02","GEO","GSE34664","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (258,"P138E258","Alternaria","treatment variations","cell line","Alternaria brassicicola infection of 25 day old mkkkc5- , pGreen::MKK2EE-myc and WT Col-0 plants. Sampling after 0, 1 and 2 days. Comparison between mutants and WT at corresponding timepoints. Additionally comparison between WT-samples at different timepoints. ","CATMA",10047,138,14,"2009-01-28","GEO","GSE14443","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (266,"P165E266","ABA-treated lpp2 mutant ","treated vs untreated comparison","cell line","Fully developed rosette leaves were immersed for  1 min  in a solution of 10-5 M ABA  with 0.1% DMSO and 0.05%silwet. RNA are extracted after a 3h incubation. This treatment is compared to control (1 min immersion in 0.1% DMSO and 0.05%silwet) then 3h incubation.","CATMA",10053,165,14,"2009-01-28","GEO","GSE14446","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (377,"P249E377","Laccase mutations","normal vs knock out comparison","genotype","Each mutant compare to wild type. All plants were harvested at the same developmental stage in the same growth chamber between 10h30 and 11h.","CATMA",10233,249,256,"2010-07-07","GEO","GSE22907","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (349,"P227E349","Genes regulated by ROS and DME","gene knock out","genotype","Seeds of  Col x ROS-/-/DME+/- 3 and 6 days after pollination (DAP) were harvested. Seeds of the cross Col x Col (3 and 6 DAP) were used as a control.  ","CATMA",10149,227,216,"2010-04-14","GEO","GSE29626","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (296,"P191E296","AFFY_cinetique_Lyon_Rose","time course","developmental stage","The objective here is to identify genes which expression associated different flower development stages from floral meristem to senescing flower. These genes are putative candidates involved in floral initiation, development and senescence. All samples were collected at the same time early in the afternoon. Meristems and early flower development stages were dissected under microscope. Total RNA was extracted from harvested tissues using the Plant RNA kit (Macherey Nagel), and then used to hybridize Rosa-Affymetrix microarrays.","Affymetrix",0,191,7,"2009-10-12","GEO","GSE18342","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (290,"P185E290","mutant vs wildtype","gene knock out","developmental stage","Ovules containing heart-stage/torpedo-stage embryos were dissected from siliques and imidiately shock frozen in liquid nitrogen. RNA was extracted and sent to Evry for comparison between mutant and wildtype transcriptome.","CATMA",10118,185,0,"2010-01-08","GEO","GSE19800","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (291,"P186E291","Analysis of deferoxamine treated leaves and roots","time course,treated vs untreated comparison","compound,time","Plants were allowed to grow for 5-6 weeks. The nutrient solution contains 0.25 mM Ca(NO3)2.4H2O, 1mM KH2PO4, 0.5 mM KNO3, 1mM MgSO4.7H2O, 50 µM H3BO3, 19 µM MnCl2.4H2O, 10 µM ZnCl2, 1 µM CuSO4.5H2O, 0.02 µM Na2MoO4.2H2O and 50 µM FeNa-EDTA. Plants were subjected to an 8 h light/16 h dark cycle, at 19°C, with 70% relative humidity. Leaves of six week old hydroponically grown A. thaliana Col0 plants were infiltrated with 1mM deferoxamine or sterile distilled water. Leaves were harvested 7 and 24 h.p.i. ","CATMA",10083,186,0,"2010-01-08","GEO","GSE18976","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (297,"P192E297","AFFY_duplicature_Lyon_Rose","genotype comparaison","genotype","In this study, we used a microarray approach to compare the transcriptome of double flower rose (CDR) versus simple flower rose (G). The objective is to identify genes which expression associated the double flower phenotype. These genes are putative candidates involved in the control of petal organ number per flower. Floral buds were dissected under microscope and pooled in eppendorf tubes. Tissues samples were harvested at the same time during 3 weeks in April 2007. Total RNA was extracted from the pools of flowers using the Plant RNA kit (Macherey Nagel), and then used to hybridize Rosa-Affymetrix microarrays.","Affymetrix",0,192,7,"2009-10-12","GEO","GSE18353","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (442,"P276E442","Knock Out Mutant N575260","gene knock out","genotype","Study of the biosynthesis of NAD in Arabidopsis. Involvment of QPRTgene using the knock out mutant N575260 from Salk.","CATMA",10215,276,256,"2011-02-04","GEO","GSE30976","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (116,"P44E116","PLP2 Paraquat","gene knock in (transgenic),time course","disease state,genotype","10 uM Paraquat was infiltated in mesophyll of leaves with a syringe without needle. Leaf material was harvested at 0, 4 and 18 hours later. 3 plant genotypes were used (Col-0 ecotype) :- siPLP2 (RNAi-silenced)- pBIN (empty pBIN-transformed)- PLP2OE (PLP2-overexpressors).","CATMA",288,44,0,"2009-04-29","GEO","GSE15882","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (289,"P184E289","AFFY_seed_kinetic_Wheat","time course","developmental stage","Study gene expression during the grain developmentalSample at 100 degree days, year 2004 and 2006Sample at 200 degree days, year 2004 and 2006Sample at 250 degree days, year 2004 and 2006Sample at 300 degree days, year 2004 and 2006Sample at 400 degree days, year 2004 and 2006","Affymetrix",0,184,7,"2009-10-12","GEO","GSE16457","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (202,"P139E202","Compare haiku mutants with wildtype","gene knock out","genotype","IKU1, IKU2 belong to same pathway which involved in seed development. Using array analysis try to find other components of this pathway.","CATMA",10010,139,0,"2009-07-06","GEO","GSE16956","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (305,"P198E305","AFFY_hypocTemp_Medicago","time course","genotype,temperature","We used Jemalong A17 and F83005.5 seeds coming from the Medicago truncatula Biological Resource Center in Montpellier, produced in 2006.Experiments were performed in the dark to mimic pre-emergence growth.Pots (6.5 cm diameter, 10 cm high) were incubated in growth chambers either at 10°C or 20°C. They were filled with 500g sand and 100ml of a nutrient solution for young seedlings growth (Saglio and Pradet, 1980).Per pot, five scarified seeds were sown at 1.5 cm depth.Seedlings were harvested at three times: 35°Cd (degree day), 50°Cd and 100°Cd.At each timepoint, about 50 seedlings were harvested and hypocotyl lengths was measured. Hypocotyl were cut and immediately frozen in liquid nitrogen for RNA extractions.","Affymetrix",0,198,7,"2009-10-12","GEO","GSE18449","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (306,"P199E306","AFFY_Ralstonia_Medicago","normal vs disease comparison,time course","disease state,genotype","Three Medicago truncatula lines, A17, F83005.5 and sickle were inoculated with GMI1000 Ralstonai solanacearum strain (107 cfu/ml). RNA were extracted from root extremities (1 cm above the root tip) at time 0, 12h and 72h post inoculation. Three biological repeats were conducted","Affymetrix",0,199,7,"2009-10-12","GEO","GSE18473","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (346,"P225E346","mpk6 flg22","treated vs untreated comparison","flg22","2 weeks mpk6 flg22 treated  1h with 1 microM flg22","CATMA",10145,225,224,"2010-03-25","GEO","GSE23193","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (283,"P179E283","AFFY_H2O2_Medicago","treated vs untreated comparison","inoculation","M. truncatula seedlings were grown in axenic condition on modified Fahraeus medium during seven days. Then they were transferred on new plates supplemented either with dimethyl sulfoxide (DMSO, mock treatement) or diphenylene iodonium (DPI, 10 µM). Twenty four hours later, they were inoculated either with water (mock inoculation; DMSO-H2O and DPI-H2O) or wild type S. meliloti 2011 strain (DMSO-INOC and DPI-INOC).Roots without apexes were harvested 48 hours after inoculation. After RNA extraction (Trizol), quality of the treatment was verified by RT-PCR (ENOD11: inoculation efficiency; GSHS1: DNA contamination; MTC27: constitutive)","Affymetrix",0,179,7,"2009-10-12","GEO","GSE15866","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (218,"P149E218","AFFY_POPSEC_Nancy_Leaves2007_poplar ","treated vs untreated comparison","dose,genotype,time","In a first experiment, two poplar clones, Soligo (S) and Carpacio (C) were submitted to 4 treatments: control, mild water deficit, moderate water deficit (12-day long for both) and early-drought stress (about 36-h long). Growth and physiology was characterised on a batch of plants and samples collected on another batch of plants. In a second experiment, two poplar clones, Soligo (S) and Carpacio (C) were submitted to 2 treatments: control and moderate water deficit.Mature leaves were collected and total RNAs were extracted from each tree individually. Two pools of 3 (or 2) individuals were made using equimolar ratio. A pool is considered as one biological replicate and corresponds to one Affimetrix slide.","Affymetrix",0,149,7,"2009-10-12","GEO","GSE17230","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (267,"P166E267","Comparison between constitutive and inducible p25 plants","gene knock in (transgenic)","age,genotype","Comparison between constitutive and inducible p25 plants.","CATMA",10054,166,0,"2010-01-11","GEO","GSE19827","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (194,"P133E194","AFFY_mildey_vitis","normal vs disease comparison","disease state,genotype","In order to limit the effect of the genetic background, the plant material consists in three pools of genotypes called B, C and D, respectively corresponding to partially resistant plants (Rpv1+/Rpv2-), totally resistant plants (Rpv1-/Rpv2+), and susceptible plants (Rpv1-/Rpv2-) derived from a segregating population. Each pool consists in 3 different genotypes. Plant leaf discs were inoculated with P. Viticola sporangium suspension (i) or mock-inoculated with water (ni) and analysed 6 hours post-inoculation. For each experimental condition, we performed two biological replicates.","Affymetrix",0,133,7,"2009-05-28","GEO","GSE11857","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (389,"P252E389","AFFY_Dormance_mRNAoxidized_Sunflower","dormancy","post transcriptional modifications : oxidation","The samples compared are the following:1) 8OHG containing mRNA isolated from total RNA extracted from a population of 100 embryonic axis of dry dormant seed (Dox) ie. Seed which have been frozen at -30°C after harvest to keep dormancy2) 8OHG containing mRNA isolated from total RNA extracted from a population of 50 embryonic axis from dry non dormant ie. Seeds which have been after-ripened (stored) at 70% RH during 2 month (NDox)Replicates correspond to different batches of seeds from the same harvest, which have been treated (after ripened) independently and from which the 8OHG containing fraction have been isolated independently.   ","Affymetrix",0,252,7,"2010-07-15","GEO","GSE23048","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (315,"P202E315","Ralstonia infection","treated vs untreated comparison","genetic background","Comparison between the mutant and the wt at different time after infection with ralstonia bacteria","CATMA",10108,202,14,"2009-10-30","GEO","GSE19081","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (286,"P149E286","AFFY_POPSEC_Nancy_Leaves2008_poplar ","treated vs untreated comparison","dose,genotype,organism part,tissue type","In a first experiment, two poplar clones, Soligo (S) and Carpacio (C) were submitted to 4 treatments: control, mild water deficit, moderate water deficit (12-day long for both) and early-drought stress (about 36-h long). Growth and physiology was characterised on a batch of plants and samples collected on another batch of plants. In a second experiment, two poplar clones, Soligo (S) and Carpacio (C) were submitted to 2 treatments: control and moderate water deficit.Mature leaves were collected and total RNAs were extracted from each tree individually. Two pools of 3 (or 2) individuals were made using equimolar ratio. A pool is considered as one biological replicate and corresponds to one Affimetrix slide. ","Affymetrix",0,149,7,"2009-10-12","GEO","GSE17226","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (364,"P242E364","evolution of genotypes and differences ","genotype and ecotype comparison,time course","genetic background,time","x","Affymetrix",10158,242,216,"2010-06-03","GEO","GSE31005","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (303,"P197E303","LST8","gene knock out","daylength conditions,genotype","Comparison between lst8 mutant vs WT in short day and in long day conditions.","CATMA",10097,197,0,"2009-09-16","GEO","GSE25731","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (203,"P140E203","CATMA1-bou","gene knock in (transgenic),normal vs disease comparison,time course","age,developmental stage,genotype,organism part","Seeds from a heterozygous plants were grown for either 5 or 8 days after germination on synthetic medium (MS/2) without sugar under continuous light.  We harvested cotyledon-stage blocked plants (bou phenotype) from three independent Petri dishes and also green seedlings with true leaves and fully developed root (heterozygotes with a wild-type phenotype) . We also grew independently Col-O plants for 5 and 8 days to compare them with the bou mutants.","CATMA",10011,140,0,"2009-09-07","GEO","GSE10399","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (391,"P254E391","AFFY_Mildew_Sunflower","normal vs disease comparison","disease state,genotype","Compare the pattern of expression of specific resistances harboured by the Pl5/Pl2 alleles and the quantitative resistance harboured by the favourable allele at the mapped QTL, during compatible and incompatible interactions in the pathosystem Helianthus annuus/ Plasmopara halstedii","Affymetrix",0,254,7,"2010-07-22","GEO","GSE25717","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (307,"P200E307","AFFY_MeJA_Arabidopsis","gene knock in (transgenic),treated vs untreated comparison","genotype,hormone treatment","Comparison of control lines and transgenic lines expressing C2 in basal conditions and after treatment with 50uM MeJA for 10 hours.--C2 lines are expressing geminivirus C2 under a 35S promoter. PR1-LUC havea luciferase transgene under the control of the promoter of PR1; since theplants are not induced, these are the control plants (resistant tokanamycin, but not expressing anything else)","Affymetrix",0,200,7,"2009-10-21","GEO","GSE18667","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (268,"P167E268","Clavata-Ralstonia","wt vs mutant comparison","genotype","3 biological repets","CATMA",10069,167,14,"2009-11-18","GEO","GSE18757","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (455,"P287E455","AFFY_virulent avirulent xoo strains_rice","avirulent vs virulent","strain","The goal of the experiment is to characterize the rice leaf transcriptome response, upon the inoculation of susceptible and resistant rice leaves 24 hours post-infection. To that end, the experimental design includes the inoculation of susceptible Nipponbare rice leaves with Xoo strains BAI3 (race A1) and MAI1 (race A3), that will be compared to the response of resistant lines IRBB4 and IR64 rice lines. In addition, Nipponbare rice leaves will also be challenged with the BAI3&#61508;hrcC mutant that is affected in the production of a functional TTSS.","Affymetrix",0,287,238,"2011-06-16","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (269,"P168E269","BAC2-CATMA","gene knock out,treated vs untreated comparison,wt vs mutant comparison","dose,genotype","Mutant and Wild-type plants were grown on grids  for 7 days on plates containing MS/2 and saccharose. Plants were transferred for 24h to liquid medium containing MS/2 saccharose (untreated plants) or 0.4M mannitol MS/2 saccharose (treated plants). ","CATMA",10101,168,216,"2010-01-12","GEO","GSE15063","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (285,"P181E285","AFFY_POPSEC_Nancy_POPHYDRO_Roots_poplar ","treated vs untreated comparison","dose,genotype,organism part,tissue type","Poplar cuttings (cv Soligo) were grown in hydroponics. A moderate water deficit was applied by adding PEG to the nutrient solution (200g/l). After 3 days, the apex (1 cm-long) of the roots of each cutting were collected. For control ans stressed treatments, RNAs were extracted from two pools of 5 to 9 roots (issued from 2 to 5 cuttings).A pool is considered as one biological replicate and corresponds to one Affimetrix slide","Affymetrix",0,181,7,"2009-10-12","GEO","GSE17225","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (288,"P183E288","photomorphogenesis","treatment variations","light","etiolated seedlings (5days) are briefly exposed to light to induce developmental transition between skoto- and photomorphogenesis without visible change in plant morphologyand differences between col0 and the mutant det1-1 grown in light or in dark","CATMA",10080,183,216,"2010-01-12","GEO","GSE19843","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (337,"P220E337","AU10-01_overexpressionoxi1","normal vs transgenic comparaison","genotype","Comparision between the overexpression of oxi1 and wt.","CATMA",10138,220,224,"2010-03-22","GEO","GSE35941","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (25,"P18E25","comparison_Col0vsSKP2","wt vs mutant comparison","genotype","Comparison Col0 versus skp2","CATMA",356,18,167,"2009-06-29","GEO","GSE36651","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (230,"P157E230","HEXA","genotype and ecotype comparison","genotype","transcriptomic changes in synthetic hexaploid derived from a cross between tetraploid and natural diploid","Affymetrix",0,157,136,"2008-07-21","GEO","GSE20169","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (489,"P318E489","AtBAF60","normal vs rnai mutant comparaison,normal vs transgenic comparaison","genotype","Transcriptome of lines over-expressing and down-expressing AtBAF60","NimbleGen",10276,318,0,"2012-07-27","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (515,"P344E515","INRA12-05_mut_flg_II","gene knock out,normal vs mutant comparison,treated vs untreated comparison"," compound,genotype","2 weeks old seedlings compared for transcriptome after a 30min mock or 1µM flg22 treatment.","NimbleGen",0,344,335,"2013-04-16","GEO","GSE52587","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (209,"P143E209","Xyleme","genotype and ecotype comparison,organ comparison,treated vs untreated comparison","dose,genotype,tissue type","Two poplar clones, named Soligo (S) and Carpacio (C) were submitted to 4 treatments: non-stressed control, severe-drought stress, mild-drought stress and early-drought stress. Two pools of 2 or 3 trees were considered as biological replicates.Two different samples were collected on each individual tree: whole stem xylem (X) and cambial zone plus very young expanding xylem (Y). Total RNAs were extracted from each tree and assemble in pool of 3 or 2 individuals using equimolar ratio for each X and for Y. One affymetrix slide corresponds to one pooled RNA sample (X or Y). A total number of slides : 4 treatments x 2 clones x 2 tissues x 2 biological replicates (pool) = 32 slides, will be done. ","Affymetrix",0,143,7,"2009-10-12","GEO","GSE17220","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (309,"P202E309","mutant comparison","wt vs mutant comparison","genotype","wt versus mutant comparison in the leaf and the root","CATMA",10105,202,14,"2009-11-18","GEO","GSE19077","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (287,"P182E287","Laccases","wt vs mutant comparison","genotype","all plants were harveted at the same developmental stage in the same growth chamber between 10h30 and 11h.","CATMA",10150,182,216,"2009-05-06","GEO","GSE21388","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (311,"P204E311","HB6","wt vs mutant comparison","genetic background","Transcriptome on Arabidopsis HB6 over-expressing plants and BPM Knock down plants during development","CATMA",10111,204,14,"2009-11-18","GEO","GSE19095","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (62,"P43E62","Profiling of the root architecture response to Phyllobacterium","treated vs untreated comparison","none","Seeds of wild-type Arabidopsis thaliana (ecotype Columbia) were surface-sterilized and sawn on agar mineral medium (see below). 4 days after storage in the dark at 4°C, seedling were cultivated 6 days in a growth chamber (16 h daily, 20-22°C) and then transferred on a fresh agar mineral medium inoculated or not with Phyllobacterium STM196 (2.108 cfu/ml). 6 days later, root and leaves were collected, froze on liquid nitrogen and stored at -80°C.","CATMA",332,43,5,"2006-12-20","GEO","GSE7991","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (271,"P170E271","knockout","gene knock out","genotype","transcriptome analysis in leaf of tctp-2 knock out versus wild type","CATMA",10133,170,216,"2010-03-17","GEO","GSE19847","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (273,"P170E273","Over Expression","gene knock in (transgenic)","genotype","transcriptome analysis in leaf of 35S::AtTCTP versus wild type","CATMA",10132,170,216,"2010-03-17","GEO","GSE19848","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (506,"P335E506","INRA12-01_mut-flg","gene knock in (transgenic)"," compound,genotype","2 weeks seedlings compared for their transcriptome after a 30min mock or 1µM flg22 treatment. ","NimbleGen",0,335,335,"2013-03-22","GEO","GSE46152","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (75,"P49E75","1mM NO3","wt vs mutant comparison","compound","plants were cultivated in hydroponie condition (light 16h / dark 8h, temperature 22°C)in 3 nitrogen medium conditions : 1mM NO3, nitrogen deficient medium (5 days) and then feed in 1mM or 4mM NH4. After 6 weeks of culture, leaves from 3 plants (6.20-6.30 boyes stage) were sampled for each treatment then pooled.","CATMA",10136,49,14,"2010-03-19","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (272,"P170E272","RNAi","normal vs rnai mutant comparaison","genotype","transcriptome analysis in leaf of 35S::RNAi-AtTCTP versus wild type","CATMA",10131,170,216,"2010-03-17","GEO","GSE19849","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (333,"P16E333","TOR_RNAi","wt vs mutant comparison","genotype","Two RNAi mutants are compared to the wild type.","CATMA",10130,16,1,"2010-02-10","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (64,"P43E64","Trancriptional response to 3 rhizobacteria","treated vs untreated comparison","none","Seeds of wild-type Arabidopsis thaliana (ecotype Columbia) were surface-sterilized and sawn on agar mineral medium. Four days after storage in the dark at 4°C, seedlings were cultivated 6 days in a growth chamber (16 h daily, 20-22°C) and then transferred on soil inoculated or not with 108 cfu.g-1 of Mesorhizobium loti, or 108 cfu.g-1 of Phyllobacterium STM196, or 107 cfu.g-1 of Bradyrhizobium ORS278.","CATMA",332,43,5,"2006-12-20","None","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (183,"P124E183","Gene profiling of viral siRNA in mutants at generation G1.","gene knock out,treated vs untreated comparison","genotype","Col-0 and dcl2/3/4 were grown until rosette state and inoculated with TCV. Ten days post-inoculation the leaves were harvested. The same was made at generation 1 (G1).","CATMA",10228,124,0,"2007-08-13","GEO","GSE29387","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (313,"P206E313","AFFY_TJ_Maize","genotype comparaison","developmental stage,genotype","Comparison of 2 maize lines : Cm484: good digestibilityF98902: bad digestibilityat the following development stage : 21 days after sowing -7 leaves-","Affymetrix",0,206,7,"2009-10-29","GEO","GSE18846","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (363,"P174E363","WT Dex Induction","treated vs untreated comparison","dose,time","Induction of 14 days old WT seedlings , different treatments ( EtOH, DEX  induction) for 8 hours","CATMA",10261,174,7,"2010-05-31","GEO","GSE36628","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (326,"P212E326","AFFY_pathogen_Medicago","organ comparison","tissue type","Parallel laser microdissection of giant cells from galls and cells from zone II of nodules were performed. The RNA pools were extracted from these cells, amplified and used for transcriptomic studies with M. truncatula Affymetrix DNA chips.","Affymetrix",0,212,7,"2010-01-12","GEO","GSE19872","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (256,"P159E256","Glutathion","gene knock out","genotype","Three single Arabidopsis mutants were used: cat2, knockout for catalase2 and so enriched in H2O2; cad2, defective in glutathione content; cytGR, knockout for cytosolic glutathione reductase. Cat2 was crossed with cad2 and cyt GR, and col0, 3 single mutants, and 2 double mutants were sampled in controlled growth conditions either in 8h or 16h photoperiod.","CATMA",10043,159,0,"2009-09-24","GEO","GSE21424","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (358,"P236E358","wt salt","treated vs untreated comparison","nacl","2 weeks seedligs were treated with 150 mM NaCl during 1h.","CATMA",10156,236,224,"2010-05-13","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (388,"P252E388","AFFY_Dormance_totalRNA_Sunflower","dormant vs non dormant seed embryo","storage protocol","The samples compared are the following:1) Population of 50 embryonic axis from dry dormant seed (D) ie. Seed which have been frozen at -30°C after harvest to keep dormancy2) Population of 50 embryonic axis from dry non dormant ie. Seeds which have been after-ripened (stored) at 70% RH during 2 month.3) Embryonic axis from dry dormant seed but after ripened at very low relative humidity (2% RH) (control of storage: S).Replicates correspond to different batches of seeds from the same harvest which have been treated (after ripened) independently. - We want to compare D vs ND; D vs S and ND vs S.","Affymetrix",0,252,7,"2010-07-15","GEO","GSE23046","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (357,"P235E357","AFFY_ZeaWall_RIL_Maize","genotype comparaison,time course","developmental stage,genotype","Five lines chosen for their allelic pattern at four main QTL were considered for expression studies, two lines with all four favorable alleles (RIL99, RIL54), two lines with the unfavorable allele at bin 6.06 QTL position (RIL39 , RIL71), and one line with the unfavorable allele at bin 9.02 QTL position (RIL118).Expression study will be investigated in ear internode at two stages emerging tassel and early silking stages with two replicates. Plants were cropped at INRA Lusignan (Vienne, France) in 2009 in a bloc design with two four-row replicates. Below-ear internodes of five representative RIL plants of one row in each replicate were harvested at the two stages and pooled. ","Affymetrix",0,235,7,"2010-05-12","GEO","GSE22439","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (340,"P223E340","DELLA-regulation in darkness versus light","gene knock out","light","Aim was to determine downstream target of DELLA proteins involved in skotomorphogenesis. Wt, ga1-3, ga1-3 global seeds were sterilized, sown on MS agar plates then put for stratification for 3 days at 4°C. Plates were placed in growth cabinet at 22°C for 5 days in darkness or in continous light.","CATMA",10171,223,5,"2010-03-16","GEO","GSE22681","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (370,"P246E370","Fireman vs WT Erwinia infection","normal vs disease comparison,normal vs transgenic comparaison","genotype,inoculation","3 inoculated leaves of 5 weeks old plants, per plant, have been  harvested 24h after inoculation with H20 or E. amylovora, strain 1430.","CATMA",10172,246,14,"2010-07-02","GEO","GSE22683","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (275,"P172E275","AU08-06_mpk6_heat_stressed","treated vs untreated comparison","temperature","Mitogen Activated Protein Kinase (MAPK) signaling pathways are key regulators of cell proliferation, differentiation and stress effectors. The core of the MAP kinase signal transduction cascade is composed of a three-kinase module consisting of a MAP kinase kinase kinase (MAPKKK), a MAP kinase kinase (MAPKK), and a MAP kinase (MAPK). The signaling pathway is activated upon stimulation by a phosphorylation cascade.In previous studies, it was shown that the mpk6 KO mutant plants are significantly more tolerant to heat stress in comparison to wt and that after 3h treatment at 37°C, an activation of heat-shock proteins occures in the mpk6 mutant.To better understand the changes occuring in the mpk6 mutant upon heat stress at the gene expression level, we would like to perform a microarray transcriptomic analysis.","CATMA",10063,172,202,"2009-03-10","GEO","GSE22568","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (390,"P253E390","AFFY_L2BCTV_Ath","gene knock in (transgenic)","genotype","10 days old plants were grown in MS at 24 degrees in long day. We used the different lines: wt, L2.4 mutants and L2.5 mutants. We performed three repetitions each.","Affymetrix",0,253,238,"2010-07-20","GEO","GSE24214","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (361,"P238E361","AFFY_syringae_Malaga_Athaliana","gene knock out,normal vs antisens mutant comparison","disease state,genotype","- We infiltrated Arabidopsis thaliana ecotype Columbia wild type with 10mM MgCl2 (as mock inoculation), the virulent pathogen Pseudomonas syringae pv. tomato (Pto), Pto expressing the effector HopZ1a and Pto expressing a HopZ1a derivative carrying a point mutation in a residue essential for the cystein-protease activity. We also grew ulp1c/ulp1d mutant plants to analyze them either without treatment or inoculated with MgCl2 (as mock) and Pto wt.  ","Affymetrix",0,238,238,"2010-05-18","GEO","GSE21920","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (453,"P285E453","AFFY_Compartimentation_Rice_Albumen_Embryon","organ comparison,time course","time,tissue type","We germinate rice seeds of the first sequenced rice cultivar i.e. Nipponbare during 0, 4, 8, 12, 16 and 24h of imbibition in sterile distilled water. Germination occurs under constant air bubbling, in the dark at 30°C. These rice seeds are then manually dissected into embryo and endosperm fractions.-The embryo-derived samples are abbreviated in E while the endosperm samples are abbreviated A. The germination time-point is indicated after the letter (e.g. E8 for embryo samples harvested after 8 hours of germination). Finally, the biological repetition number is indicated before the letter and the time digit (e.g. 1-E8 for an embryo sample from the first repetition at 8 hours of imbibition).","Affymetrix",0,285,238,"2011-05-09","GEO","GSE43780","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (100,"P66E100","Gene profiling in silencing suppressor plants or in miRNA mutants","gene knock in (transgenic),gene knock out","genotype","HcPro, P15,  P19, CHS-RNAi (control), were grown on MS solid medium, and tissues were harvested at different developmental stages.dcl1-9, hen1-1 and La-er (control) were grown on MS solid medium, and tissues harvested at different developmental stages. ","CATMA",10281,66,5,"2006-12-19","GEO","GSE24693","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (396,"P256E396","WIP overexpressing plants","gene knock in (transgenic),normal vs transgenic comparaison","gene overexpression","The objective is to identify genes responsible for the phenotypes observed in WIP overexpressing lines. WIP transcription factors are involved in reproductive and vegetative development by acting on the remodeling of various tissues. The WIP related processes have a major impact, in particular on reproduction (sex determination, fertilization,  development of the seed). The way of action of these transcription factors is probably conserved, their effect depending only on their specificity of expression. Two WIP genes, NTT and TT1, was overexpressed in Arabidopsis thaliana. The objective is to compare the common deregulated genes (and in particular repressed) in the transcriptome of these two overexpressor lines, to highlight their mode of action and the way they act on the tissue in which they are expressed.","CATMA",10187,256,65,"2010-08-26","GEO","GSE24019","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (325,"P211E325","Groundworms","treated vs untreated comparison","none","Plants were grown in a sandy soil from the Centre de Recherche en Ecologie Expérimentale et Prédictive (CEREEP, France), in the presence or absence of earthworms. Soil was dried and sieved at 2 mm before to be put in the pots. There was 2 levels of treatment (presence or absence of earthworms in the soil), and 5 plants per treatments.","CATMA",10122,211,0,"2010-01-12","GEO","GSE24393","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (328,"P214E328","WOX13 orthologous group","gene knock out","developmental stage,genotype","Transcriptome analyses of the transposon donor line N8514 and the wox14 mutant seedlings grown for 6 and 10 days (growth stages 1.02 and 1.04) in LD condition (i. e. 4 days of cold treatment on 0.5 MS medium, 1% sucrose, 0.7% agar + respectively 6 and 10 days of in vitro growth in LD condition).","CATMA",10127,214,0,"2010-01-18","GEO","GSE24394","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (425,"P261E425","AFFY_Sunyfuel_drought_Sunflower","treated vs untreated comparison","water input","Ninety six individual pots were arranged in six blocks of 16 plants each, which were sown in individual pots. Each block contained 2 plants of 8 different genotypes, one of which was used as control and the other one as treated plant. Plants within each block were randomly arranged.At the right developmental stage, treated plants were water-deprived and their water status was monitored as compared to the untreated control. Three out of the six blocks were harvested on a given date, so water stress among genotypes varied. The plants in the other three blocks were harvested when the treated plant reached a given water stress.","Affymetrix",0,261,7,"2010-10-01","GEO","GSE25719","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (449,"P280E449","Gene profiling of Turnip Crinkle Virus (TCV) siRNA","gene knock out,treated vs untreated comparison","genotype","This is a plant evolution project on TCV in which, the biological questions are:1. What are the genes (including miRNA precursors) that are differentially regulated in Col0 and dcl234 mutant in wt conditions?2. What are the genes (including miRNA precursors) that are differentially regulated in a set of viral siRNA in Col0 and dcl234 mutant? 3. There are also differentially regulated between the plant generations 1 (G1) and 11 (G11)?4. What are the genes (including miRNA precursors) that are differentially regulated in a set of viral siRNA after fitness experiment?","CATMA",10224,280,0,"2011-04-19","GEO","GSE29732","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (97,"P64E97","Time course analysis of atrazine and sucrose effects","time course,treated vs untreated comparison","compound,time","Seeds of Arabidopsis thaliana (ecotype Columbia) were surface-sterilized inbayrochlore/ethanol (1:1, v/v), rinsed in absolute ethanol and dried overnight. Germination and growth were carried out under axenic conditions in square Petri dishes. After seeds were sowed, Petri dishes were placed at 4°C for 48 h in order to break dormancy and homogenize germination, and then transferred to a control growth chamber at 22°C under a 16-h light period regime at 4500 lux for 4 weeks. Growth medium consisted of 0.8% (w/v) agar in 1x Murashige and Skoog (MS) basal salt mix (Sigma, St. Louis, MO, USA) adjusted to pH 5.7. After 4 weeks of cultivation on vertical plates plantlets were transferred on fresh medium complemented with atrazine 10 uM and with sucrose 80mM or mannitol 80 mM which were directly added during preparation of agar-MS media prior to sterilisation. Atrazine was sterilized by microfiltration through 0.2 um cellulose acetate filters (Polylabo, Strasbourg, France) and then axenically added to melted agar-MS medium prior to pouring into Petri dishes. Then plantlets were harvested after 12, 24 and 48 hours of transfer and extracted for RNA. Plantlets grown on MS medium were harvested and extracted for RNA as control.","CATMA",367,64,0,"2008-10-03","ArrayExpress","E-MEXP-574","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (53,"P37E53","dry seeds","treated vs untreated comparison","seed imbibition","Freshly harvested C10 seeds (obtained from mother plants grown with 10 mM nitrate in growth chambers) were kept dry (C10dry control) or were sown on 0.5% Litex agarose plates and imbibed for 6 hrs in a culture chamber (at 25°C, 100uE m-2 sec-1 at 70% relative humidity, C10H20 seeds)","CATMA",329,37,5,"2009-04-03","ArrayExpress","E-MEXP-447","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (433,"P266E433","Transcripomics on the Arabidopsis Root tip during phosphate starvation","time course,treated vs untreated comparison"," compound,time","The objective of this project is to identify genes that are expressed in the Arabidopsis thaliana root tip and that are early induced  (or repressed) by phoshate deficiency. Seedlings were germinated and grown (for 6 days) on phosphate rich medium and transfered to either a phosphate poor medium (Pi- = 20&#956;M Pi treatment) or a phosphate rich medium (Pi+ = 500&#956;M Pi = control). Fifteen and 60 minutes after  transfer, the tip (&#8776; 800&#956;m) of the primary root was cut under a dissecting microscope. About 100 root tips per condition were harvested and there mRNA was analysed with the use of microarrays.","CATMA",10204,266,256,"2010-11-30","GEO","GSE31135","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (437,"P270E437","Induction of DspA/E expression in a transgenic A. thaliana line","gene knock out,genotype comparaison,treated vs untreated comparison","dose,genotype","Transgenic lines expressing a GFP::DspA/E fusion or GFP alone were treated with estradiol or with DMSO. Sample were collected 6 hours following treatment.","CATMA",10208,270,0,"2010-12-17","GEO","GSE33046","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (354,"P232E354","Quadruple RRM experience","normal vs transgenic comparaison","none","Plants were grown on soil in controlled environment under LD (16 h light/8 h dark) and the rosette leaves (8-leaf stage seedlings) were collected for RNA preparation.","CATMA",10225,232,256,"2010-05-05","GEO","GSE21701","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (323,"P209E323","GEMSUPV2","gene knock in (transgenic)","genotype","One transgenic line of Arabidopsis overexpressing the geminiviral silencing suppressor V2 was grown in adition to a control and their total RNA was extracted in order to study its genetic expression profile.","CATMA",10121,209,0,"2010-01-12","GEO","GSE26399","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (451,"P282E451","cell wall mutants ","gene knock in (transgenic),normal vs transgenic comparaison","genotype","Determine differentially expressed genes in stems of Arabidopsis plants lacking 2 monolignol exporter proteins. mRNA from stems of wild-type and transgenic 7-week-old plants grown in the same conditions were extracted and used for transcriptomic analysis. Three independant cultures were conducted for biological replicates.Determine differentially expressed genes in stems of Arabidopsis plants that express a hydroxycinnamoyl-CoA hydratase/lyase. mRNA from stems of wild-type and transgenic seven-week-old plants grown in the same conditions were extracted and used for transcriptomic analysis. Three independant cultures were conducted for biological replicates.","CATMA",10226,282,0,"2011-05-04","GEO","GSE31975","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (439,"P272E439","Response of ein3eil1 mutants to Fe deficiency","gene knock out,treated vs untreated comparison","genotype,nutrition factor","Wild type seedlings and ethylene insensitive ein3eil1 seedlings were germinated and grown in the presence of 50 µM Fe or absence of Fe (0 µM) on Hoagland medium agar plates until the age of 6 days. Under these growth conditions symptoms of Fe deficiency develop in the 0 Fe plants. Ethylene is known to promote Fe acquisition responses.  Whole seedlings were harvested for transcriptome analysis, in a total of three biological replicates.","CATMA",10210,272,256,"2010-12-27","GEO","GSE26510","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (229,"P153E229","mkk1-mkk2","gene knock out","genotype","mkk1 and mkk2 k.o. plants were crossed. Seedlings homozygous for mkk1 and/or mkk2, in parallel with control wild type seedlings, were grown on sterile medium. RNA was extracted from non-treated seedlings. aRNA was synthesised using the Ambion aRNA kit. Transcriptome data will be compared with those from a (already available) microarray of a mutant lacking the MAPKKkinase that is upstream of MKK1 and MKK2 (MEKK1). ","CATMA",10038,153,1,"2008-09-25","GEO","GSE24516","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (263,"P162E263","Trancript profiling in Arabidopsis GOLLUM mutant.","wt vs mutant comparison","genotype","Seeds of WT and RNAi lines were sowed on soil and conserved 2 days at 4°C. The pots were transferred to the greenhouse under long-day conditions (16h light, 22°C). The leaves of one-month old plants were harvested and total RNA were extracted.","CATMA",10221,162,14,"2008-10-31","GEO","GSE13417","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (441,"P275E441","AFFY_ZeaWall_RIL2_Maize","time course","genotype","Five lines chosen for their allelic pattern at four main QTL were considered for expression studies, two lines with all four favorable alleles (RIL99, RIL54), two lines with the unfavorable allele at bin 6.06 QTL position (RIL39 , RIL71), and one line with the unfavorable allele at bin 9.02 QTL position (RIL118). The defavorable parent F271 was also studied in complement and is the object of this last array.Expression study will be investigated in ear internode at two stages emerging tassel and early silking stages with two replicates. Plants were cropped at INRA Lusignan (Vienne, France) in 2009 in a bloc design with two four-row replicates. Below-ear internodes of five representative RIL plants of one row in each replicate were harvested at the two stages and pooled. ","Affymetrix",0,275,238,"2011-02-03","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (329,"P215E329","wt vs mutants 09"," fwa-d,gain of fuction epimutation,gene knock in (transgenic),gene knock out","genotype","Determination of the genes misregulated upon ectopic expression of the FWA imprinted gene.Mutants impaired for the DNA methylation maintenance pathway and RNA interference machinery were compared to mutants simultenously impaired for both.Impact of FWA and SDC ectopic expression.","CATMA",10213,215,0,"2010-01-19","GEO","GSE26844","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (369,"P245E369","GEMSUPV2TS-7","gene knock in (transgenic)","genotype","One transgenic line of Arabidopsis overexpressing the geminiviral silencing suppressor V2TS was grown in adition to a control and their total RNA was extracted in order to study its genetic expression profile.","CATMA",10169,245,0,"2010-06-28","GEO","GSE30766","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (432,"P265E432","Oligosaccharide treatment - 24 hpt","treated vs untreated comparison","oligosaccharide treatment","6,5-week-old arabidopsis plants grown in short-day photoperiod were spray-treated with water, an adjuvant or oligosaccharides in the adjuvant (10 plants per condition, 4 replicates per condition). 6 hrs, 24 hrs and 48 hrs after the treatment, samples were taken (2 7mm diameter leaf disks per plant, from medium-aged leaves), frozen in liquid nitrogen, and later on total RNA was extracted.","CATMA",10203,265,256,"2010-11-29","GEO","GSE30983","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (406,"P259E406","Comparison of IND induced vs mock treated seedlings","induced vs non-induced comparison","presence of inducer","Transgenic seeds containing an IND transgene under the control of a dexamethasone inducible promoter were germinated and grown in liquid Murashige/Skoog medium under a 16h light/8h darkness light regime. After seven days of growth, seedlings were either mock-treated or treated with 10 micromolar dexamethasone for 24 hours.","CATMA",10227,259,256,"2010-09-24","GEO","GSE28898","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (459,"P291E459","defenses mechanism in response to cold stress","treated vs untreated comparison","genotype,temperature","Plantlets of wild type and mutant were grown vitro for 14 days in a growth chamber (80% humidity, 80 micro Enstein without photoperiod). The 15th day, plantlets were transferred at 4°C or 12°C during 4 hours (80 micro Enstein) for the cold stress.","CATMA",10238,291,216,"2011-07-05","GEO","GSE36624","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (465,"P295E465","BF_vs_F","organ comparison","organism part","The two selected samples are the flower buds and leaves of Arabidopsis. Two organs known as the transcriptome is very different. This experience will serve as reference","NimbleGen",0,295,7,"2011-08-04","GEO","GSE45345","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (450,"P281E450","AFFY_MolecularMechanism_Medicago","resistance  vs susceptibility comparison,time course,treated vs untreated comparison","disease state,genotype,time","-Fifteen-day-old 5B2 or 5D2  plants were grown in vitro on M medium. The entire roots were then inoculated (or not = 0dpi) with a A. euteiches zoospores (75000zsp/ml) and harvested one or six days after inoculation. Three independent repeats were performed.","Affymetrix",0,281,238,"2011-04-29","GEO","GSE31198","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (318,"P207E318","Cordycepin","treatment variations"," compound","To evaluate the importance of mRNA stability control in glucose signaling, genome wide RNA transcript levels changes will be evaluated in response to short term treatments with glucose or glucose plus abscisic acid (ABA) under conditions of transcriptional inhibition in Arabidopsis. The underlying mechanisms will be investigated for a selected number of co-regulated genes (transcription factors, kinases).","CATMA",10112,207,14,"2009-11-25","GEO","GSE26045","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (255,"P125E255","Multiplexage : dual-target hybridization microarray experiment","genotype and ecotype comparison","genotype,time","Dual-target hybridization microarray experiment with doble factors comparison : two mutants and its wild-types at different times.The aim of this part is  to compare the classical design with the new design with a third dye.","CATMA",10115,284,65,"2009-11-29","GEO","GSE29104","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (461,"P242E461","differences between meristemoid cells and entire callus","tissue comparison","tissue type","Comparison of meristemoid cell cluster in B2 cotton (Gossypium hirsutum) calli of 64 days versus entire 21 days (non meristemoid) B2 calli","Affymetrix",10241,242,216,"2011-07-05","GEO","GSE31034","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (300,"P195E300","RILKIT-eQTL analysis (random pair design)","genotype and ecotype comparison","genotype","Comparison of 164 Cvi/Col(8RV) Recombinant Inbred Lines (RILs) and 164 Bur/Col (20RV) RILs ","CATMA",10091,195,174,"2009-09-08","GEO","GSE28791","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (446,"P278E446","CUC2","tissue comparison,wt vs mutant comparison","genotype,tissue type","To compare wt and cuc2 leaf margins. And compare teeth and hollow inside the arabidopsis leaf margin.","CATMA",10220,278,216,"2011-03-29","GEO","GSE30260","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (316,"P207E316","BZIP9","genotype comparaison","gene induction","To elucidate the role of selected bZIP transcription factors (AtbZIP3, AtbZIP9, AtbZIP63; the expression of these genes is repressed by glucose and are poorly characterized) involved in glucose-triggered transcriptional cascade, the target genes of these bZIP TF will be identified. The glucose-mediated regulations of these bZIP genes will be investigated by identifying the cis-regulatory sequences and the upstream regulatory factors. The functional conservation of the corresponding bZIP orthologues in sugar cane and rice will be evaluated.","CATMA",10114,207,216,"2009-11-26","GEO","GSE26041","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (460,"P292E460","transdifferenciation","time course","hormone treatment,time","culture in middle with different hormons, permits transdifferenciation from root to shoot tissues.","CATMA",10240,292,216,"2011-07-05","GEO","GSE30515","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (40,"P28E40","transcriptome profiling from a protoplast culture of arabidopsis thaliana","time course","time","protoplasts were extracted from plantlet cultivated during 2 weeks then placed in a culture midium which stimulate cell division. Protoplasts were harvested after 0, 24, 48, 96 or 168 hours of culture. Biological repeat has been done (experiments A and C)","CATMA",324,28,5,"2006-12-20","GEO","GSE7984","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (471,"P301E471","Phosphatidic acid in gene expression","treated vs untreated comparison"," compound","Compounds were added 4h before cell harvesting.","CATMA",10250,301,0,"2011-11-17","GEO","GSE35872","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (429,"P262E429","cdka mutants","wt vs mutant comparison","genotype","compare wild type plants with mutant plants that show a reduced activity of CDKA1, the key cell cycle regulator in Arabidopsis thaliana","CATMA",10242,262,216,"2010-10-06","GEO","GSE36275","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (152,"P100E152","CaMV treatment","gene knock out,normal vs disease comparison","cell line","4/5 weeks old seedlings (Col-0, dcl2/dcl3, dcl2/dcl4, dcl3/dcl4, dcl2/dcl3/dcl4) infected with CaMV and harvested 21 days post infection.","CATMA",380,100,5,"2006-12-20","GEO","GSE36457","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (447,"P284E447","Nitrogen starvation ","time course,tissue comparison","sources of nitrogen,time","Dual-target hybridization microarray experiment.","CATMA",10259,284,65,"2011-04-13","GEO","GSE29103","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (401,"P258E401","AFFY_Cellwall_Maize","organ comparison","cell line","Comparison of internodes from 5 maize lines grown in field conditions","Affymetrix",0,258,238,"2010-09-14","GEO","GSE24624","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (458,"P290E458","Action of microgravity on root development","treated vs untreated comparison","null","Arabidopsis were grown on horizontal or vertical clinostat for 4, 8 or 12 days. Seedlings on horizontal clinostat were in simulated microgravity and seedlings on vertical clinostat are considered as a control. Comparison was made between plants grown on simulated microgravitry and vertical position.","CATMA",10237,290,0,"2011-07-01","GEO","GSE39631","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (470,"P300E470","Time course expression profile of ap1-cal after induction of an AP1-transgene","time course","developmental stage,time","An ap1-cal double mutants carrying an Glucocorticoid-receptor fusion to AP1  was indcued and samples were collected 2 days after induction (dai), 8dapi, 9dai, 10dai and 11dai. Then, the expression profile of 2dai were compared with 8dai, 8 with 9, 9 with 10, and 10 with 11.","CATMA",10249,300,0,"2011-11-09","GEO","GSE43646","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (368,"P244E368","microdissection vs no microdissection","treated vs untreated comparison","microdissection","To compare RNA from flowers and RNA from microdissected flowers to know bias induce by microdissection (dissection from all tissues were collected to have a representation of entire flower)","CATMA",10290,244,216,"2010-06-21","GEO","GSE30265","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (454,"P286E454","Impact of sucrose on the tolerance of phenanthrene","treated vs untreated comparison"," compound,time","We test 3 conditions plants non-treated, plants treated with phenanthrene and plants tread with phenanthrene and sucrose. The plants were grown on MS/2 media for 17 days and then transferred on the corresponding condition. We took a sample of 30 plants at different times (0, 30 min, 2h, 4h, 8h and 24h).","CATMA",10234,286,216,"2011-05-23","GEO","GSE48181","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (438,"P271E438","Photooxidative stress ","genotype comparaison,treated vs untreated comparison","light","The extracts are carried out starting from sheets of Arabidopis thaliana having pushed on compost in controlled conditions (light: 250 µmol.m-2.s-1, temperature: 22°C day 18°C night, moisture: 55%, photoperiod: 8:00 jour/16h night) during 4 and 8 weeks for respectively the Col0 genotype and the mutant ch1. The acclimatization of the plants is done during 48 H under an average luminous intensity (450 µmol.m-2.s-1, photoperiod 8:00), whereas the stress requires a strong luminous intensity (900 µmol.m-2.s-1, photoperiod 8:00) and low temperature (10°C day, 14°C night) also during 48 hours.","CATMA",10209,271,0,"2010-12-22","GEO","GSE34247","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (394,"P225E394","Bak1Bkk1","gene knock out","genotype","comparision between bak1bkk1 and col-0.","CATMA",10253,225,224,"2010-07-26","GEO","GSE23185","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (443,"P277E443","microdissection","time course"," compound,time","middle of growth permits to induce transdifferentiation of root apical meristem to shoot apical meristem","CATMA",10217,277,216,"2011-02-07","GEO","GSE30259","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (276,"P173E276","mpk_flagellin","treated vs untreated comparison","stress","Mitogen Activated Protein Kinase (MAPK) signaling pathways are key regulators of cell proliferation, differentiation and stress effectors. The core of the MAP kinase signal transduction cascade is composed of a three-kinase module consisting of a MAP kinase kinase kinase (MAPKKK), a MAP kinase kinase (MAPKK), and a MAP kinase (MAPK). The signaling pathway is activated upon stimulation by a phosphorylation cascade.Flagellin is a protein present in the flagellum of almost all bacteria and triggers Innate Immunity in plants, mediated by a MAPK phosphorylation signal cascade.To better understand the changes occuring in the mpk3, mpk4, mpk6 mutants upon flagellin treatment at the gene expression level, we would like to perform a microarray transcriptomic analysis.","CATMA",10064,173,202,"2009-03-11","GEO","GSE34659","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (277,"P173E277","wt-Col0_flagellin","treated vs untreated comparison","stress","Mitogen Activated Protein Kinase (MAPK) signaling pathways are key regulators of cell proliferation, differentiation and stress effectors. The core of the MAP kinase signal transduction cascade is composed of a three-kinase module consisting of a MAP kinase kinase kinase (MAPKKK), a MAP kinase kinase (MAPKK), and a MAP kinase (MAPK). The signaling pathway is activated upon stimulation by a phosphorylation cascade.Flagellin is a protein present in the flagellum of almost all bacteria and triggers Innate Immunity in plants, mediated by a MAPK phosphorylation signal cascade.To better understand the changes occuring in wt-Col0 flagellin treatment at the gene expression level, we would like to perform a microarray transcriptomic analysis.","CATMA",10065,173,202,"2009-03-11","GEO","GSE34659","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (448,"P279E448","WS vs Clca2/35A2/352.7","gene knock out,genotype comparaison,normal vs transgenic comparaison","genotype","Plants were grown for 12 days after germination in vertical Petri dishes onto a 2mM NO3 medium (derived from Armengaud et al 2004) in long days (16h light at 65 µE). Samples consisted of pooled roots from at least 30 to 50 plants per sample. The experiment has been repeated 4 times (exp IFX1-IFX3-CPLA2-CPLA3)and samples were harvested around 11h in the morning (ie 3h after illumation).","CATMA",10223,279,0,"2011-04-19","GEO","GSE35870","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (331,"P216E331","2 doses of ionising radiations (X-rays) ","dose response,time course","dose,time","Arabidopsis thaliana (Col-0) seeds were grown in Petri dishes under sterile conditions until they had 2-rosette-leaves (on average 10 days). Then they received a dose of 10 or 40 gray of X-ray (Faxitron HP). Plant were harvested 2 and 26 hours after irradiation, immediatly frozen and stored at -80°C until RNA extraction (Rneasy plant mini Kit, Qiagen) . Experiment was carried in duplicates and a non irradiated control was done for each time.","CATMA",10129,216,0,"2010-01-25","GEO","GSE35942","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (474,"P303E474","AFFY_TOUR_2010_21","treated vs untreated comparison","water input","Plants were grown in the field. The assay was arranged in plots with one genotype per plot. The assay was divided in two identically seized parts, containing the same number of plots.Starting from the capitule, flower # -3 was harvested in every plant. In some cases, several plants from the same plot were harvested. In other, plants from different plots were harvested. We expect that this sampling will allow us to assess intra- as well as inter-plot variability in the analysis of drought-driven gene expression modulation.","Affymetrix",0,303,238,"2012-02-07","GEO","GSE36304","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (176,"P117E176","Nitrogen starvation","time course","time","young plantlets were grown on N-sufficient medium and transfered on N-starvation medium for time course transcription analyses (T0, 24h, 4 days, 10 days).","CATMA",10258,117,1,"2007-06-13","GEO","GSE36621","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (440,"P273E440","Transcriptome analyses of N8514 and wox14","gene knock out,normal vs transgenic comparaison","genotype","The WOX13 OG contained the most conserved plant WOX proteins including the only WOX detected in the highly proliferating basal unicellular and photosynthetic organism Ostreococcus tauri. In Arabidopsis thaliana, an intriguing clade, represented by the functional AtWOX14 gene inside the WOX13 OG, was only found in the Brassicaceae. Compared to AtWOX13, the gene expression profile of AtWOX14 was transient during lateral root formation, anthers and stem development . A mutational insertion upstream of the AtWOX14 homeodomain sequence led to abnormal root development, a delay in the floral transition and premature anther differentiation. Transcriptome analyses of the N8514 transposon donor line and the issued wox14 floral stem basis grown on soil for 8 weeks (growth stage 6) in LD condition is done  to identify genes that are deregulated in the wox14 mutant background.","CATMA",10211,273,256,"2011-01-04","GEO","GSE31137","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (475,"P305E475","AFFY_MED_2011_06","gene knock out,organ comparison","organism part","-Seeds of the WT and noot mutant lines (two mutant alleles: Tnk507 and NF2717) were surface sterilized and placed at 4°C for three days on a minimal BNM medium square plate (12cm X 12cm). Seeds were germinated at room temperature and 10 seeds were placed in a row on nitrogen poor (BNM) plate. Seedlings were inoculated using Sinorhizobium meliloti Rm41 for nodule production. Some WT plants were not infected by the bacteria in order to collect roots. Nodules and roots were harvested 18 to 21 days post inoculation. Each experimental repetition (3 WT nodules, 3 WT roots and 2 mutant nodules of each allele) was harvested at a separated day. RNA preparations were done using a Quiagen Kit. ","Affymetrix",0,305,238,"2012-03-08","GEO","GSE36558","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (189,"P128E189","APC Transcriptome","normal vs rnai mutant comparaison","genotype","Wild Type Col0, APC10S and CDC16S lines (down regulation of APC10 and CDC16 endogenous genes due to overexpression of a sequence corresponding to a part of APC10 or CDC16 gene, respectively) are grown in growth chamber (16h photoperiod 22°C) for 15 days. Whole seedlings are then frozen in liquid nitrogen and total RNA are extracted.","CATMA",445,128,167,"2007-10-26","GEO","GSE36653","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (278,"P174E278","LEC1::GR Induction","treated vs untreated comparison","dose,time","Induction of 14 days old LEC1::GR seedlings , different treatments (EtOH, DEX, DEX+ABA,  induction) for 8 hours","CATMA",10260,174,7,"2009-03-16","GEO","GSE22173","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (292,"P187E292","SG1-SG3","genotype comparaison","genotype","Plants were grown in the in vitro conditions where we have mapped and cloned the concerned QTL and which allows us to see the phenotype linked to the segregation of the natural alleles. We are comparing 2 genotypes descending from a RIL from the Bur x Col RIL set, homozygous and identical everywhere in the genome except for the 23.4 kb at SG1 loci where one genotype is Bur (97.4 and 175.4) and the other is Col (97.1 and 175.1).","CATMA",10084,187,0,"2009-07-09","GEO","GSE36656","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (372,"P247E372","Salmonell invA","normal vs antisens mutant comparison","infection","Plants were grown during 14 days in solid medium and were transfered to liquid medium during 16 h. at 9 am samonella was added to a final concentration of OD600=0.2","CATMA",10255,247,224,"2010-07-02","GEO","GSE36699","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (293,"P187E293","EGO3","genotype comparaison","genotype","Plants were grown in the in vitro conditions with 60 mM mannitol, where we have mapped and cloned the concerned QTL which allows us to see the phenotype linked to the segregation of the natural alleles. We are comparing 2 genotypes descending from a RIL from the Cvi x Col (H1 is Col allele containing HIF, and A2 is Cvi allele containing HIF).","CATMA",10265,187,0,"2009-07-09","GEO","GSE36698","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (160,"P106E160","ARCOLE / PGPR","dose response,organ comparison,treated vs untreated comparison","dose,inoculation,tissue type","A.thaliana seeds germinated and grew during ten days until they were transfered in 6 different media: 0,5 mM nitrate with PGPR (Plant Growth Promoting Rhizobacteria), 0,5mM nitrate without PGPR, 2mM nitrate with PGPR, 2mM nitrate without PGPR, 20 mM nitrate with PGPR, 20 mM nitrate without PGPR. Young plantlets grew 7 days in these new mediums. Shoots are collected in  eppendorf.","CATMA",10264,106,149,"2007-02-27","GEO","GSE36730","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (476,"P306E476","AFFY_COTTON_2011_12","x comparison between two genotypes in cell type","genotype","Total RNA was extracted from 11 dpa-old fibers from the two genotypes, Guazuncho 2 (Gossypium hirsutum) and VH8-4602 (G. barbadense), and included two replicates of each. RNA was checked for quality and quantity using an Agilent Bioanalyser 2100 (Agilent Technologies, Santa Clara, CA, USA, http://www.home.agilent.com) following the manufacturers recommendations. The RNA was sent to the Australian Genome Research Facility Ltd. (http://www.agrf.org.au, Melbourne, Victoria, Australia) for labeling and hybridization to the Affymetrix Genechip® Cotton Genome Array (21,854 genes) (Affymetrix, http://www.affymetrix.com/). -","Affymetrix",0,306,238,"2012-03-27","GEO","GSE36876","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (478,"P307E478","POPLAR BENT STUDY","gene knock in (transgenic)","genotype","- The laboratory previously established a poplar transgenic line overexpressing PtaZFP2 under the control of an estradiol-inducible promoter.- the experiment, conducted on 3-month-old hydroponically-grown poplars, consists in the comparison of WT poplars treated with estradiol and the PtaZFP2-overexpressing line treated with estradiol. We also compared unbent and bent PtaZFP2-overexpressing poplars. The applied strain is quantitatively controlled (Coutand & Moulia, 2000, JExpBot; coutand et al., 2009, Plant Physiology)-","Affymetrix",0,307,238,"2012-04-13","GEO","GSE44321","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (482,"P311E482","Leaf transcriptome comparison between wt and URT1 (SALK_087647) ","gene knock out","genotype","Identification of the RNA targets of the terminal uridylyl transferase URT1 (AT2G45620). Mature rosettte leaf transcriptomes were compared between WT and a tut1 knock-out mutant (SALK_087647).","NimbleGen",10268,311,335,"2012-06-18","GEO","GSE38902","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (479,"P308E479","Transcriptomic analysis of Phosphatin effect","treated vs untreated comparison"," compound","40 µM Phosphatin was added to MS (diluted 10 times) containing 20µM phosphate  medium and controls were grown on MS (diluted 10 times) containing 20µM phosphate or supplemented with 500 µM of Pi. For each ARN extraction their was 3 plates containing each 10 seedlings grown 13 days in vertical petri dishes.","NimbleGen",10266,308,0,"2012-05-07","GEO","GSE45937","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (467,"P296E467","Study of the transcriptional consequences of a growth QTL candidate (Sg1)","gene knock out","null","We are analysing a QTL named Sg1 quantitatively affecting the shoot growth in Arabidopsis thaliana. Sg1 mutation exhibits a large set of phenotypic consequences (growth, fitness,), and the penetrance of these defects increase over generations. The CATMA collaboration aims to perform a T-DNA mutant vs WT analysis, with the WT control being derived from the heterozygous T-DNA parent. A first homozygous generation hypotheticaly derived from the heterozygous T-DNA parent and a 4th homozygous generation showing more severe phenotypes are being compared to the WT (dye swap, biological repetitions). Whole plants were harvested 10 DAG at 11am for each genotype.","CATMA",10246,296,0,"2011-11-02","GEO","GSE39248","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (375,"P241E375","AFFY_AMs_MAIZE","treated vs untreated comparison","genotype","After 7 days in phytochambers, plantlets were transplanted in 2L of humidified bedrock with 3000 spores for inoculation conditions or without spore for mock conditions then plants have grown in greenhouse. Maize roots have been sampled 7 weeks post-inoculation and freezed in liquid nitrogen then stocked at -80°C. Total RNA was isolated from frozen root tissue by using the RNeasy Plant Mini Kit (Qiagen) following the manufacturers instructions.","Affymetrix",0,241,238,"2010-07-06","GEO","GSE22950","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (378,"P250E378","AFFY_Diversity_Maize","organ comparison","cell line","Comparison of two maize lines Cm484 (line of good digestibility) and F98902 (line of bad digestibility) at the same developmental stage (male flowering).","Affymetrix",0,250,238,"2010-07-08","GEO","GSE22948","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (339,"P222E339","AFFY_FSH_Human","treated vs untreated comparison,wt vs mutant comparison","fsh treatment,genotype","In the present study, we compare the wild-type FSH receptor to either the R466A or the T469F mutants. These two mutations are localized in the second intra cellular loop of the FSH receptor and prevent G protein coupling to the active FSH receptor. Each receptor was permanently expressed in HEK-293 cells at comparable levels. Cells were treated or not for 6 hours with 3 nM FSH.","Affymetrix",0,222,7,"2010-03-02","GEO","GSE21982","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (480,"P309E480","RNAPATHS--3_02/2012","genotype comparaison","genotype","Identify transcripts that are significantly changed in double mutants (L1 vcs sgs2) (xrn4-5/sgs3-11)  versus their respective single mutants (L1 vcs and L1 sgs2) (xrn4-5 and sgs3-11) , and identify transcripts that are changed among the single and double mutants compared to WT (Col or L1).","NimbleGen",10267,309,0,"2012-06-18","GEO","GSE38903","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (362,"P240E362","AFFY_Ralstonia_Peeters_Medicago","gene knock out,normal vs disease comparison,time course,treated vs untreated comparison","disease state,genotype,organism part,time","-Medicago truncatula (A17 line) are grown in vitro on Farheus medium (with Nitrogen source) plantlets are inoculated with water R. solanacearum wt, gala7 and hpap mutants, and root tips are collected at 13h and 72h postinoculation. Experiment was performed 3 times independently.4 bacteria conditions x 2 harvest times x 3 biological repeats = 24 samples","Affymetrix",0,240,238,"2010-05-20","GEO","GSE21985","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (324,"P210E324","AFFY_Xoo_Rice","wt vs virulence mutant","strain","The goal of the experiment is to identify the rice genes up- or down-regulated by the type III effector Talc from Xoo African strain BAI3, upon the inoculation of susceptible rice leaves 24 hours post-infection. To that end, the experimental design includes the inoculation of Nipponbare rice leaves with the virulent Xoo strain BAI3, that will be compared to Nipponbare rice leaves inoculated with a talc K.O. mutant strain and water. ","Affymetrix",0,210,7,"2010-01-12","GEO","GSE19844","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (352,"P230E352","AFFY_meloidogyne_rice","normal vs disease comparison,time course","disease state","Oryza sativa (var. Nipponbare) seedlings were grown on 6 cm3 SAP substrate completed with diluted Hoaglands solution (Reversat et al., 1999). Culture units were placed in a growth chamber illuminated with fluorescent tubes 9/24 h and maintained at 23°C for 5 days before being inoculated with a 100 J2-stage juveniles M. graminicola. One day after inoculation (dai), the rice seedlings were immersed in de-ionised water to remove all J2s that had not penetrated the roots and allowing synchronization of the infection. Each seedling was transferred to a hydroponic mini chamber (Reversat et al., 2004). Sampling was performed at 2 and 4 dai and each of them contained galls from 70 infected plants, they were then hand-dissected, frozen in liquid-nitrogen and stored at -80°C. As reference samples, uninfected meristematic root fragments were dissected from seedlings grown under the same conditions. Each sample was replicated 3 times.","Affymetrix",0,230,238,"2010-05-05","GEO","GSE21919","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (353,"P231E353","AFFY_meloidogyne_rice2","normal vs disease comparison","disease state","Oryza sativa (var. Nipponbare) seedlings were grown on 6 cm3 SAP substrate completed with diluted Hoaglands solution (Reversat et al., 1999). Culture units were placed in a growth chamber illuminated with fluorescent tubes 9/24 h and maintained at 23°C for 6 days before being inoculated with a 300 J2-stage juveniles M. incognita. One day after inoculation (dai), the rice seedlings were immersed in de-ionised water to remove all J2s that had not penetrated the roots and allowing synchronization of the infection. Each seedling was transferred to a hydroponic mini chamber (Reversat et al., 2004). Sampling was performed at 6 dai and each of them contained galls from 45 infected plants, they were then hand-dissected, frozen in liquid-nitrogen and stored at -80°C. As reference samples, uninfected meristematic root fragments were dissected from seedlings grown under the same conditions. Each sample was replicated 3 times. ","Affymetrix",0,231,238,"2010-05-05","GEO","GSE21772","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (485,"P314E485","correlation transcriptome/small RNAs","genotype comparaison","genotype","correlation transcriptome/small RNAs","NimbleGen",10272,314,0,"2012-07-26","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (472,"P299E472","comparison between Salmonella Wild type 14028s and Salmonella mutant prgh in Arabidopsis infection.","time course,treated vs untreated comparison","disease state,time","Seedlings treated with S.Typhimurium 14028 WT and St 14028 prgH mutant.Mock treatment done with equal volume of MgCl2.","CATMA",10269,299,335,"2011-11-24","GEO","GSE38828","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (435,"P268E435","AFFY_RNAI_CADpoplars","normal vs rnai mutant comparaison","genotype","RNAi-CAD transgenic poplars were produced using hairpin RNAi strategy (pHellsgate 8 vector, Helliwell and Waterhouse, 2003). For this transcriptome anaylsis, 2 independent transgenic lines (named pHG8-CAD2 and pHG8-CAD19) from the same transformation procedure were used as biological repeats. Four-month-old poplar plants were inclined at 30° in the greenhouse and sampled after 26 days. Young differentiating xylem originating from the lower side of stems - opposite wood - (ODX) was sampled on each individual tree by scrapping slightly the debarked stem with a scalpel. Samples were immediately flash frozen in liquid nitrogen, ground with mortar and pestle, and total RNAs were extracted from fine ground powder using the QIAGEN miRNeasy kit according to the manufacturer.  One Affymetrix slide corresponds to a pool of RNA samples from 2-4 individual trees (WT, RNAi-CAD transgenic lines 2 and 19). Total number of slides = 2 genotypes (WT/RNAi line) x 1 tissue x 2 biological replicates = 4 slides were done.","Affymetrix",0,268,238,"2010-12-07","GEO","GSE27063","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (484,"P313E484","AtNSRs","gene knock out","hormone treatment","comparaison between Arabidopsis thaliana Col-0 and Atnsra/b double mutant treated or not with 24h 1µM NAA","NimbleGen",10271,313,0,"2012-07-25","GEO","GSE39659","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (486,"P315E486","Transcriptomic analysis of MC2 effect","treated vs untreated comparison"," compound","Plants were grown vertically on petri dishes  9 days on MS medium (diluted 10 times)containing 100µM Phosphate. Then plants were transferred during 24h to medium with (P100) or without Phosphate(P0), with or without MC2 5µM (P100 MC2 or P0 MC2).","NimbleGen",10273,315,0,"2012-07-26","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (490,"P319E490","Flagellin experiment in prr2 mutant and overexpressor (OE)","treated vs untreated comparison"," compound"," 10 day-old seedlings grown in a climate chamber on MS solid medium were transfered in MS liquid medium and then supplied with flg22, a flagellin-derived peptide to a final concentration of 1 µM. The whole plantlets were collected 30 min or 12h after treatment, frozen in liquid nitrogen and stored at -80°C.","NimbleGen",10277,319,0,"2012-07-30","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (468,"P297E468","Dark hypocotyls TOR RNAi","normal vs rnai mutant comparaison","genotype","Sowing after 24h imbibition at 4°C in the dark, on MS1/5, no sucrose, 10 mM ethanol, 8 g/l agar, vertical growth with 3h light, 6 days growth in the dark (20°C), hypocotyls were harvested under green light ; cotyledons and root were removed.","CATMA",10247,297,0,"2011-11-04","GEO","GSE41404","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (469,"P298E469","shoots and roots treated with Methotrexate and 5-Formyl-THF","treated vs untreated comparison"," compound,tissue type","The objective of this study is to investigate the changes on the plant expression profiles due to 5-CHO-THF assimilation and transformation. Additionally, plants supplemented with methotrexate (MTX) (an inhibitor of folate biosynthesis) will complement the study. Results from this work will serve to better understand the effects of folate accumulation in plants. ","CATMA",10248,298,0,"2011-11-08","GEO","GSE41406","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (491,"P320E491","Viromouv","normal vs disease comparison","infection","Transgenic plants expressing GFP under the control of a companion cell specific promoter were infected by wo different viruses: LMV and TuYV. Companion cell protoplasts from these plants were sorted by FACS and extracted RNA were compared to healthy companion cell protoplasts.","CATMA",10278,320,0,"2012-08-02","GEO","GSE42059","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (495,"P323E495","AFFY_MED_2011_10","near isogenic lines (hif),normal vs mutant comparison","inoculation 3dpi with ae","Fourteen days old A17, lyr3-1 or lyr3-2 plants were grown in vitro on M medium. The entire roots were then inoculated (or not = controls) with A. euteiches zoospores (105/ml) and harvested three days after inoculation. Three independent repeats were performed.","Affymetrix",0,323,238,"2013-01-21","GEO","GSE43715","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (494,"P307E494","POPLAR ESTRADIOL STUDY","time course","strain","The laboratory previously established a poplar transgenic line overexpressing PtaZFP2 under the control of an estradiol-inducible promoter.- the experiment, conducted on 3-month-old hydroponically-grown poplars, consists in the comparison of WT poplars treated with estradiol and the PtaZFP2-overexpressing line treated with estradiol. We also compared unbent and bent PtaZFP2-overexpressing poplars. The applied strain is quantitatively controlled (Coutand & Moulia, 2000, JExpBot; coutand et al., 2009, Plant Physiology)","Affymetrix",0,307,238,"2013-01-10","GEO","GSE43533","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (481,"P310E481","AFFY_MED_2011_09","wt vs mutant comparison","am symbiosis),dose","We will compare the root transcriptome of  Medicago truncatula plants inoculated with Rhizophagus irregularis to that of non-inoculated plants grown under P limitation (or both P and N limitation) after 4 weeks of culture","Affymetrix",0,310,238,"2012-06-18","GEO","GSE38847","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (507,"P336E507","efd-1 and wt roots and nodules","gene knock out,time course","developmental stage,genotype","Comparison of wild type and efd-1 transcriptomes in non inoculated nitrogen-starved control roots and  nodules at 4 dpi, 6 and 11 dpi. Comparison of wild type nodule (11 dpi) and root transcriptomes, using mixed random primed and polydT primed probes.","NimbleGen",10279,336,174,"2013-04-10","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (497,"P325E497","AFFY_MED_2012_03","normal vs ems mutant comparison","treatment with cos or a.eut chitosaccharides ","7-day-old A17 wild type or mutant nfp-1 or nfp-2 plants were grown in vitro on liquid M medium. The entire roots were then treated (or not = controls) with a preparation of crab shell chitin chitooligosaccharides of a mean DP of 7 or with chitosaccharides from A. euteiches cell wall, names fraction P2F1, and harvested after 4 hours of incubation. Three independent repeats were performed.","Affymetrix",0,325,238,"2013-01-28","GEO","GSE43839","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (500,"P328E500","AFFY_MED_2011_14","dose response,gene knock out","dose,genotype","- Roots were harvested from 4 week-old WT and B9 mutant plants grown in a culture chamber with limiting (P/10, 130 µM phosphate) or non limiting (P2, 2.6 mM phosphate) phosphate under long days (16 hrs day at 22°C with 380 µE.m-2.s-1  light and 8hrs night at 19°C). RNAs were then extracted for transcriptomic analysis. We will compare WT (or mutant) root transcriptome on P/10 versus WT (or mutant) root transcriptome on P2 as well as mutant versus WT transcriptome on either P/10 or P2.","Affymetrix",0,328,238,"2013-01-31","GEO","GSE44102","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (505,"P333E505","Leaf transcriptome comparison between wt and PNM1DNLS plants","normal vs transgenic comparaison","genotype","Identification of nuclear genes regulated by PNM1 (At5g60960) in Arabidopsis thaliana. Mature rosettte leaf transcriptomes were compared between WT and PNM1DNLS mutants. ","NimbleGen",0,333,335,"2013-03-21","GEO","GSE45870","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (464,"P294E464","AFFY_SUNFLOWER_2010_13","treated vs untreated comparison"," compound","We aim to understand the mechanisms controlling sunflower seed dormancy at the transcriptomic level, by the application of treatments which maintain dormancy as ABA, or alleviate dormancy as ROS and ethylene. Transcripts comparison will be performed between dormant and non-dormant sunflower embryo imbibed 24h on water, on ABA, on methylviologen, a pro-oxidant compound or on ethylene.","Affymetrix",0,294,238,"2011-08-04","GEO","GSE44165","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (503,"P331E503","AFFY_ROSE_2012_06","normal vs rnai mutant comparaison","scent gene","Study of rose perfume by comparing a wild-tpe rose with one in which the expression of a scent gene has been altered.","Affymetrix",10136,331,238,"2013-02-08","GEO","GSE45236","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (504,"P332E504","AFFY_SUNFLOWER_2011_02","treated vs untreated comparison","temperature","5 genotypes of sunflower were grown in a growth chamber 1 (23°C day/18°C night, 63% air humidity, 14 hours day photoperiod). At the stage of 6 leaves well-developed, 12 plants of each genotype were subjected to cold acclimation (+4°C during 2 days) in another growth chamber 2. Then, these plants were subjected to 2 nights at -3°C (frost treatment). Chlorophylle fluorescence, Osmotic potential, Relative electrolyte leakage were then determined in the following conditions :- 6th October on 6 plants X 5 genotypes from growth chamber 1 (C1)- 6th October on 6 plants X 5 genotypes from growth chamber 2 (S1)- 11th October on 6 plants X 5 genotypes from growth chamber 1 (C2)- 11th October on 6 plants X 5 genotypes from growth chamber 1 (S2)","Affymetrix",0,332,238,"2013-02-14","GEO","GSE45586","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (522,"P350E522","cpsf30 mips1 transcriptome","genotype comparaison","genotype","All the plants were cultivated in the same hanging conditions 10j in short Days then in long Days.The takings were later made 4j in long days what corresponds to the appearance of the symptoms of cellular death at the mutant mips1-1. On the other hand, the gene CPSF30 codes for two transcribed. We have a double mutant mips1 cpsf30 complemented with the transcribed young. ","NimbleGen",10293,350,0,"2013-04-22","GEO","GSE53247","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (514,"P343E514","IVT","treated vs untreated comparison","temperature","We applied a Controlled Deterioration Treatment (CDT) to seeds from the reference rice cultivar Nipponbare. First, all seeds are equilibrated at 25°C and 85% relative hygrometry. Then, depending on the treatment, seeds are placed at 25, 40 or 45°C in 85% relative hygrometry before being finally equilibrated at 25°C and 32% relative hygrometry. The germination of the three seed batches was measured during five days with one measure every 8h. Seeds placed at 25°C during the whole experiment were similar to control seeds kept in the fridge and germinated at nearly 100% in 48h. Seeds placed at 40°C during 15 days germinate at 74% but show altered seedling phenotypes (loss of seed vigor). Finally, seeds placed at 45°C do not germinate. ","Affymetrix",0,343,238,"2013-04-15","GEO","GSE46342","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (509,"P338E509","Response to salt stress of Raptor mutants","gene knock out,treated vs untreated comparison"," compound,genotype","Sowing on metal grids on MS/2 medium with continuous light (90 &#956;E/m2/s 22°C). Transfer after 12 days of culture for 3h to liquid MS/2 medium containing or not 200 mM NaCl.","CATMA",10285,338,0,"2013-04-12","GEO","GSE47391","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (516,"P345E516","Seedling transcriptome comparison between wt and URT1 (SALK_087647).","gene knock out","genotype","Identification of the RNA targets of the terminal uridylyl transferase TUT1 (AT2G45620). 14 day-old seedling transcriptomes were compared between WT and a tut1 knock-out mutant (SALK_087647).","NimbleGen",0,345,335,"2013-04-18","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (510,"P339E510","Benzene effects on leaves","treated vs untreated comparison"," compound","Cabbages (Brassica oleracea Var. Prover)are directly submitted to a one dose of benzene (3ppm)during 1 hour or 3 hours. The comparison is realized between samples treated and untreated  to the benzene for both exhibitions.","CATMA",10286,339,0,"2013-04-12","GEO","GSE48179","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (508,"P337E508","myb118 mutation","normal vs transgenic comparaison","genotype","Plants of the different genotypes (wild-type, myb118 mutant, myb118 antisens lines named 0E3 and T16) were grown in a green house, their flower were tagged and seeds were harvested 10 days after anthesis. Seeds were extracted from the siliques, stored at -80°C prior to RNA extraction.","NimbleGen",10284,337,0,"2013-04-11","GEO","GSE48180","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (501,"P329E501","AFFY_BRACHY_2011_11","normal vs disease comparison,time course","disease state,strain","Three conditions on B. distachyon spikes:1-Mock inoculation (Tween 20 0,01%)2-Inoculation by a F. graminearum wild-type strain3-Inoculation by a F. graminearum mutant strain, unable to produce DONSpikes were point inoculated with 3 &#61549;l of either Tween 20 0.01%, wild-type strain or mutant strain (300 spores) and incubated for 96 hours. Six inoculated spikes were collected and pooled for each condition and biological replicate. Three independent biological replicates were conducted. ","Affymetrix",0,329,238,"2013-01-31","GEO","GSE50665","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (327,"P213E327","Analyses of endogenous RNA substrates of XRN and exosome and PTGS pathways.","gene knock in (transgenic),gene knock out","genotype","Genome-wide analyses of endogenous RNA substrates of RNA Quality Control pathways.wild type versus 15 mutants.","CATMA",10126,213,0,"2010-01-15","GEO","GSE48192","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (520,"P348E520","CDT1_bis","normal vs rnai mutant comparaison,normal vs transgenic comparaison","genotype","Plantlets of crl mutant and CDT1-RNAi lines were grown in vitro on MS1/2 medium for 14 days. CDT1-RNAi lines were compared to the corresponding wild-type (Ws), whereas crl mutants were compared to their wild-type siblings and are in the Col0 ecotype.","NimbleGen",10292,348,0,"2013-04-19","GEO","GSE48465","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (511,"P340E511","Down regulation of QPTgene in Arabidopsis","normal vs transgenic comparaison","genotype","Study of the biosynthesis of NAD in Arabidopsis. Involvment of QPTgene using the knock out Salk mutant N575260.","CATMA",10287,340,0,"2013-04-12","GEO","GSE49034","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (523,"P334E523","Col-0 vs cyp715A1","gene knock out,genotype comparaison","genotype","Flower buds of Arabidopsis Col-0 (wild-type) and cyp715A1 mutant were harvested for a comparative analysis of their transcriptomes.","NimbleGen",0,334,335,"2013-04-22","GEO","GSE52269","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (519,"P343E519","OVATION","treated vs untreated comparison","temperature","We applied a Controlled Deterioration Treatment (CDT) to seeds from the reference rice cultivar Nipponbare. First, all seeds are equilibrated at 25°C and 85% relative hygrometry. Then, depending on the treatment, seeds are placed at 25, 40 or 45°C in 85% relative hygrometry before being finally equilibrated at 25°C and 32% relative hygrometry. The germination of the three seed batches was measured during five days with one measure every 8h. Seeds placed at 25°C during the whole experiment were similar to control seeds kept in the fridge and germinated at nearly 100% in 48h. Seeds placed at 40°C during 15 days germinate at 74% but show altered seedling phenotypes (loss of seed vigor). Finally, seeds placed at 45°C do not germinate. ","Affymetrix",0,343,238,"2013-04-19","GEO","GSE46334","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (533,"P359E533","hdt2 transcriptome","gene knock out,treated vs untreated comparison","genotype","Total RNA from 14-day-old seedlings Col and hdt2 mutant, treated or not in liquid medium 30 minutes with flagellin 22, have been extracted.","NimbleGen",10303,359,0,"2013-09-06","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (521,"P349E521","tiling hen2 and mtr4","gene knock out,genotype comparaison,normal vs transgenic comparaison","genotype","Total RNA extracted from WT, hen2 and mtr4 mutants was used for oligo-dT primed cDNA preparation, samples were hybridised to Nimble-gene whole genome tiling arrays.","NimbleGen",0,349,7,"2013-04-19","GEO","GSE48178","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (527,"P354E527","Transcriptomic comparison of  tumoric tissues induced by different C58 Agrobacterium lines","null","genotype of pathogen","Arabidopsis WT Col0 plants were grown in greenhouse till stage 1.03-1.04 (growth conditions 1). They were then confined in short day conditions and at 22°C (growth conditions 2). At stage 5.10 plants were wounded at the base of the stem and part of them were inoculated with 3 different C58 Agrobacterium strains (WT, accR or traI). Eighteen days post inoculation, wounded but non-infected tissues (control) as well as tumoric tissues were collected and processed for RNA extraction.","CATMA",10299,354,0,"2013-06-27","GEO","GSE48402","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (518,"P347E518","miR775_1st","gene knock out","generational stage","Look for potential targets of miR775 at Arabidopsis thaliana by the analysis of the transcriptome of a mutant not producing any more the miARN","NimbleGen",10291,347,0,"2013-04-19","GEO","GSE68084","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (524,"P351E524","Inhibition des PI-PLC par le U73122","treated vs untreated comparison,treatment variations"," compound","Arabidopsis thaliana cells in suspension . 5 days (transplanting every 7 days). Cultivated in environment(middle) Gamborg + 2,4-D.Added inhibitors 4 hr before extraction of the ARNs.","NimbleGen",10295,351,0,"2013-04-23","GEO","GSE46941","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (531,"P357E531","MtEFD1 and wt roots and nodules","gene knock out,time course","developmental stage,genotype","Comparison of wild type and efd-1 transcriptomes in non inoculated nitrogen-starved control roots and  nodules at 4 dpi, 6 and 11 dpi. Comparison of wild type nodule (11 dpi) and root transcriptomes, using mixed random primed and polydT primed probes.","NimbleGen",10301,357,0,"2013-07-16","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (532,"P358E532","12plex_MED_2012-06","genotype comparaison","genotype","Medicago truncatula wild-type and lyr3 mutant seedlings were treated with three different types of lipo-chitooligosaccharides (Sinorhizobium meliloti Nod factors, sulphated Myc-LCOs and non-sulphated Myc-LCOs), in order to compare the transcriptomic responses to these molecules between genotypes. Two independent allelic lyr3 mutants were used to be able to identify changes due soley to mutation of the LYR3 gene. Two independent biological repeats were performed for all samples.","NimbleGen",10302,358,0,"2013-07-19","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (535,"P362E535","Gravity","time course,treated vs untreated comparison","cell type,time","Experiments consists on exposures of Arabidopsis cell cultures to 1g control/simulated microgravity (RPM) conditions. Asynchronous cells exposed for 14 h + Syncronous populations choosen to have an enrichment of cell cycle phases were used (being T7/T10 samples on G2 phase, T14/T16 samples on G1 phase).","NimbleGen",10305,362,0,"2013-10-16","GEO","GSE60473","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (530,"P356E530","12PLEX_VITIS_2011_02_24h","normal vs disease comparison,treated vs untreated comparison"," compound","Identification of grape genes regulated in response to resistance inducer treatments.This transcriptomic analysis should provide insight in the mode of action of the compounds under study and yield generic data relevant to grape defense reaction.","NimbleGen",0,356,238,"2013-07-08","GEO","GSE97700","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (529,"P356E529","12PLEX_VITIS_2011_02_48h","normal vs disease comparison"," compound","The experiment took place in the greenhouse. Six week-herbaceous grape cutting (cv Marselan) were spray-treated with 5 resistance inducers, an adjuvant (used along with some inducers) and water as control. Plants were inoculated by Plasmopara viticola (downy midew) 24 h post-treatment. We used 5 plants per treatment. Leaf samples were harvested 24 h post-treatment and 48 h post-inoculation. Three biological repetitions were done.Each hybridization experiment was performed in comparison with water.","NimbleGen",0,356,238,"2013-07-03","GEO","GSE97699","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (534,"P360E534","oxi1 Transcriptome","genotype comparaison","genotype,light","5 weeks old mutannt (M) and wild type (WT) plants were exposed to high light and low temperature (1300-1350 µmol photons m-2s-1, 7°C/14°C day/night and 380ppm CO2) for 25 hours. ~100 mg fresh weight leaves were harvested, and total RNA prepared from them. For control experiments, leaves were harvested directly from the phytotron (No light stress). Three microarray comparisons were made: Mcontrol/WTcontrol, Mstress/WTstress and Mstress/Mcontrol . For each comparison, 2 biological replicates are performed.","NimbleGen",10304,360,0,"2013-09-30","GEO","GSE61478","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (562,"P385E562","Transcriptomic analysis of Medicago root hairs","treated vs untreated comparison","rhizobial nod factors","Control : plant 5 days with H20NF_TC (short time) : treated during 4h with NOD factorsNF_TL (long time) : treated during 20h with NOD factors","Illumina",0,385,238,"2015-02-25","GEO","GSE67921","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (556,"P379E556","transcript modification by MYB1 interactants","normal vs transgenic comparaison","genotype","We produced 5 transgenic lines, overexpressing Eg MYB1,Eg MYB1+ EgOVATE, EgMYB1+EgHISTONE, EgHISTONE, EgOVATE and the CONTROL with the empty vectors. We have 2 lines of each construction. Plants were grown in short day conditions and we sampled the inflorescence stem.","NimbleGen",10312,379,0,"2014-08-22","GEO","GSE68024","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (567,"P391E567","LLHH clock transcriptome","time course","light,temperature,time","Wild type seedlings(Col-0)have been grown under Light/Dark conditions(12 h Light:12 h Dark)and thermocycles(23°C day:19°C night).After 10 days of entrainment, the conditions were switched to continuous light and temperature (LLHH) for 2 days. Seedlings have been harvested the 2nd day after the switch at Zeitgeber time 24 (T0) and 36 (T12) that correspond to dawn and dusk, respectively.","Agilent",10323,391,335,"2015-04-22","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (564,"P387E564","Seed transcriptome of xrn4-5 and vcs-8 mutants.","gene knock out","genotype","Study of the seeds regulation by the RNA-decay. Comparison between Col0 total RNA and mutants xrn4-5 and vcs-8 RNA, affected in the dormancy and in RNA metabolism.","Agilent",10321,387,335,"2015-03-19","GEO","GSE68268","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (552,"P376E552","Transcriptional profilling of Arabidopsis plants.","genotype comparaison,treated vs untreated comparison","drought","Study on empty vector, overexpressor and antisens A.Thaliana lines.","NimbleGen",10322,376,335,"2014-05-30","GEO","GSE58219","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (528,"P355E528","Fe-FIT-Diff","normal vs transgenic comparaison,treated vs untreated comparison","genotype","Transcriptome changes in response to Fe deficiency versus the ctrl in wild type and 2 mutants.","NimbleGen",10300,355,0,"2013-07-02","GEO","GSE65934","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (546,"P372E546","4PLEX_TOMATO_2013_03","genotype comparaison","genotype","2 strains of tomato (Cervil, a cherry tomato, and Levovil, an large fruited tomato) were cultivated under normal hydric conditions and drought stress. Young leaves were gathered 3 1/2 months after sawing for analysis.","Agilent",10320,372,238,"2014-02-04","GEO","GSE69898","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (561,"P384E561","pi4kbeta1beta2 sid2 triple mutant","genotype comparaison","genotype","The pi4kbeta1beta2 sid2 triple mutant plants were compared to sid2 plants or to pi4kbeta1beta2  double mutant plant.   As a control  the pi4kbeta1beta2  double mutant was compared to WT.","Agilent",10316,384,0,"2015-01-27","GEO","GSE93118","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (568,"P393E568","flavodentata","gene knock out,organ comparison","genotype,tissue type","mRNA-Seq on wild-type Col and FLV mutants knock out.","Ion Torrent",0,393,393,"2015-06-10","GEO","GSE71149","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (569,"P392E569","Transcriptome deficient mutants for the epsilon DNA polymerase","normal vs transgenic comparaison","genotype","Plantlets were grown in vitro on half strength MS for 9 days and used for total RNA extraction.","Ion Torrent",0,392,393,"2015-06-11","GEO","GSE71002","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (570,"P394E570","cuc2-3 - wt - CUC2gm-4 lines","genotype comparaison","genotype","3-weeks old plants grown in short condition from each genotype (cuc2-3, WT and CUC2gm-4 lines) were dissected and meristem tissue and young leaf primordia were collected and snap frozen in liquid nitrogen. Total RNA were extracted using miRVANA kit following manufacturers instructions. 3 biological repeats were performed for each genotype.","Ion Torrent",0,394,393,"2015-06-12","GEO","GSE72134","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (571,"P395E571","otp90","gene knock out","genotype","Comparison of the otp90 knock-out mutant with the corresponding WT genotype in regular growth conditions.","Ion Torrent",0,395,393,"2015-06-12","GEO","GSE71202","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (557,"P380E557","Observation des effets du produit Tn sur lexpression des gènes dA.thaliana.","dose response"," compound,dose","Each plant is sprayed with a product. 24 h or 48h after treatment, a leaf is removed for RNA extraction and analysis.","Agilent",10324,380,335,"2014-09-01","GEO","GSE93095","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (563,"P386E563","Study of a kinetic answer","time course,treated vs untreated comparison","cell line","Cells were submitted to products and harveted at time 0, 1 4 and 4 hrs","Agilent",10317,386,0,"2015-03-04","GEO","GSE93119","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (572,"P361E572","Effect of Pseudomonas fluorescens pyoverdine on A.thaliana defence responses and iron homeostasis","organ comparison,treated vs untreated comparison"," compound,tissue type","Arabidopsis thaliana plants cultivated in hydroponic conditions were treated for 3 days by pyoverdin, a bacteriosiderophore from Pseudomonas fluorescens C7R12 in a medium supplemented or deprived of iron.","NimbleGen",10330,361,335,"2015-06-30","GEO","GSE71163","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (537,"P364E537","12PLEX_MED_2013_04","treated vs untreated comparison"," compound","2 day old M. truncatula (accession A17) plantlets grown on M medium were transferred for 2 days on M medium+ 10-5M NPA (1-N-Naphthylphthalamic acid, auxin transport inhibitor) then transferred for 10 h on M medium containing 10-6 M NAA (naphthalene-1-acetic acid, permeant auxin analog) or 10-7 M Nod factors (NF) or a combination of both (10-6 M NAA + 10-7 M NF). These are compared to mock treated (solvent only) plants.","NimbleGen",10331,364,238,"2013-12-17","GEO","GSE74099","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (579,"P402E579","Effect of N nutrition of Arabidopsis response to Erwinia infection","normal vs disease comparison,treated vs untreated comparison","disease state","Comparison of Arabidopsis responses to Erwinia infection in relation to nitrogen nutrition (sufficient vs limited).","Agilent",10336,402,335,"2015-11-06","GEO","97582","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (577,"P400E577","Transcriptomic analysis of the effects of low levels of atrazine and by-products","treated vs untreated comparison"," compound","Seeds of Arabidopsis thaliana (Columbia ecotype, Col-0) were surfaced-sterilized in bayrochlore/ethanol (1/1, v/v), rinsed in absolute ethanol and dried overnight. Germination and growth were carried out under axenic conditions in square Petri dishes. After seeds were sowed, Petri dishes were placed in the dark at 4 °C for 72 h in order to break dormancy and homogenize germination, and were then transferred to a control growth chamber at 22 °C/20 °C under a 16 h light (6000 lux)/8 h dark regime. Growth medium consisted of 0.8% (w/v) agar in Hoagland basal salt mix (H2395, Sigma-Aldrich) adjusted to pH 6.  After 13 days of growth under optimal conditions, seedlings were transferred to fresh growth medium containing pesticides during 24 h (Control, Atrazine 1µM, Hydroxyatrazine 1µM or Desethylatrazine 1µM). Then control and treated seedlings were harvested and ground in liquid nitrogen until RNA extraction by RNeasy kit (Qiagen) and DNase digestion. RNA was extracted on at least 2 independent biological replicates of 30 pooled plantlets.","Agilent",0,400,335,"2015-10-28","GEO","GSE88788","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (575,"P398E575","transcription of ectopic lignification","genotype comparaison","genotype","RNA extractions have been performed on mur1 and WT stems of 13 cm. Three biological repeats were used for the transcriptomic analysis.","Agilent",10333,398,0,"2015-10-20","GEO","GSE74857","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (576,"P399E576","AtMYC4 overexpression in Arabidopsis (ws)","normal vs transgenic comparaison","genotype","Compare the gene transcription in the mature stem of Arabidopsis wild type (ws) and AtMYC4 overexpressors (inws background)","Agilent",10334,399,0,"2015-10-20","GEO","GSE93120","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (582,"P405E582","CLUSTOP transcriptomics.","treated vs untreated comparison"," compound","Phytotoxicity of  Cs2[Mo6Br14]@SiO2 nanoparticles and their components, i.e. Cs2[Mo6Br14] clusters and SiO2 nanoparticles, has been studied usign Arabidopsis thaliana cell suspension culture. Cs2[Mo6Br14]@SiO2 nanoparticles used here are composed of 7,5% clusters and 92,5% SiO2. Thus, to compare to the impact of a 100 mg/L Cs2[Mo6Br14]@SiO2 nanoparticle treatment (CS), we also treated Arabidopsis cells with clusters at 7,5 mg/L (C), SiO2 at 92,5 mg/L (S), and combined 7,5 mgCMB/L plus 92,5 mgSiO2/L (C+S).","Agilent",10339,405,335,"2016-03-08","GEO","GSE93182","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (574,"P397E574","Effet of light on translatome of Arabidopsis seeds during germination","treated vs untreated comparison","light,temperature,time","At harvest seeds are dormant.They stay dormant if stored at -20°C.A.Th dormant seeds dont germinate at 25°C in darkness.Total RNA and polysomal RNA (polysomal fractions  purified on sucrose gradients)were extracted from imbibed seeds for 20h at 22°C in darkness and light(3 biological replicates).Transcriptome and translatome are compared for  light vs dark for 20h of imbibition. In silico comparison will allow to compare transcriptome and translatome  for each type of sample.","Agilent",10332,397,335,"2015-09-30","GEO","GSE93181","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (578,"P401E578","BurxCol-465","gene knock out","genotype","The candidate gene (AT1G36310) was identified through QTL mapping and cloning for rosette leaf growth trait in a Bur x Col RIL set.  It corresponds to a medium effect on growth and a major effect on other traits (leaf colour and pigments). We are now exploiting an independent TDNA-insertion line (SALK_135308) in the Col-0 background) to study its effect in a simple genetic context. This is a simple comparison between homozygous T-DNA (2 lines) and their respective WT (2 lines).","Agilent",10335,401,335,"2015-11-04","GEO","GSE93513","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (580,"P403E580","Identification of MKK3-dependent wonding-induced genes","gene knock out,genotype comparaison,normal vs transgenic comparaison,treated vs untreated comparison","genotype,time","Col and mkk3 KO plants were wounded with a forceps. Leaves were harvested 2h after the stress.","Agilent",10337,403,335,"2016-03-01","GEO","GSE95621","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (581,"P404E581","Differential gene expression in A.Tha. depending on mutation in FIT","gene knock out,genotype comparaison,normal vs transgenic comparaison","genotype","Differential gene expression in A. thaliana seedlings depending on the introduction of a point mutation in FIT (FER-LIKE IRON DEFICIENCY-INDUCED TRANSCRIPTION FACTOR) under iron deficient conditions.","Agilent",10338,404,335,"2016-03-03","GEO","GSE97401","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (592,"P295E592","RNA-Seq part","organ comparison","tissue type","The two selected samples are the flower buds and leaves of Arabidopsis. Two organs known as the transcriptome is very different. This RNA-Seq experiment will serve as reference to compare to Array experiment (BF_vs_F)","Illumina",0,295,8,"2016-08-25","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (593,"P414E593","Vitis Disease","normal vs disease comparison,treated vs untreated comparison","disease state","We have made a sampling  of 15 vines (Chardonnay, Avize) in September 2014. Out of 15 plants, 5 expressed foliar symptoms and a treatment with sodium arsenite was done on those plants in February 2014. The other 10 plants have not been treated, and in these 10 plants, 5 expressed foliar symptoms and 5 others not (in 2014). Plants have been cut in different organs and we aim to evaluate the effect of arsenite treatment on the vine naturally infected by the fungi responsible for grapevine trunk disease, by comparing  in two different organs, the healthy trunk wood and the green stem.","Illumina",0,414,0,"2016-09-07","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (584,"P407E584","Fe-FIT-Diff_6d","gene knock in (transgenic),gene knock out,treated vs untreated comparison"," compound,genotype","Col-0, HA-FIT and fit-3.","Agilent",0,407,335,"2016-03-10","GEO","GSE80281","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (583,"P406E583","Ppmax2 response to GR24","genotype comparaison,normal vs transgenic comparaison,treated vs untreated comparison"," compound,genotype","Two moss genotypes are used: WT and the Ppmax2 mutant. Moss tissues are fragmented,  then plated on medium (Petri dish with cellophane disks) and cultivated for 3 weeks. Moss tissues are then transfered for 6 hours on acetone-containing medium (control treatment, for WT and Ppmax2) or GR24 (1 microM, in acetone)-containing medium (for Ppmax2). After 6 hours, the moss tissues are collected, quickly forzen in liquid nitrogen. RNA are isolated using the Quiagen RNeasy Plant mini kit (including a RNase-free DNase treatment on column). Two similar experiments (T1 and T2) have been led. ","Agilent",10340,406,335,"2016-03-09","GEO","GSE80578","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (587,"P410E587","TopoVI and TFIIFa-mediated transcriptional response to photooxidative stress","genotype comparaison,treated vs untreated comparison","genotype,light","In order to identify the molecular mechanisms by which the plant Topoisomerase VI and its interactor TFIIFa may act together as regulator of gene expression in response to photooxidative stress, a RNA-seq analysis is performed on the two mutants caa39 and tfIIfa-1 (1171a-1) and their corresponding wild types (WT Col0 and 1171WT) under photo-oxidative stress conditions or not (high light treatment, 0 and 30 time points) (4 genotypes, 2 time points, 3 biological replicates).","Illumina",0,410,0,"2016-04-28","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (430,"P263E430","AFFY_infection_Medicago","normal vs disease comparison,normal vs ems mutant comparison,time course","disease state,genotype","Fifteen-day-old A17 or Mtnf-ya1-1 plants were grown in vitro on M medium. The entire roots were then inoculated (or not = controls) with a A. euteiches zoospores (105/ml) and harvested one or six days after inoculation. Three independent repeats were performed.","Affymetrix",0,263,238,"2010-10-12","GEO","26046","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (590,"P412E590","bigun transcriptomics","gene knock out,normal vs disease comparison","disease state,genotype","5 weeks after germination of 12 seedlings of Arabidopsis thaliana Col-0,bigun 1 mutant and bigun 2 mutant genotypes(growth condition n°1),6 plants are infected by spraying a solution of Hyaloperonospora arabidopsidis spores,and 6 plants are left untreated,and all plants were cultivated under growth condition n°2.For each condition,leaves were sampled and pooled one week after infection.The experiment was repeated three times.","Agilent",0,412,335,"2016-07-07","GEO","GSE84277","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (594,"P415E594","Gravitropic stimulation / Isotropic light","treated vs untreated comparison","age,tilted versus straight","young poplar trees were staked one week before use. For the experiment, the plants were placed in an isotropic light environment and were tited at 35° during 30 min. Control plants were kept straight in the isotropic light device. Xylem samples were collected from the upper and lower face of the stems. 4 biological replicates were made (4 controls and 4 tilted plants). RNA was extracted from each xylem sample. cDNA was synthetized. The discovery of genes whose expression is modulated by tilting was performed by a microarray approach.","NimbleGen",0,415,0,"2016-09-12","GEO","GSE86951","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (526,"P353E526","cold stress ","treated vs untreated comparison","temperature,time","Experiments were performed on inflorescence of Vitis vinifera L. cv. Pinot noir fruiting cuttings.First, inflorescences at female meiosis were collected the day of the experiment, then cuttings with the inflorescence were placed at 0°C for a 8 h night. Control plants were maintained in a growth chamber for 8 h at 19°C. Inflorescences were collected after   1 h and 8 h of cold. Additional time point was performed 2 h after the end of the cold night by placing the inflorescence at 25°C after the end of the cold night. Control plants were also collected.","NimbleGen",10298,353,0,"2013-06-26","GEO","GSE87296","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (588,"P411E588","Albumen Vitreux vs Albumen Farineux","null","tissue type","Comparaison of vitreous (outer) and floury (inner) endosperm from flint and dent maize inbred lines harvested at 15 and 20 day after pollinisation.","Agilent",10342,411,335,"2016-05-10","GEO","GSE88993","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (586,"P409E586","High Temp WT","treated vs untreated comparison","temperature","plants were grown at 21°C or 26°C, 16h light (90µE)/8h dark for 10 days before harveting roots.","Illumina",0,409,238,"2016-04-26","GEO","GSE99283","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (536,"P363E536","AFFY_MED_2013_05","gene knock out","nodule/ normal versus symbiotic mutants,organism part","Medicago truncatula WT (R108) and two mutant lines (NF737 and MS240/dnf2) were nodulated using WSM419 Sinorhizobium medicae symbiotic bacteria. Plants were grown in vitro on BN medium and the nodules harvested 14 days post inoculation (dpi). RNA was extracted using the plant RNA extraction kit from Thermo Scientific followed by a DNAse I treatment (TurbO DNAse I Ambion). ","Affymetrix",0,363,238,"2013-11-21","GEO","GSE54862","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (589,"P411E589","CINETIQUE comparison","time course","developmental stage,tissue type,vitreous vs floury endosperm regions","comparaison of vitreous (outer) and floury (inner) endosperm from flint and dent maize inbred lines harvested at 15 and 20 day after pollinisation.","Agilent",10343,411,335,"2016-05-24","GEO","GSE119550","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (573,"P396E573","Identification of genes misregulated to myb92 mutant and MYB92 OE.","gene knock in (transgenic),gene knock out,normal vs transgenic comparaison","genotype","Plants of the different genotypes (wild-type, myb92 mutant, MYB92 overexpressing line) were grown in a green house, their flower were tagged and seeds were harvested 14 days after anthesis. Seeds were extracted from the siliques, stored at -80°C prior to RNA extraction.","Agilent",0,396,335,"2015-09-25","GEO","GSE93189","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (596,"P417E596","ISO 2016 Projet : effect on Arabidopsis thaliana.","dose response","dose","Apply differents dose of the product on plants.","Agilent",0,417,335,"2016-12-23","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (595,"P416E595","Comparison 39Ox and WT samples with or without Fe","normal vs transgenic comparaison,treated vs untreated comparison","fe deficiency (-fe) vs. fe sufficiency (+fe),genotype","WT (Col-0) and 39Ox seeds were surface sterilized and grown directly on +Fe or -Fe Hoagland medium for 6 days. 60 whole seedlings were harvest in liquid nitrogen and grinded to a fine powder. Total RNA was extracted with the Spectrum Plant total RNA kit (Sigma-Aldrich). 8 µg total RNA was sent to analysis. For validation of the results, RT-qPCR was performed using the same RNA samples.","NimbleGen",10344,416,335,"2016-12-22","GEO","GSE93093","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (566,"P390E566","Septante soil","normal vs transgenic comparaison","genotype","Arabidopsis thalian Col- line was transformed in order to obtain transgenic lines that produce opine compound (octopine and mannopine). Transgenic lines producing respectively octopine and mannopine and the WT line were grown in greenhouse under long-day condition in pots containing half commercial compost and half soil of la Mérantaise and watered with water. Whole plant aged of one month were harvested and frozen in liquid nitrogen. The plants were ground with a mortar an pestls and RNA extraction was performed with the RNeasy extraction kit (QIAGEN) with cristal of PVP. The RNA concentration was measured on a NANODrop spectrophotometer.","CATMA",10319,390,0,"2015-04-22","GEO","GSE93121","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (565,"P389E565","Microarray experiment 4","time course","light,time","Following a previously established strategy (Val et al., 2011, Nucleic Acids Res. 39, 92629274), we express, from an estradiol-inducible nuclear transgene, a trans-cleaving ribozyme directed against the nad9 mitochondrial mRNA and associated as a trailor sequence to a tRNA mimic. The latter serves as a shuttle and ensures mitochondrial uptake of the chimeric RNA through the natural tRNA import pathway. In mitochondria, the ribozyme triggers cleavage and degradation of the target mRNA. The impact of the nad9 mRNA knockdown on the overall plant transcriptome is analyzed over 4 successive days. Control plants express the shuttle RNA combined with a ribozyme that has no specific target in A. thaliana.","Agilent",10318,389,0,"2015-04-21","GEO","GSE93122","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (585,"P408E585","Effect of FTR mutation on Arabidopsis trancriptome","gene knock out","developmental stage,genotype"," - Comparison Mut1 vs sauv1 at the same developmental stage (8-10 leaves) as Mut1 at the time of sampling - Comparison Mut1 vs sauv1 at the same age (11 weeks) as Mut1 at the time of sampling - Comparison Mut2 vs sauv2 at the same developmental stage (8-10 leaves) as Mut1 at the time of sampling - Comparison Mut2 vs sauv2 at the same age (9 weeks) as Mut1 at the time of samplingMut1 = ftrbSauv1 = Col-0Mut2 = ftra1.a2Sauv2 = WT segregant from crossing between ftra1 (in WS genetic background) and ftra2 (in Col background)","Agilent",10341,408,335,"2016-03-15","GEO","GSE93183","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (456,"P288E456","AFFY_RIZ_2011_07","avirulent vs virulent","strain","The goal of the experiment is to characterize the rice leaf transcriptome response, upon the inoculation of susceptible and resistant rice leaves 24 hours post-infection. To that end, the experimental design includes the inoculation of susceptible Nipponbare rice leaves with Xoo strains BAI3 (race A1) and MAI1 (race A3), that will be compared to the response of resistant lines IRBB4 and IR64 rice lines. In addition, Nipponbare rice leaves will also be challenged with the BAI3&#61508;hrcC mutant that is affected in the production of a functional TTSS.","Affymetrix",0,288,238,"2011-06-21","GEO","34192","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (338,"P221E338","AFFY_Aphanomyces_Medicago","near isogenic lines (hif),normal vs ems mutant comparison","inoculation","Fifteen-day-old A17 or nfp-2 plants were grown in vitro on M medium. The entire roots were then inoculated (or not = controls) with a A. euteiches zoospores (105/ml) and harvested one day after inoculation. Three independent repeats were performed.","Affymetrix",0,221,7,"2010-03-02","GEO","20587","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (360,"P237E360","AFFY_ABA_Sunflower","treated vs untreated comparison"," compound","The experiment consisted of 3 repeats of four 12-day-old-plantlets of sunflower genotype SF193 (INRA code: XRQ) grown in growth chamber conditions and submitted to a 6-hour-treatment of 10 µM absissic acid or not.Growth conditions were 14h light at 23°C and 10h night at 20°C under fluorescent bulbs.Plants were grown in 6 hydroponic boxes containing 20 litres of aerated liquid culture medium (as described in Massonneau et al., 2001 Planta).Leaves (not cotyledons) 1 to 4 were harvested 4 hours after light onset and frozen immediately in liquid nitrogen.","Affymetrix",0,237,7,"2010-05-18","GEO","GSE22519","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (452,"P283E452","eQTL","treated vs untreated comparison"," compound,genotype","The experiment consisted of 3 repeats of 135 recombinant imbred lines and 9 repeats of the population parents SF193 (INRA code: XRQ), SF326 (INRA code: PSC8) and the F1 hybrid INEDI.The genotypes were grown in growth chamber conditions and submitted to a 6-hour-treatment of 0.5 µM absissic acid or not.Growth conditions were 14h light at 23°C and 10h night at 18°C under fluorescent bulbs.Plants were grown in 18 hydroponic boxes (9 for ABA treatment, 9 for control) containing 20 litres of aerated liquid culture medium (as described in Massonneau et al., 2001 Planta).Leaves (not cotyledons) from the 15-day-old-plantlets of sunflower were harvested 6 hours after treatment; the three repeats were pooled for each genotype and frozen immediately in liquid nitrogen.","Affymetrix",0,283,216,"2011-05-05","GEO","38812","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (492,"P321E492","AFFY_TOUR_2012-02","time course,treated vs untreated comparison","temperature,time","- At harvest seeds are dormant. They stay dormant if they are stored at -20°C  (D) and become non-dormant (ND) if they are stored 2 months at +20°C.Polysomal fractions were purified on sucrose gradients from sunflower axis isolated from dormant and non-dormant seeds imbibed at 10°C during 3h, 15h or 24h. - These fractions allow to identify the transcripts addressed to translation (translatome) during the seed imbibition process (3, 15 and 24h)- The translatome of 2 types of seeds are compared: Dormant vs Non-Dormant at the 3 time points.","Affymetrix",0,321,329,"2012-10-19","GEO","41944","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (545,"P371E545","Down/up regulation of NAD biosynthesis in Arabidopsis and role of L-Aspartate oxidase","genotype comparaison","genotype","Study of the biosynthesis of NAD in Arabidopsis. Involvment of L-Aspartate oxidase gene using T-DNA mutant (SAIL1145_B10) and overexpressor lines (promotor 35S, vector PCW162) at the same developmental stage (12 leaves)","NimbleGen",0,371,238,"2014-02-04","GEO","GSE120564","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (601,"P421E601","NGS micro D-2","null","cell type","One sample of total RNA extracted from whole embryo by laser assisted microdissection  was diluted from 5ng to 10pg to check sensitivity of the library construction and subsequent sensitivity of  transcript identification by RNA seq.","Illumina",0,421,0,"2017-04-05","GEO","GSE98313","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (602,"P422E602","NGS micro D-1","organ comparison","cell type","comparison of transcript accumulation in epidermis and mesophyll in Total RNA extracted by laser assisted microdissection from developping embryos at late heart stage.","Illumina",0,422,0,"2017-04-05","GEO","GSE98176","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (600,"P419E600","DYW2","gene knock out","genotype","TheT-DNA KO mutations are embryo-ethal for DYW2 and OTP100 so the homozygous mutants were produced by a temporal embryo-specific complementation using the promoter of ABI3. The resulting homozygous mutants were compared to their WT siblings. The T-DNA lines were GK_332A07 for DYW2 and SAIL_784_A11 for OTP100","Illumina",0,419,407,"2017-02-22","GEO","GSE100298","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (604,"P424E604","Hevea_TPD-seq","genotype comparaison,normal vs disease comparison","age","Identification of Tapping Panel Dryness (TPD)-affected trees in a polyclonal trials grown under standard condition. Trees were tapped since November 2010 every 2 days. Latex yield and TPD occurrence were monitored as well as latex RNA samples were collected twice a year for further analysis. At the end of the experiment, gene expression in latex of healthy and TPD trees were compared.","Illumina",0,424,0,"2017-05-05","GEO","GSE101568","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (605,"P425E605","RNAseq","genotype and ecotype comparison","genotype","Chinese Spring wheat (CS), Betzes barley (B) and CS/B ditelosomic addition line7HL (CS+7HL) plants were raised under a 16 h photoperiod and a day/night temperature regime of 20°C/16°C. For each genotype, total RNA was extracted from top leaves of three plants at Zadoks stage 11 (first leaf emerged) pooled into a single sample. Six biological replicates were collected for each genotype. cDNA libraries were generated for each sample using a TruSeq RNA sample Preparation Kit, randomly pooled by six and sequenced on 2 x 100bp paired-end flow cells using HiSeq2000 device (Illumina). ","Illumina",0,425,0,"2017-09-15","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (603,"P423E603","Rotenone","time course,treated vs untreated comparison"," compound,cell type,tissue type","NA","Agilent",0,423,335,"2017-04-27","GEO","GSE107673","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (606,"P426E606","AUXIN_DECAPITUATION_2exp","treated vs untreated comparison","developmental stage","Plants were decapitated Plants were treated with auxin","Illumina",0,426,238,"2017-10-24","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (544,"P370E544","Reproductive and Vegetative leaves","time course,treated vs untreated comparison","time,tissue type,traitment","Analysis of expression in Pea vegetative and reproductive leaves in untreated and nitrogen deficient plants during remobilization process between beginning of flowering, pod filling and the end of pod filling.","NimbleGen",0,370,335,"2014-01-31","GEO","GSE109789","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (613,"P433E613","Omics analysis of spontaneous albino orchids","genotype and ecotype comparison","species,genotype","Spontaneous albino adult orchids of Epipactis helleborine, Epipactis purpurata and Cephalanthera damasonium were compared to their green counterparts using couplped metabolomics and transcriptomics analyses. These plants were collected in the wild.","Illumina",0,433,407,"2018-04-09","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (608,"P428E608","NUC2 22°C versus 37°C","dose response,gene knock out,treated vs untreated comparison","genotype,temperature","A. thaliana  nuc2 seedlings (15 days-old) grown on soil (Percival) were treated or not at 37°C for 24h. ","Illumina",0,428,335,"2018-02-08","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (609,"P429E609","Fruits (3 stades) and Leaves (3 stages) comparisons.","genotype comparaison,time course,tissue comparison","developmental stage,time,tissue type","The objective of this study is to identify more precisely the genes used for the synthesis of a specific flavone in the orange which we know are expressed at the beginning of the growth of the young fruit in the zest as well as in the young leaf, and which are more from a certain stage of growth (1cm for the fruit, 3-4 cm for the leaf).","Illumina",0,429,335,"2018-02-08","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (610,"P430E610","CUC3 direct targets using DEX and DEX+Cyclo.","treated vs untreated comparison","traitment","10-days old seedlings (35S::CUC3-GR) grown in vitro in liquid growth medium were treated with either mock, DEX (10µM), Cycloheximide (100µM), or Cycloheximide+DEX. Seedlings were collected 6h after the treatment and snap frozen in liquid nitrogen. Total RNA were extracted using RNeasy Plant Mini kit from Qiagen following manufacturers instructions. 3 biological repeats were performed for each conditions.","Illumina",0,430,335,"2018-02-08","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (597,"P418E597","Transcriptome analysis of dgk5 mutant in response to flagellin","normal vs transgenic comparaison,treated vs untreated comparison"," compound,strain","We compare seedlings grown in liquid medium treated or not with flagellin. Seedlings are either Col-0 or the mutant dgk5.","Agilent",0,418,335,"2016-12-23","GEO","GSE107745","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (611,"P431E611","NGS2016_16_PATHO_DEPLETION_RIBO_ZERO","normal vs transgenic comparaison","bacterial ppr effector overexpressing lines vs. col0 (arabidopsis thaliana","Identification of molecular targets in A.thaliana  of 2 bacterial effectors from Ralstonia solanacearum and Erwinia amylovora. These proteins are overexpressed in A.thaliana lines and we compare their total and small RNA transcriptome to WT plants. ","Illumina",0,431,238,"2018-02-27","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (598,"P420E598","RNA-seq Seedcom","organ comparison","tissue type","Quantification of transcripts  accumulation in different compartments and sub-compartment of maize at 13 DAP","Illumina",0,420,329,"2017-01-26","GEO","GSE110060","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (614,"P434E614","RNAseq of microdissected tissues of maize internodes","genotype and ecotype comparison,treated vs untreated comparison","cell type","gene expression of xylem, parenchym and sclerechym from maize internodes in well watered and drought field conditions","Illumina",0,434,407,"2018-04-11","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (612,"P432E612","ZeaWall2","allele,genomic comparison","allele,genotype","Two pairs of BC2S2 maize lines carrying F271 or F288 alleles in an 18-20 Mb interval of chromosome 6 in an otherwise F271 genetic background were grown in the field at Mauguio (Hérault, France) in 2015 with two row replicates. Irrigation was applied during the summer to prevent water stress. The bottom third of upstream-ear internodes (without nodes) of three representative plants in each of the two replicates were harvested two days before silking. All samples were immediately frozen in liquid nitrogen and stored at -80°C.","CATMA",0,432,0,"2018-03-21","GEO","GSE115157","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (607,"P427E607","Transition","time course","age,developmental stage,light,organism part","Arabidopsis thaliana Col-0 plants were grown in soil, in growth chamber under white fluorescent light, under short-day (8 hours light/16 hours dark, SD) or long-day (16 hours light/8 hours dark, LD) conditions. Temperature in SD was 21°C during the light period and 18°C during the dark, humidity (65%) remained constant. In LD, temperature (21°C) and humidity (70%) remained constant. Plants were cultured for 4 weeks in individual pot, in SD then transferred in LD. Plants were analysed at different time points before transfer (T0) and after two, three and five days of transfer (T2, T3, T5). The second pair of leaves was collected before dusk, at Zeitgeber time 15 (ZT15) considering ZT 0 the switched on of the light. Three biological replicates were performed. The floral transition occurs between T0 and T5 based on AP1:GUS marker, a inflorescence meristem is not yet visible during this developmental window at the center of the rosette.","Illumina",0,427,0,"2018-02-07","GEO","GSE116123","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (615,"P435E615","BotrytisxN","normal vs disease comparison","disease state","Comparison of Arabidopsis responses to Botrytis infection in relation to nitrogen nutrition (sufficient vs limited).","Illumina",0,435,0,"2018-05-02","GEO","GSE116135","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (620,"P438E620","Cold Response","dose response,time course","temperature","The study is carried out on a once-flowering rose (R. x wichurana) on 22 external  plants in soil. The quantity of cold during the winter is estimated according to the Q10 method (Bidabé B., 1967.Ann. Physiol. vég.). The 6 first dormants buds of each shoot were harvested for each time point. 3 time point was selected: TQ10=0 (reference- no cold accumulation), TQ10=50 and TQ10=140. Two biological replicat (2 years) were carry out for a total of 6 samples. ","Illumina",0,438,407,"2018-07-06","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (622,"P440E622","ASCO","normal vs rnai mutant comparaison,normal vs transgenic comparaison","developmental stage","Splicing regulation in lncRNA ASCO overexpressing and depleted lines in comparison to the WT.","Illumina",0,440,238,"2018-07-10","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (624,"P441E624","PPR_patho","normal vs transgenic comparaison","bacterial ppr effector overexpressing lines vs. col0 (arabidopsis thaliana wt)","Identification of molecular targets in A.thaliana  of 2 bacterial effectors from Ralstonia solanacearum and Erwinia amylovora. These proteins are overexpressed in A.thaliana lines and we compare their total and small RNA transcriptome to WT plants. ","Illumina",0,441,238,"2018-07-17","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (591,"P413E591","Pilot transcriptomic study","genotype and ecotype comparison","block (2 levels) and population (6 levels)","RNA was extracted from young differentiating xylem and cambium collected on 24 4-year-old trees trialed in a common garden experiment at INRA Orléans. The trees corresponded to corresponding to 2 replicates of 12 genotypes from 6 natural populations. After RNA quantification, the RNA from xylem and cambium from the same tree were pooled and subjected to high-throughput sequencing on an Illumina HiSeq 2000 using 100 bp single-read indexing runs. All the samples were sequenced on one Illumina lane, yielding around 20 million of single-reads per sample.","Illumina",0,413,0,"2016-07-20","GEO","GSE117346","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (621,"P439E621","Transcriptomic assay on embryos after barley seeds gamma_irradiation","dose response,treated vs untreated comparison","developmental stage,dose","Barley dry seeds of variety Nur were irradiated by gamma-rays (Co-60) at doses 20 Gy (stimulatory) and 100 Gy (inhibitory). Immediately after irradiation we put seeds for germination to Petri dishes containing water filter paper; embryos were taken 2h after irradiation, embryos with embrionic roots were taken 24 and 48h after irradiation. The following comparisons have been made: non-irradiated (0 Gy), 2h vs. 20 Gy, 2h; 0 Gy, 24h vs. 20 Gy, 24h; 0 Gy, 48h vs. 20 Gy, 48h; 0 Gy, 24h vs. 100 Gy, 24h. Three independent biological replicates were used.","Agilent",0,439,335,"2018-07-06","GEO","GSE117356","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (618,"P423E618","Rotenone_Complex I mutants","gene knock out,genotype and ecotype comparison,time course","developmental stage,genotype,time","See Protocol INRA16-03_Rotenone_mutants.docx","Agilent",0,423,335,"2018-07-03","GEO","GSE117269","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (616,"P436E616","RNA seq on mpk8 and tcp14 mutants on dry and imbibited seeds","gene knock out","time","Dry harversted seed:  WT, mpk8, tcp14, 24h of imbibition in dark at 25°C: WT, mpk8, tcp14.","Illumina",0,436,0,"2018-06-25","GEO","GSE119344","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (617,"P437E617","Analysis of Arabidopsis mutants TOR/LST8/YAK1/MAKTOR","gene knock out","genotype","Comparison of in vitro grown plantlets for lst8, yak1 mutants and double mutants corresponding to suppressor lines. Comparison of in vitro grown plantlets for  mutants affected in LST8 interacting proteins (MAKTOR) compared to wild type. ","Illumina",0,437,0,"2018-06-25","GEO","GSE120003","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (619,"P438E619","Auxin sugar antagonist effect on budburst  ","treated vs untreated comparison","combined effect of two endogenous factors   ,other (specify)","Identification of the molecular network involved in the antagonistic effect between sugar and auxin on the budburst of the vegetative budworm of the rose bush ","Illumina",0,438,407,"2018-07-06","GEO","GSE123298","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (623,"P413E623","Validation of pilot transcriptomic study","genotype and ecotype comparison","block (2 levels) and population (6 levels)","Validation of pilot transcriptomic study","Illumina",0,413,0,"2018-07-13","GEO","GSE117347","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (625,"P442E625","Benzene effects on Brassica leaves","treated vs untreated comparison"," compound","Cabbages (Brassica oleracea variety Prover), placed in a 300L exposure chamber, are directly subjected to a single dose of benzene (3ppm) for 24 hours.","CATMA",10355,442,0,"2018-07-18","GEO","GSE119486","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (637,"P453E637","Fertirrigation experiment","dose response"," compound,dose","Each plant is incubated in a liquid medium containing or not the compounds. Several times after treatment, roots are removed for RNA extraction and analysis.","Agilent",10354,453,335,"2018-11-13","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (628,"P445E628","VASC1-BRAVO","genotype and ecotype comparison","genotype","During a screening of the Brachypodium mutant collection at Versailles, a line was identified with a marked vascular phenotype. The causal locus segregates Mendelian way (a recessive locus). A candidate gene was identified by ILLUMINA sequencing at the Joint Genome Institute (JGI) and by the ShoreMAP technique.The objective is to identify which genes whose expression is deregulated in this mutant explain the observed phenotype.","Agilent",0,445,335,"2018-07-24","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (629,"P446E629","Identification of differentially g","genotype and ecotype comparison,normal vs disease comparison","disease state,genotype","in liquid nitrogen. RNA were extracted.","Illumina",0,446,329,"2018-08-02","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (630,"P446E630","Identification of differentially expressed genes between two grapevines infected","genotype and ecotype comparison,normal vs disease comparison","disease state,genotype","CS and M were infected via the natural insect vector of the FD phytoplasma on one leaf (7 days contact).  Eight weeks post inoculation, the leaf in contact with the insects were sampled and put in liquid nitrogen. RNA were extracted.","Illumina",0,446,329,"2018-08-22","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (626,"P443E626","Impact of different doses of benzene on leaves of Brassica oleracea","treated vs untreated comparison"," compound","16 slides: 4 doses of benzene, 2 biological repetitions and dye-swap repetitions.Brassica oleracea plants (var Prover), placed in a 300L exposure chamber, are directly subjected to a single dose of benzene (150, 300, 1500 or 3000 ppb). Exposition time: during 1h. Comparison will be made between samples from plants exposed and unexposed to benzene.","Agilent",0,443,335,"2018-07-19","GEO","GSE119180","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (631,"P447E631","Treatment of Col0 and cpk","genotype comparaison,treated vs untreated comparison","genotype,stress","Seeds are sterilized and incubated 2 days in water in dark at 4°C. After sowing in liquid medium, seedlings are grown in a walk-in room (16h photoperiod) for 10 days. Then they are transferred to salt stress (150 mM NaCl) or control medium for 2h.","Illumina",0,447,329,"2018-08-31","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (632,"P448E632","Stem transcriptome in a triple WD40 mutant.","genotype comparaison","genotype","Not specified","Agilent",0,448,335,"2018-09-06","GEO","GSE119889","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (633,"P449E633","Acyltransferase Brachypodium-Arabidopsis","normal vs transgenic comparaison","genotype","Transcripts of Arabidopsis stem overexpressing an acyl transferase from Brachypodium are compared to wild type plants. The overexpression is under a specific promoter (C4H)","Agilent",0,449,335,"2018-09-13","GEO","GSE120114","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (635,"P451E635","NAR nodule vs root transcriptome","organ comparison","organism part,tissue type","biological material:  aeroponically grown cuttings of a Medicago sativa (alfalfa) accession that produces empty nodules when nitrogen-starved. Samples for transcriptome comparison:  isolated NAR nodules  (10 days post N-starvation) vs.  roots of the same plants (pools of 3 roots).","NimbleGen",0,451,335,"2018-09-28","GEO","GSE121571","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (547,"P373E547","R108 permissive medium versus non permissive medium.","null"," compound","Medicago truncatula R108 seedlings were inoculated with S. medicae WSM419 and were cultivated during three days on buffered nidulation medium solidified with Phytagel or Agar.","NimbleGen",0,373,335,"2014-02-21","GEO","GSE121771","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (548,"P373E548","R108 in symbiosis with rhizobia WT or rhizobia mutant for BacA.","gene knock out","am symbiosis)","Medicago truncatula ecotype R108 was inoculated with the symbiotic rhizobium Sinorhizobium meliloti strain Sm1021 and with its derivative mutant delta bacA. Nodules were collected 13 days after inoculation, and RNA were prepared for transcriptome analysis, there were three biological independant experiements.","NimbleGen",0,373,335,"2014-02-21","GEO","GSE121775","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (638,"P454E638","Characterization of 3 products by transcriptomic analysis on tomato leaves","treated vs untreated comparison","type of treatment product,-----,-----","4 modalities (products): M1(negative treatment= water), M2, M3 and M4.2 kinetic times: foliar samples 2h and 48h after treatment.2 répétitions biologiques : A et B (semis des 2 répétitions espacés de 24h).4 technical replicats: 4 plants/ modality/ kinetic time/ biological replictas.ARNs of technical replicats have been pooled post-lyse.","Agilent",0,454,335,"2018-11-15","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (636,"P452E636","Identification of target genes regulated by CUC2 in A.thaliana shoot apical meristem.","genotype comparaison","genotype","identify genes specifically targeted by CUP-SHAPED COTYLEDON 2 (CUC2), a transcription factor required for embryonic shoot meristem formation and specification of the organ boundary in A.thaliana","NimbleGen",10353,452,335,"2018-10-23","GEO","GSE121953","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (627,"P444E627","Transcriptome changes in pea leaves with sulfur deficency/sufficienty during reproductive phase.","gene knock out,genotype comparaison,treated vs untreated comparison","genotype","Comparison of:1- The leaf transcriptome of pea subjected or not to sulfur deficiency during the reproductive phase (S+ versus S )2- The leaf transcriptome of wild-type and mutant lines for a sulfur transporter (two TILLING alleles) grown under sulfur sufficient conditions : WT1/Mut1 S+ et WT2/Mut2 S+3- The leaf transcriptome of wild-type and mutant lines for a sulfur transporter (two TILLING alleles) grown under sulfur deficient conditions : WT1/Mut1 S+ et WT2/Mut2 S+","Agilent",0,444,335,"2018-07-23","GEO","GSE121967","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (639,"P455E639","Transcriptome analysis of dgk5 mutant in response to flagellin, with 3 biological replicats.","normal vs transgenic comparaison,treated vs untreated comparison"," compound,strain","We compare seedlings grown in liquid medium treated or not with flagellin. Seedlings are either Col-0 or the mutant dgk5.","Agilent",0,455,335,"2018-11-21","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (634,"P450E634","Transilice","treated vs untreated comparison","other (specify)","Brassica napus were grown on hydropnic conditions using Hoagland nutrient solution containing or not 1.7 mM of Si","Illumina",0,450,329,"2018-09-28","GEO","GSE122968","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (502,"P330E502","RA12-02_ABA-seed","gene knock out,genomic comparison","genotype","Transcriptome comparison of  developing seeds harvested at 10 DAP from two ABA-deficient mutants (nced2 nced5 nced9 and nced2 nced5 nced6 nced9)  and wild type (Columbia-0)","NimbleGen",0,330,329,"2013-02-04","GEO","None","Arrays");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (640,"P413E640","Large genotype and population comparisons in Populus nigra","genotype and ecotype comparison","block (4 levels) and population (11 levels)","Large genotype comparisons: 241 genotype from 11 populations.RNA was extracted from young differentiating xylem and cambium collected on 480 4-year-old trees trialed in a common garden experiment at INRA Orléans. The trees corresponded to corresponding to 2 replicates of 241 genotypes from 11 natural populations. After RNA quantification, the RNA from xylem and cambium from the same tree were pooled and subjected to high-throughput sequencing on an Illumina HiSeq 2000 using 100 bp single-read indexing runs. All the samples were sequenced on one Illumina lane, yielding around 20 million of single-reads per sample.","Illumina",0,413,8,"2019-01-11","GEO","None","RNA-Seq");
INSERT INTO experiment (experiment_id,experiment_idname,experiment_name,experiment_type,experiment_factors,note,array_type,protocol_id,project_id,user_id,submission_date,repository_db,repository_access,analysis_type) VALUES (641,"P456E641","transcriptome of siliques from 4 disctinct genotypes","genomic comparison","genotype","RNA comparison from siliques of 4 disctincts genotypes ","Illumina",0,456,238,"2019-02-01","GEO","None","RNA-Seq");
